<!doctype html><html><!-- InstanceBegin template="/Templates/mainPage.dwt" codeOutsideHTMLIsLocked="false" -->
<head>
<!-- Global site tag (gtag.js) - Google Analytics --> 
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-74740093-2"></script> 
<script>
    window.dataLayer = window.dataLayer || [];
    function gtag() { dataLayer.push(arguments); }
    gtag('js', new Date());

    gtag('config', 'UA-74740093-2');
  </script>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- InstanceBeginEditable name="doctitle" -->
<title>4DN Preprints</title>
<!-- InstanceEndEditable -->
<link href="Assets/styles/mainStyle.css" rel="stylesheet" type="text/css">
<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
<link rel="stylesheet" type="text/css" href="Assets/js/slick/slick.css" />
<link rel="stylesheet" type="text/css" href="Assets/js/slick/slick-theme.css" />
<!-- InstanceBeginEditable name="head" -->
<!-- InstanceEndEditable -->
<script defer type="text/javascript" src="Assets/js/jquery/jquery-3.3.1.min.js"></script> 
<script defer type="text/javascript" src="Assets/js/jquery/jquery-migrate-1.4.1.min.js"></script> 
<script defer type="text/javascript" src="Assets/js/jquery/jquery-migrate-3.0.0.min.js"></script> 
<script defer type="text/javascript" src="Assets/js/slick/slick.min.js"></script> 
<script defer type="text/javascript" src="Assets/js/main.js"></script>
</head>

<body>
<div class="mainMargin"> 
  <!-- left margin --> 
</div>
<div id="columnWrapper">
  <div id="mainwrapper">
    <header class="framedContainer">
      <div id="logo"><a href="index.html"><img src="Assets/images/4dn-logo_1.png" alt="" /></a>
        <div>4DN Web Portal</div>
      </div>
      <div id="headerRight">
        <nav> <span> <a href="https://data.4dnucleome.org/" target="_blank" title="Launch Data Portal">Launch Data
          Portal</a> </span> <span> <a href="mailto:oh@4dnucleome.org" title="Contact us">Contact Us</a> </span> </nav>
        <div id="searchInput">  
		  <script>
			(function() {
			  var cx = '012010507559311424712:8m_5bphxwsi';
			  var gcse = document.createElement('script');
			  gcse.type = 'text/javascript';
			  gcse.async = true;
			  gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
			  var s = document.getElementsByTagName('script')[0];
			  s.parentNode.insertBefore(gcse, s);
			})();
		  </script>
          <gcse:searchbox-only gname="4dnsitewidesearch"></gcse:searchbox-only>
           </div>
      </div>
    </header>
    <div id="content" class="framedContainer">
      <nav id="topMenu" tabindex="0">
        <div class="mobileOnly"> <i class="material-icons" id="navMenuIcon"> menu </i> Navigation Menu</div>
        <div id="topMenuContent">
          <div class="topMenuItem"> <a href="index.html">Home</a> </div>
          <div class="topMenuItem"> <a href="https://commonfund.nih.gov/4dnucleome" target="_blank">Overview <i class="material-icons openInNewIcons">open_in_new</i></a> </div>
          <div class="topMenuItem"> <a href="https://data.4dnucleome.org/" target="_blank">Data <i class="material-icons openInNewIcons">open_in_new</i></a> </div>
          <div class="topMenuItem"> Resources
            <div class="subMenu">
              <div> <a href="cell-lines.html">Cell Lines</a> </div>
              <div> <a href="protocols.html">Protocols</a> </div>
              <div> <a href="software.html">Software</a> </div>
            </div>
          </div>
          <div class="topMenuItem"> Publications
            <div class="subMenu">
              <div> <a href="https://www.ncbi.nlm.nih.gov/sites/myncbi/1FUN7Gh4ocnMkh/collections/60033344/public/"
                    target="_blank">4DN Publications <i class="material-icons openInNewIcons">open_in_new</i></a> </div>
              <div> <a href="pre-prints.html">4DN Preprints</a> </div>
              <div> Further Reading
                <div class="subMenu">
                  <div> <a href="https://www.ncbi.nlm.nih.gov/sites/myncbi/1FUN7Gh4ocnMkh/collections/60033609/public/"
                        target="_blank">Genome Interactions <i class="material-icons openInNewIcons">open_in_new</i></a> </div>
                  <div> <a href="https://www.ncbi.nlm.nih.gov/sites/myncbi/1FUN7Gh4ocnMkh/collections/60033466/public/"
                        target="_blank">Data Analysis <i class="material-icons openInNewIcons">open_in_new</i></a> </div>
                </div>
              </div>
            </div>
          </div>
          <div class="topMenuItem"> <a href="policies.html">Policies</a> </div>
          <div class="topMenuItem"> <a href="outreach.html">Outreach</a>
		  	<div class="subMenu"> <a href="4dn-kickoff-meeting-2020.html">2020 4DN Phase-II Kick-off Meeting</a> </div>
		  </div>
		<div class="topMenuItem"> <a href="4dn-annual-meeting-2021.html">2021 Annual Meeting</a> </div>
			<!--
          <div class="topMenuItem"> 2019 4DN-ASCB Meeting
            <div class="subMenu">
              <div> <a href="../4dn-ascb-meeting-2019.html">Meeting Home</a> </div>
              <div> <a href="../4dn-ascb-meeting-2019-agenda.html">Agenda</a> </div>
              <div> <a href="../4dn-ascb-meeting-2019-speakers.html">Keynote and Invited Speakers</a> </div>
              <div> <a href="../4dn-ascb-meeting-2019-hotel-travel.html">Hotel/Travel</a> </div>
              <div> <a href="../4dn-ascb-meeting-2019-abstract-guidelines.html">Abstract guidelines</a> </div>
              <div> <a href="https://washington.org/" target="_blank">Explore Washington, DC</a> </div>
              <div> <a href="../4dn-ascb-meeting-2019-committee.html">Planning Committee Members</a> </div>
            </div>
          </div>
			-->
          <div class="topMenuItem"> Contacts
            <div class="subMenu">
              <div> <a href="centers-and-teams.html">4DN Centers and Teams</a> </div>
			  <div> <a href="genomic-program-administrator.html">4DN Genomic Program Administrator</a> </div>
              <div> <a href="TCPA-members.html">TCPA Awardees</a> </div>
              <div> <a href="associate-members.html">Associate Members</a> </div>
              <div> <a href="external-program-consultants.html">External Program Consultants</a> </div>
            </div>
          </div>
        </div>
      </nav>
      <div id="middleContainer">
        <section id="mainContent"> <!-- InstanceBeginEditable name="Main contents" -->
			<h2>4DN pre-prints</h2>
			<b>See the <a target="_blank" href="http://connect.biorxiv.org/relate/content/66">4DN bioRxiv channel.</a></b>
			
			<br/>
			<br/>
			<br/>
			<table id="GeneralTables" style="width:100%">
        
        <tr id="Header1">
            <th colspan="4">Pre-print publications funded by NIH 4DN program</th>
        </tr>
        
        <tr id="Header2">
            <th width="40%">Title</th>
            <th width="35%">Authors</th>
            <th width="15%">DOI</th>
            <th width="10%">Date</th>
        </tr>
        

        <!--2021-->
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2022.02.21.481225">Nucleome Browser: An integrative and multimodal data navigation platform for 4D Nucleome</a></td><td>Zhu, X., Zhang, Y., Wang, Y., Tian, D., Belmont, A. S., Swedlow, J. R., Ma, J.
        </td><td>10.1101/2022.02.21.481225</td><td>2022-02-22</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2022.02.13.480215">Quantitative fate mapping: Reconstructing progenitor field dynamics via retrospective lineage barcoding</a></td><td>Fang, W., Bell, C. M., Sapirstein, A., Asami, S., Leeper, K., Zack, D. J., Ji, H., Kalhor, R.
        </td><td>10.1101/2022.02.13.480215</td><td>2022-02-14</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2022.02.01.475239">Widespread contribution of transposable elements to the rewiring of mammalian 3D genomes and gene regulation</a></td><td>Choudhary, M. N., Quaid, K., Xing, X., Schmidt, H., Wang, T.
        </td><td>10.1101/2022.02.01.475239</td><td>2022-02-03</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2022.01.18.476849">Exploring genomic data coupled with 3D chromatin structures using the WashU Epigenome Browser</a></td><td>Li, D., Purushotham, D., Harrison, J. K., Wang, T.
        </td><td>10.1101/2022.01.18.476849</td><td>2022-01-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2022.01.18.476836">Massively parallel genomic perturbations with multi-target CRISPR reveal new insights on Cas9 activity and DNA damage responses at endogenous sites</a></td><td>Zou, R. S., Marin-Gonzalez, A., Liu, Y., Liu, H. B., Shen, L., Dveirin, R., Luo, J. X. J., Kalhor, R., Ha, T.
        </td><td>10.1101/2022.01.18.476836</td><td>2022-01-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.02.24.432639">Reliance of neuronal gene expression on cohesin scales with chromatin loop length</a></td><td>Calderon, L., Weiss, F. D., Beagan, J. A., Oliveira, M. S., Wang, Y.-F., Carroll, T., Dharmalingam, G., Gong, W., Tossell, K., de Paola, V., Whilding, C., Ungless, M. A., Fisher, A. G., Phillips-Cremins, J. E., Merkenschlager, M.
        </td><td>10.1101/2021.02.24.432639</td><td>2022-01-02</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.12.16.472900">Divergent clonal differentiation trajectories of T cell exhaustion</a></td><td>Daniel, B., Yost, K. E., Sandor, K., Xia, Y., Qi, Y., Hiam-Galvez, K. J., Meier, S. L., Belk, J., Giles, J. R., Wherry, E. J., Chang, H., Egawa, T., Satpathy, A.
        </td><td>10.1101/2021.12.16.472900</td><td>2021-12-17</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.12.10.472156">Single-fiber nucleosome density shapes the regulatory output of a mammalian chromatin remodeling enzyme</a></td><td>Abdulhay, N. J., Hsieh, L. J., McNally, C. P., Ketavarapu, M., Kasinathan, S., Nanda, A. S., Ostrowski, M. S., Wu, K., Moore, C. M., Goodarzi, H., Narlikar, G. J., Ramani, V.
        </td><td>10.1101/2021.12.10.472156</td><td>2021-12-11</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.03.31.437978">Chromatin interaction aware gene regulatory modeling with graph attention networks</a></td><td>Karbalayghareh, A., Sahin, M., Leslie, C. S.
        </td><td>10.1101/2021.03.31.437978</td><td>2021-12-10</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.12.03.471109">CRISPR Cas13-based tools to track and manipulate endogenous telomeric repeat-containing RNAs in living cells</a></td><td>Xu, M., Chigumira, T., Chen, Z., Tones, J., Zhao, R., Dahl, K. N., Chenoweth, D. M., Zhang, H.
        </td><td>10.1101/2021.12.03.471109</td><td>2021-12-04</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.11.26.470134">Allelic correlation is a marker of tradeoffs between barriers to transmission of expression variability and signal responsiveness in genetic networks</a></td><td>Boe, R. H., Ayyappan, V., Schuh, L., Raj, A.
        </td><td>10.1101/2021.11.26.470134</td><td>2021-11-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.11.05.467513">Single-Cell Multiomics Defines Tolerogenic Extrathymic Aire-Expressing Populations with Unique Homology to Thymic Epithelium</a></td><td>Wang, J., Lareau, C. A., Bautista, J. L., Gupta, A. R., Sandor, K., Germino, J., Yin, Y., Arvedson, M. P., Reeder, G. C., Cramer, N. T., Xie, F., Ntranos, V., Satpathy, A. T., Anderson, M. S., Gardner, J. M.
        </td><td>10.1101/2021.11.05.467513</td><td>2021-11-06</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.11.02.466589">Loop-extrusion and polymer phase-separation can co-exist at the single-molecule level to shape chromatin folding</a></td><td>Conte, M., Irani, E., Chiariello, A. M., Abraham, A., Bianco, S., Esposito, A., Nicodemi, M.
        </td><td>10.1101/2021.11.02.466589</td><td>2021-11-02</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.10.30.465508">CTCF blocks anti-sense transcription initiation at divergent gene promoters</a></td><td>Luan, J., Syrett, C. M., Vermunt, M. W., Cote, A., Tome, J. M., Zhang, H., Huang, A., Luppino, J. M., Keller, C. A., Giardine, B. M., Zhang, S., Dunagin, M. C., Zhang, Z., Joyce, E. F., Lis, J. T., Raj, A., Hardison, R. C., Blobel, G. A.
        </td><td>10.1101/2021.10.30.465508</td><td>2021-10-30</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.10.21.465351">Kinetic principles underlying pioneer function of GAGA transcription factor in live cells</a></td><td>Tang, X., Li, T., Liu, S., Wisniewski, J., Zheng, Q., Lavis, L., Rong, Y., Wu, C.
        </td><td>10.1101/2021.10.21.465351</td><td>2021-10-23</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.10.14.464435">The 4D Nucleome Data Portal: a resource for searching and visualizing curated nucleomics data</a></td><td>Reiff, S. B., Schroeder, A. J., Kirli, K., Cosolo, A., Bakker, C., Mercado, L., Lee, S. B., Veit, A. D., Balashov, A. K., Vitzthum, C., Ronchetti, W., Pitman, K. M., Johnson, J., Ehmsen, S. R., Kerpedjiev, P., Abdennur, N. A., Imakaev, M., Ozturk, S. U., Camoglu, U., Mirny, L., Gehlenborg, N., Alver, B. H., Park, P. J.
        </td><td>10.1101/2021.10.14.464435</td><td>2021-10-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.10.13.464280">Optogenetic control of the Bicoid morphogen reveals fast and slow modes of gap gene regulation</a></td><td>Singh, A. P., Wu, P., Ryabichko, S., Raimundo, J., Swan, M., Wieschaus, E., Gregor, T., Toettcher, J. E.
        </td><td>10.1101/2021.10.13.464280</td><td>2021-10-14</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.09.27.461866">Spatiotemporal co-dependency between macrophages and exhausted CD8+ T cells in cancer</a></td><td>Kersten, K., Hu, K. H., Combes, A. J., Samad, B., Harwin, T., Ray, A., Rao, A. A., Cai, E., Marchuk, K., Artichoker, J., Courau, T., Shi, Q., Belk, J., Satpathy, A. T., Krummel, M. F.
        </td><td>10.1101/2021.09.27.461866</td><td>2021-09-28</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.09.27.461389">Regional and clonal T cell dynamics at single cell resolution in immune checkpoint blockade</a></td><td>Pai, J. A., Chow, A., Sauter, J., Mattar, M., Rizvi, H., Woo, H. J., Shah, N., Uddin, F., Quintanal-Villalonga, A., Chan, J. M., Manoj, P., Allaj, V., Baine, M., Chaft, J. E., Plodkowski, A. J., Won, H., Wells, D., Donoghue, M. T. A., de Stanchina, E., Sen, T., Wolchok, J. D., Houck-Loomis, B., Merghoub, T., Rudin, C. M., Satpathy, A. T., Hellmann, M. D.
        </td><td>10.1101/2021.09.27.461389</td><td>2021-09-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.08.24.457552">Pairs and Pairix: a file format and a tool for efficient storage and retrieval for Hi-C read pairs</a></td><td>Lee, S., Vitzthum, C., Alver, B. H., Park, P. J.
        </td><td>10.1101/2021.08.24.457552</td><td>2021-08-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.08.24.457555">Biochemically distinct cohesin complexes mediate positioned loops between CTCF sites and dynamic loops within chromatin domains</a></td><td>Liu, Y., Dekker, J.
        </td><td>10.1101/2021.08.24.457555</td><td>2021-08-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.08.06.455141">BCL6-dependent TCF-1+ progenitor cells maintain effector and helper CD4 T cell responses to persistent antigen</a></td><td>Xia, Y., Sandor, K., Pai, J. A., Daniel, B., Raju, S., Wu, R., Hsiung, S., Qi, Y., Yangdon, T., Okamoto, M., Schreiber, R. D., Murphy, K. M., Satpathy, A. T., Egawa, T.
        </td><td>10.1101/2021.08.06.455141</td><td>2021-08-09</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.08.08.455532">Cell type determination for cardiac differentiation occurs soon after seeding of human induced pluripotent stem cells</a></td><td>Jiang, C. L., Goyal, Y., Jain, N., Wang, Q., Truitt, R. E., Cote, A. J., Emert, B., Mellis, I. A., Kiani, K., Yang, W., Jain, R., Raj, A.
        </td><td>10.1101/2021.08.08.455532</td><td>2021-08-08</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.08.05.455340">Heterochromatin diversity modulates genome compartmentalization and loop extrusion barriers</a></td><td>Spracklin, G., Abdennur, N. A., Imakaev, M., Chowdhury, N., Pradhan, S., Mirny, L., Dekker, J.
        </td><td>10.1101/2021.08.05.455340</td><td>2021-08-06</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.07.20.453146">SARS-CoV-2 Restructures the Host Chromatin Architecture</a></td><td>Wang, R., Lee, J.-H., Xiong, F., Kim, J., Al Hasani, L., Yuan, X., Shivshankar, P., Krakowiak, J., Qi, C., Wang, Y., Eltzschig, H. K., Li, W.
        </td><td>10.1101/2021.07.20.453146</td><td>2021-07-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.07.11.451976">Mapping the nuclear microenvironment of genes at a genome-wide scale</a></td><td>Yildirim, A., Hua, N., Boninsegna, L., Polles, G., Gong, K., Hao, S., Li, W., Zhou, X. J., Alber, F.
        </td><td>10.1101/2021.07.11.451976</td><td>2021-07-12</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.06.24.449850">Macrophage inflammatory and regenerative response periodicity is programmed by cell cycle and chromatin state</a></td><td>Daniel, B., Belk, J. A., Meier, S. L., Chen, A. Y., Sandor, K., Qi, Y., Kitano, H., Wheeler, J. R., Foster, D. S., Januszyk, M. S., Longaker, M. T., Chang, H. Y., Satpathy, A. T.
        </td><td>10.1101/2021.06.24.449850</td><td>2021-06-25</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.06.21.449316">ShinyArchR.UiO: User-friendly, integrative and open-source tool for visualisation of single-cell ATAC-seq data using ArchR</a></td><td>Sharma, A., Akshay, A., Rogne, M., Eskeland, R.
        </td><td>10.1101/2021.06.21.449316</td><td>2021-06-22</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.06.10.447969">Three-dimensional organization of chromatin associated RNAs and their role in chromatin architecture in human cells</a></td><td>Calandrelli, R., Wen, X., Nguyen, T. C., Chen, C.-J., Qi, Z., Chen, W., Yan, Z., Wu, W., Zaleta-Rivera, K., Hu, R., Yu, M., Wang, Y., Ma, J., Ren, B., Zhong, S.
        </td><td>10.1101/2021.06.10.447969</td><td>2021-06-11</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.06.09.447760">Multi-color super-resolution imaging to study human coronavirus RNA during cellular infection</a></td><td>Wang, J., Han, M., Wang, H., Moeckl, L., Zeng, L., Moerner, W. E., Qi, L. S.
        </td><td>10.1101/2021.06.09.447760</td><td>2021-06-09</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.06.08.447626">Systematic reconstruction of the cellular trajectories of mammalian embryogenesis</a></td><td>Qiu, C., Cao, J., Li, T., Srivatsan, S., Huang, X., Calderon, D., Noble, W. S., Disteche, C. M., Spielmann, M., Moens, C. B., Trapnell, C., Shendure, J.
        </td><td>10.1101/2021.06.08.447626</td><td>2021-06-09</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.06.06.447300">Interrogation of the Dynamic Properties of Higher-Order Heterochromatin Using CRISPR/dCas9</a></td><td>Gao, Y., Han, M., Shang, S., Wang, H., Qi, L. S.
        </td><td>10.1101/2021.06.06.447300</td><td>2021-06-07</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.05.31.446382">Micro-Meta App: an interactive software tool to facilitate the collection of microscopy metadata based on community-driven specifications</a></td><td>Rigano, A., Ehmsen, S., Ozturk, S. U., Ryan, J., Balashov, A., Hammer, M., Kirli, K., Bellve, K., Boehm, U., Brown, C. M., Chambers, J. J., Coleman, R. A., Cosolo, A., Faklaris, O., Fogarty, K., Guilbert, T., Hamacher, A. B., Itano, M. S., Keeley, D. P., Kunis, S., Lacoste, J., Laude, A., Ma, W., Marcello, M., Montero-Llopis, P., Nelson, G., Nitschke, R., Pimentel, J. A., Weidtkamp-Peters, S., Park, P. J., Alver, B., Grunwald, D., Strambio-De-Castillia, C.
        </td><td>10.1101/2021.05.31.446382</td><td>2021-05-31</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.04.02.020990">Cell-type specialization in the brain is encoded by specific long-range chromatin topologies</a></td><td>Winick-Ng, W., Kukalev, A., Harabula, I., Zea Redondo, L., Meijer, M., Serebreni, L., Bianco, S., Szabo, D., Chiariello, A. M., Irastorza Azcarate, I., Fiorillo, L., Musella, F., Thieme, C., Irani, E., Torlai Triglia, E., Kolodziejczyk, A. A., Abentung, A., Apostolova, G., Paul, E. J., Franke, V., Kempfer, R., Akalin, A., Teichmann, S., Dechant, G., Ungless, M. A., Nicodemi, M., Castelo-Branco, G., Pombo, A.
        </td><td>10.1101/2020.04.02.020990</td><td>2021-05-11</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.04.26.441547">Integrated spatial genomics in tissues reveals invariant and cell type dependent nuclear architecture
        </a></td><td>Takei, Y., Zheng, S., Yun, J., Shah, S., Pierson, N., White, J., Schindler, S., Tischbirek, C., Yuan, G.-C., Cai, L.
        </td><td>10.1101/2021.04.26.441547</td><td>2021-04-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.04.25.441198">Towards community-driven metadata standards for light microscopy: tiered specifications extending the OME model
        </a></td><td>Hammer, M., Huisman, M., Rigano, A., Boehm, U., Chambers, J. J., Gaudreault, N., North, A. J., Pimentel, J. A., Sudar, D., Bajcsy, P., Brown, C. M., Corbett, A. D., Faklaris, O., Lacoste, J., Laude, A., Nelson, G., Nitschke, R., Farzam, F., Smith, C., Grunwald, D., Strambio-De-Castillia, C..
        </td><td>10.1101/2021.04.25.441198</td><td>2021-04-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://arxiv.org/abs/1910.11370">A perspective on Microscopy Metadata: data provenance and quality control
        </a></td><td>Maximiliaan Huisman, Mathias Hammer, Alex Rigano, Ulrike Boehm, James J. Chambers, Nathalie Gaudreault, Alison J. North, Jaime A. Pimentel, Damir Sudar, Peter Bajcsy, Claire M. Brown, Alexander D. Corbett, Orestis Faklaris, Judith Lacoste, Alex Laude, Glyn Nelson, Roland Nitschke, David Grunwald, Caterina Strambio-De-Castillia
        </td><td>arXiv:1910.11370</td><td>2021-04-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.04.16.440239">Stripenn detects architectural stripes from chromatin conformation data using computer vision
        </a></td><td>Yoon, S., Vahedi, G.
        </td><td>10.1101/2021.04.16.440239</td><td>2021-04-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.03.31.436355">Quantification of durable CRISPR-based gene silencing activity
        </a></td><td>Nakamura, M., Ivec, A., Gao, Y., Qi, L. S.
        </td><td>10.1101/2021.03.31.436355</td><td>2021-03-31</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.03.01.433416">Polymer physics and machine learning reveal a combinatorial code linking chromatin 3D architecture to 1D epigenetics
        </a></td><td>Esposito, A., Bianco, S., Chiariello, A. M., Abraham, A., Fiorillo, L., Conte, M., Campanile, R., Nicodemi, M.
        </td><td>10.1101/2021.03.01.433416</td><td>2021-03-01</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.02.09.430314">Disruption of nuclear architecture as a cause of COVID-19 induced anosmia
        </a></td><td>Zazhytska, M., Kodra, A., Hoagland, D. A., Fullard, J., Shayya, H. J., Omer, A., Firestein, S., Gong, Q., Canoll, P., Goldman, J. E., Roussos, P., tenOever, B. R., Overdevest, J. B., Lomvardas, S.
        </td><td>10.1101/2021.02.09.430314</td><td>2021-02-09</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.01.06.425599">Genome-wide variability in recombination activity is associated with meiotic chromatin organization
        </a></td><td>Jin, X., Fudenberg, G., Pollard, K. S.
        </td><td>10.1101/2021.01.06.425599</td><td>2021-01-07</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2021.01.05.425437">Cohesin-mediated loop anchors confine the location of human replication origins
        </a></td><td>Emerson, D. J., Zhao, P. A., Klein, K., Ge, C., Zhou, L., Sasaki, T., Yang, L., Venvev, S. V., Gibcus, J. H., Dekker, J., Gilbert, D. M., Phillips-Cremins, J. E.
        </td><td>10.1101/2021.01.05.425437</td><td>2021-01-06</td></tr>

        <!--2020-->
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.12.26.424448">Systematic evaluation of chromosome conformation capture assays
        </a></td><td>Akgol Oksuz, B., Yang, L., Abraham, S., Venev, S. V., Krietenstein, N., Parsi, K. M., Ozadam, H., Oomen, M. E., Nand, A., Mao, H., Genga, R. M. J., Maehr, R., Rando, O., Mirny, L., Gibcus, J. H., Dekker, J.
        </td><td>10.1101/2020.12.26.424448</td><td>2020-12-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.12.24.424222">Mechanical frustration of phase separation in the cell nucleus by chromatin
        </a></td><td>Zhang, Y., Lee, D. S. W., Meir, Y., Brangwynne, C. P., Wingreen, N. S.
        </td><td>10.1101/2020.12.24.424222</td><td>2020-12-24</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.11.02.366062">Multi-omics analysis of chromatin accessibility and interactions with transcriptome by HiCAR
        </a></td><td>Wei, X., Xiang, Y., Shan, R., Peters, D. T., Sun, T., Lin, X., Li, W., Diao, Y.
        </td><td>10.1101/2020.11.02.366062</td><td>2020-12-19</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.12.18.423551">Simple biochemical features underlie transcriptional activation domain diversity and dynamic, fuzzy binding to Mediator
        </a></td><td>Sanborn, A. L., Yeh, B. T., Feigerle, J. T., Hao, C. V., Townshend, R. J. L., Aiden, E. L., Dror, R. O., Kornberg, R. D.
        </td><td>10.1101/2020.12.18.423551</td><td>2020-12-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.12.13.422543">SnapHiC: a computational pipeline to map chromatin contacts from single cell Hi-C data
        </a></td><td>Yu, M., Abnousi, A., Zhang, Y., Li, G., Lee, L., Chen, Z., Fang, R., Wen, J., Sun, Q., Li, Y., Ren, B., Hu, M.
        </td><td>10.1101/2020.12.13.422543</td><td>2020-12-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.12.13.422537">Multiscale and integrative single-cell Hi-C analysis with Higashi
        </a></td><td>Zhang, R., Zhou, T., Ma, J.
        </td><td>10.1101/2020.12.13.422537</td><td>2020-12-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.11.29.383067">SPICEMIX: Integrative single-cell spatial modeling for inferring cell identity
        </a></td><td>Chidester, B., Zhou, T., Ma, J.
        </td><td>10.1101/2020.11.29.383067</td><td>2020-11-30</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.11.27.386599">3D super-resolution fluorescence microscopy maps the variable molecular architecture of the Nuclear Pore Complex
        </a></td><td>Jimenez Sabinina, V., Hossain, M. J., Heriche, J.-K., Hoess, P., Nijmeijer, B., Mosalaganti, S., Kueblbeck, M., Callegari, A., Szymborska, A., Beck, M., Ries, J., Ellenberg, J.
        </td><td>10.1101/2020.11.27.386599</td><td>2020-11-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.11.26.400739">DANGO: Predicting higher-order genetic interactions
        </a></td><td>Zhang, R., Ma, J., Ma, J.
        </td><td>10.1101/2020.11.26.400739</td><td>2020-11-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.11.24.396713">Chromatin Fiber Folding Represses Transcription and Loop Extrusion in Quiescent Cells
        </a></td><td>Swygert, S. G., Lin, D., Portillo-Ledesma, S., Lin, P.-Y., Hunt, D. R., Kao, C.-F., Schlick, T., Noble, W. S., Tsukiyama, T.
        </td><td>10.1101/2020.11.24.396713</td><td>2020-11-24</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.11.20.390765">Single-cell landscape of nuclear configuration and gene expression during stem cell differentiation and X inactivation
        </a></td><td>Bonora, G., Ramani, V., Singh, R., Fang, H., Jackson, D., Srivatsan, S., Qiu, R., Lee, C., Trapnell, C., Shendure, J., Duan, Z., Deng, X., Noble, W. S., Disteche, C. M.
        </td><td>10.1101/2020.11.20.390765</td><td>2020-11-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.11.03.366914">Transcriptional Network Orchestrating Regional Patterning of Cortical Progenitors
        </a></td><td>Ypsilanti, A. R., Pattabiraman, K., Catta-Preta, R., Golonzhka, O., Lindtner, S., Tang, K., Jones, I., Abnousi, A., Juric, I., Hu, M., Shen, Y., Dickel, D. E., Visel, A., Pennachio, L. A., Hawrylycz, M., Thompson, C., Zeng, H., Barozzi, I., Nord, A. S., Rubenstein, J. L. R.
        </td><td>10.1101/2020.11.03.366914</td><td>2020-11-04</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.10.28.359869">HiC-ACT: Improved Detection of Chromatin Interactions from Hi-C Data via Aggregated Cauchy Test
        </a></td><td>Lagler, T. M., Yang, Y., Abnousi, A., Hu, M., Li, Y.
        </td><td>10.1101/2020.10.28.359869</td><td>2020-10-29</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.10.27.357756">HiCRep.py: Fast comparison of Hi-C contact matrices in Python
        </a></td><td>Lin, D., Sanders, J., Noble, W. S.
        </td><td>10.1101/2020.10.27.357756</td><td>2020-10-28</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.10.26.356329">DNA-loop extruding SMC complexes can traverse one another in vivo
        </a></td><td>Brandão, H. B., Ren, Z., Karaboja, X., Mirny, L. A., Wang, X.
        </td><td>10.1101/2020.10.26.356329</td><td>2020-10-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.10.26.355164">Deep learning enables fast and dense single-molecule localization with high accuracy
        </a></td><td>Speiser, A., Müller, L.-R., Matti, U., Obara, C. J., Legant, W. R., Kreshuk, A., Macke, J. H., Ries, J., Turaga, S. C.
        </td><td>10.1101/2020.10.26.355164</td><td>2020-10-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.10.15.340356">MCM complexes are barriers that restrict cohesin-mediated loop extrusion
        </a></td><td>Dequeker, B. J. H., Brandao, H. B., Scherr, M. J., Gassler, J., Powell, S., Gaspar, I., Flyamer, I. M., Tang, W., Stocsits, R., Davidson, I. F., Peters, J.-M., Duderstadt, K. E., Mirny, L. A., Tachibana, K.
        </td><td>10.1101/2020.10.15.340356</td><td>2020-10-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.10.09.331900">HP1α is a chromatin crosslinker that controls nuclear and mitotic chromosome mechanics
        </a></td><td>Strom, A. R., Biggs, R. J., Banigan, E. J., Wang, X., Chiu, K., Herman, C., Collado, J., Yue, F., Ritland Politz, J. C., Tait, L. J., Scalzo, D., Telling, A., Groudine, M., Brangwynne, C. P., Marko, J., Stephens, A. D.
        </td><td>10.1101/2020.10.09.331900</td><td>2020-10-09</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.10.06.328948">Automated CUT&Tag profiling of chromatin heterogeneity in mixed-lineage leukemia
        </a></td><td>Janssens, D., Meers, M. P., Wu, S. J., Babaeva, E., Meshinchi, S., Sarthy, J. F., Ahmad, K., Henikoff, S.
        </td><td>10.1101/2020.10.06.328948</td><td>2020-10-08</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2019.12.20.885533">Coarse-Grained Modelling of DNA Plectoneme Pinning in the Presence of Base-Pair Mismatches
        </a></td><td>Desai, P. R., Brahmachari, S., Marko, J. F., Das, S., Neuman, K. C.
        </td><td>10.1101/2019.12.20.885533</td><td>2020-10-04</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.09.29.315671">PYMEVisualize: an open-source tool for exploring 3D super-resolution data
        </a></td><td>Marin, Z., Graff, M., Barentine, A. E. S., Soeller, C., Chung, K. K. H., Fuentes, L. A., Baddeley, D.
        </td><td>10.1101/2020.09.29.315671</td><td>2020-09-30</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.09.15.297994">DMA-tudor interaction modules control the specificity of in vivo condensates
        </a></td><td>Courchaine, E. M., Barentine, A. E. S., Straube, K., Bewersdorf, J., Neugebauer, K. M.
        </td><td>10.1101/2020.09.15.297994</td><td>2020-09-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.08.31.276402">Micromanipulation of prophase I chromosomes from mouse spermatocytes reveals high stiffness and gel-like chromatin organization
        </a></td><td>Marko, J. F., Qiao, H., Liu, N., Biggs, R. J., Peng, Y.
        </td><td>10.1101/2020.08.31.276402</td><td>2020-09-01</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.08.30.274035">Homeotic Regulation of Olfactory Receptor Choice via NFI-dependent Heterochromatic Silencing and Genomic Compartmentalization
        </a></td><td>Bashkirova, E., Monahan, K., Campbell, C. E., Osinski, J. M., Tan, L., Schieren, I., Barnea, G., Xie, S., Gronostajski, R. M., Lomvardas, S.
        </td><td>10.1101/2020.08.30.274035</td><td>2020-08-30</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.08.28.263020">A dual mechanism of enhancer activation by FOXA pioneer factors induces endodermal organ fates
        </a></td><td>Geusz, R. J., Wang, A., Lam, D. K., Vinckier, N. K., Alysandratos, K.-D., Roberts, D. A., Wang, J., Kefalopoulou, S., Qiu, Y., Chiou, J., Gaulton, K. J., Ren, B., Kotton, D. N., Sander, M.
        </td><td>10.1101/2020.08.28.263020</td><td>2020-08-29</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.08.25.267435">RNA promotes the formation of spatial compartments in the nucleus
        </a></td><td>Quinodoz, S. A., Bhat, P., Ollikainen, N., Jachowicz, J. W., Banerjee, A. K., Chovanec, P., Blanco, M. R., Chow, A., Markaki, Y., Plath, K., Guttman, M.
        </td><td>10.1101/2020.08.25.267435</td><td>2020-08-25</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.08.19.254706">Epigenetic memory as a time integral over prior history of Polycomb phase separation
        </a></td><td>Eeftens, J. M., Kapoor, M., Brangwynne, C. P.
        </td><td>10.1101/2020.08.19.254706</td><td>2020-08-19</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.08.11.242081">A single-cell method to map higher-order 3D genome organization in thousands of individual cells reveals structural heterogeneity in mouse ES cells
        </a></td><td>Arrastia, M. V., Jachowicz, J. W., Ollikainen, N., Curtis, M. S., Lai, C., Quinodoz, S., Selck, D. A., Guttman, M., Ismagilov, R. F.
        </td><td>10.1101/2020.08.11.242081</td><td>2020-08-12</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.07.31.231027">Transcript assembly improves expression quantification of transposable elements in single cell RNA-seq data
        </a></td><td>Shao, W., Wang, T.
        </td><td>10.1101/2020.07.31.231027</td><td>2020-07-31</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.07.31.230284">Multiplex-GAM: genome-wide identification of chromatin contacts yields insights not captured by Hi-C
        </a></td><td>Beagrie, R. A., Thieme, C. J., Annunziatella, C., Baugher, C., Zhang, Y., Schueler, M., Kramer, D. C., Chiariello, A. M., Bianco, S., Kukalev, A., Li, Y., Kempfer, R., Scialdone, A., Welch, L. A., Nicodemi, M., Pombo, A.
        </td><td>10.1101/2020.07.31.230284</td><td>2020-07-31</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.07.29.227199">Single-Molecule Micromanipulation Studies Of Methylated DNA
        </a></td><td>Zaichuk, T., Marko, J.
        </td><td>10.1101/2020.07.29.227199</td><td>2020-07-30</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.07.16.206789">PartSeg, a Tool for Quantitative Feature Extraction From 3D Microscopy Images for Dummies</a></td><td>Bokota, G., Sroka, J., Basu, S., Das, N., Trzaskoma, P., Yushkevich, Y., Grabowska, A., Magalska, A., Plewczynski, D.
        </td><td>10.1101/2020.07.16.206789</td><td>2020-07-17</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.07.15.204719">Simultaneous Epigenetic Perturbation and Genome Imaging Reveal Distinct Roles of H3K9me3 in Chromatin Architecture and Transcription</a></td><td>Feng, Y., Wang, Y., Wang, X., He, X., Yang, C., Naseri, A., Pederson, T., Zheng, J., Zhang, S., Xiao, X., Xie, W., Ma, H.
        </td><td>10.1101/2020.07.15.204719</td><td>2020-07-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.07.07.192526">CTCF Mediates Dosage and Sequence-context-dependent Transcriptional Insulation through Formation of Local Chromatin Domains</a></td><td>Huang, H., Zhu, Q., Jussila, A., Han, Y., Bintu, B., Kern, C., Conte, M., Zhang, Y., Bianco, S., Chiariello, A., Yu, M., Hu, R., Juric, I., Hu, M., Nicodemi, M., Zhuang, X., Ren, B.
        </td><td>10.1101/2020.07.07.192526</td><td>2020-07-08</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.06.13.149195">Unsupervised manifold alignment for single-cell multi-omics data</a></td><td>Singh, R., Demetci, P., Bonora, G., Ramani, V., Lee, C., Fang, H., Duan, Z., Deng, X., Shendure, J., Disteche, C., Noble, W. S.
        </td><td>10.1101/2020.06.13.149195</td><td>2020-06-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.06.11.147207">Responsiveness to perturbations is a hallmark of transcription factors that maintain cell identity</a></td><td>Mellis, I. A., Edelstein, H. I., Truitt, R., Beck, L. E., Symmons, O., Goyal, Y., Dunagin, M. C., Linares Saldana, R. A., Shah, P. P., Yang, W., Jain, R., Raj, A.
        </td><td>10.1101/2020.06.11.147207</td><td>2020-06-12</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.06.03.128561">Chromatin Mechanics Dictates Subdiffusion and Coarsening Dynamics of Embedded Condensates</a></td><td>Lee, D. S. W., Wingreen, N. S., Brangwynne, C. P.
        </td><td>10.1101/2020.06.03.128561</td><td>2020-06-04</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.05.26.116962">Gene regulation gravitates towards either addition or multiplication when combining the effects of two signals</a></td><td>Sanford, E. M., Emert, B. L., Cote, A., Raj, A.
        </td><td>10.1101/2020.05.26.116962</td><td>2020-05-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.05.16.099275">Chromatin folding variability across single-cells results from state degeneracy in phase-separation</a></td><td>Conte, M., Fiorillo, L., Bianco, S., Chiariello, A. M., Esposito, A., Nicodemi, M.
        </td><td>10.1101/2020.05.16.099275</td><td>2020-05-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.05.05.078436">Impact of chromatin context on Cas9-induced DNA double-strand break repair pathway balance</a></td><td>Schep, R., Brinkman, E. K., Leemans, C., Vergara, X., Morris, B., van Schaik, T., Manzo, S. G., Peric Hupkes, D., van den Berg, J., Beijersbergen, R., Medema, R. H., van Steensel, B.
        </td><td>10.1101/2020.05.05.078436</td><td>2020-05-05</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.04.28.066787">Gromov-Wasserstein optimal transport to align single-cell multi-omics data</a></td><td>Demetci, P., Santorella, R., Sandstede, B., Noble, W. S., Singh, R.
        </td><td>10.1101/2020.04.28.066787</td><td>2020-04-29</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.04.27.060061">The qBED track: a novel genome browser visualization for point processes</a></td><td>Moudgil, A., Li, D., Hsu, S., Purushotham, D., Wang, T., Mitra, R. D.
        </td><td>10.1101/2020.04.27.060061</td><td>2020-04-29</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.04.27.060145">HiTea: a computational pipeline to identify non-reference transposable element insertions in Hi-C data</a></td><td>Jain, D., Chu, C., Alver, B. H., Lee, S., Lee, E. A., Park, P. J.
        </td><td>10.1101/2020.04.27.060145</td><td>2020-04-28</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.04.24.059915">Comparison of the Hi-C, GAM and SPRITE methods by use of polymer models of chromatin</a></td><td>Fiorillo, L., Musella, F., Kempfer, R., Chiariello, A. M., Bianco, S., Kukalev, A., Irastorza-Azcarate, I., Esposito, A., Conte, M., Prisco, A., Pombo, A., Nicodemi, M.
        </td><td>10.1101/2020.04.24.059915</td><td>2020-04-25</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.04.10.036442">Single-molecule diffusometry reveals no catalysis-induced diffusion enhancement of alkaline phosphatase as proposed by FCS experiments</a></td><td>Chen, Z., Shaw, A., Wilson, H., Woringer, M., Darzacq, X., Marqusee, S., Wang, Q., Bustamante, C.
        </td><td>10.1101/2020.04.10.036442</td><td>2020-04-11</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.04.06.028092">The spatial distributions of pre-mRNAs suggest post-transcriptional splicing of specific introns within endogenous genes</a></td><td>Cote, A. J., Cote, C. J., Bayatpour, S., Drexler, H. L., Alexander, K. A., Chen, F., Wassie, A. T., Boyden, E. S., Berger, S., Churchman, L. S., Raj, A.
        </td><td>10.1101/2020.04.06.028092</td><td>2020-04-07</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.03.21.001693">CTCF Promotes Long-range Enhancer-promoter Interactions and Lineage-specific Gene Expression in Mammalian Cells</a></td><td>Kubo N., Ishii, H., Xiong, X., Bianco, S., Meitinger, F., Hu, R., Hocker, J.D., Conte, M., Gorkin, D., Yu, M., Li, B., Dixon, J.R., Hu, M., Nicodemi, M., Zhao, H., Ren, B.
        </td><td>10.1101/2020.03.21.001693</td><td>2020-03-23</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.03.21.001917">Exploring Chromosomal Structural Heterogeneity AcrossMultiple Cell Lines</a></td><td>Cheng, R. R., Contessoto, V., Aiden, E. L., Wolynes, P. G., Di Pierro, M., Onuchic, J. N.
        </td><td>10.1101/2020.03.21.001917</td><td>2020-03-22</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.03.18.997163">Accurate 4Pi single-molecule localization using an experimental PSF model</a></td><td>Li, Y., Buglakova, E., Zhang, Y., Thevathasan, J. V., Bewersdorf, J., Ries, J.
        </td><td>10.1101/2020.03.18.997163</td><td>2020-03-19</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.03.18.997494">EMU: reconfigurable graphical user interfaces for Micro-Manager</a></td><td>Deschamps, J., Ries, J.
        </td><td>10.1101/2020.03.18.997494</td><td>2020-03-19</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.03.18.996660">Retrospective identification of rare cell populations underlying drug resistance connects molecular variability with cell fate</a></td><td>Emert, B., Cote, C., Torre, E. A., Dardani, I. P., Jiang, C., Jain, N., Shaffer, S. M., Raj, A.</td><td>10.1101/2020.03.18.996660</td><td>2020-03-19</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.03.10.986208">Detecting chromatin interactions along and between sister chromatids with SisterC</a></td><td>Oomen, M. E., Hedger, A. K., Watts, J. K., Dekker, J.</td><td>10.1101/2020.03.10.986208</td><td>2020-03-11</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.03.09.982967">SPIN reveals genome-wide landscape of nuclear compartmentalization</a></td><td>Wang, Y., Zhang, Y., Zhang, R., van Schaik, T., Zhang, L., Sasaki, T., Hupkes, D. P., Chen, Y., Gilbert, D. M., van Steensel, B., Belmont, A. S., Ma, J.</td><td>10.1101/2020.03.09.982967</td><td>2020-03-10</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.03.05.979070">Chromatin Topology Reorganization and Transcription Repression by PML/RARα in Acute Promyeloid Leukemia</a></td><td>Ruan, Y.</td><td>10.1101/2020.03.05.979070</td><td>2020-03-06</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.03.03.975680">Regulation of gene expression by repression condensates during development</a></td><td>Treen, N., Shimobayashi, S. F., Eeftens, J., Brangwynne, C. P., Levine, M.</td><td>10.1101/2020.03.03.975680</td><td>2020-03-04</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.02.29.971317">An Improved 4’-Aminomethyltroxsalen-Based DNA Crosslinker for Biotinylation of DNA</a></td><td>Wielenberg, K., Wang, M., Yang, M., Ozer, A., Lis, J. T., Lin, H.</td><td>10.1101/2020.02.29.971317</td><td>2020-02-29</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.02.26.966697">Subtype-specific epigenomic landscape and 3D genome structure in bladder cancer</a></td><td>Iyyanki, T. S., Zhang, B., Jin, Q., Yang, H., Liu, T., Wang, X., Song, F., Luan, Y., Yamashita, H., Wang, L., Warrick, J., Raman, J., Meeks, J., DeGraff, D. J., Yue, F.</td><td>10.1101/2020.02.26.966697</td><td>2020-02-29</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.02.26.966697">Fluorophore-labelled RNA aptamers to common protein tags as super-resolution imaging reagents.</a></td><td>Wang, J., Singh, A., Ozer, A., Zipfel, W. R.</td><td>10.1101/2020.02.27.968578</td><td>2020-02-28</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.02.23.961672">Non-coding germline GATA3 variants alter chromatin topology and contribute to pathogenesis of acute lymphoblastic leukemia</a></td><td>Yang, H., Zhang, H., Luan, Y., Liu, T., Roberts, K. G., Qian, M., Zhang, B., Yang, W., Perez-Andreu, V., Xu, J., Iyyanki, S., Kuang, D., Reshmi, S. C., Gastier-Foster, J., Smith, C., Pui, C., Evans, W. E., Hunger, S. P., Platanias, L. C., Relling, M. V., Mullighan, C. G., Loh, M. L., Yue, F., Yang, J. J.</td><td>10.1101/2020.02.23.961672</td><td>2020-02-25</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.02.18.954610">CRISPR-based Live Imaging of Epigenetic Modification-Mediated Genome Reorganization</a></td><td>Feng, Y., Wang, Y., Yang, C., Naseri, A., Pederson, T., Zheng, J., Xiao, X., Zhang, S., Xie, W., Ma, H.</td><td>10.1101/2020.02.18.954610</td><td>2020-02-19</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.02.10.941047">Single-Cell Analysis of the 3D Topologies of Genomic Loci Using Genome Architecture Mapping</a></td><td>Welch, L. R., Baugher, C., Zhang, Y., Davis, T., Marzluff, W. F., Welch, J. D., Pombo, A.</td><td>10.1101/2020.02.10.941047</td><td>2020-02-13</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.02.07.939124">Exploring the coronavirus epidemic using the new WashU Virus Genome Browser</a></td><td>Flynn, J., Purushotham, D., Choudhary, M. N., Zhuo, X., Fan, C., Matt, G., Li, D., Wang, T.</td><td>10.1101/2020.02.07.939124</td><td>2020-02-11</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.02.05.935106">Ultrastructural Visualization of 3D Chromatin Folding Using Serial Block-Face Scanning Electron Microscopy and In Situ Hybridization (3D-EMISH)</a></td><td>Trzaskoma, P., Ruszczycki, B., Lee, B., Pels, K. K., Krawczyk, K., Bokota, G., Szczepankiewicz, A. A., Aaron, J., Walczak, A., Sliwinska, M., Magalska, A., Kadloff, M., Wolny, A., Parteka, Z., Arabasz, S., Kiss- Arabasz, M., Plewczynski, D., Ruan, Y., Wilczynski, G. M.</td><td>10.1101/2020.02.05.935106</td><td>2020-02-06</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.01.30.927061">GAMIBHEAR: whole-genome haplotype reconstruction from Genome Architecture Mapping data</a></td><td>Markowski, J., Kempfer, R., Kukalev, A., Irastorza-Azcarate, I., Loof, G., Pombo, A., Schwarz, R. F.</td><td>10.1101/2020.01.30.927061</td><td>2020-01-31</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.01.22.916171">Probing multi-way chromatin interaction with hypergraph representation learning</a></td><td>Zhang, R., Ma, J.</td><td>10.1101/2020.01.22.916171</td><td>2020-01-23</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2020.01.16.909333">Strong intracellular signal inactivation produces sharper and more robust signaling from cell membrane to nucleus</a></td><td>Ma, J., Do, M., Le Gros, M. A., Peskin, C. S., Larabell, C. A., Mori, Y., Isaacson, S. A.</td><td>10.1101/2020.01.16.909333</td><td>2020-01-17</td></tr>

        <!--2019--> 
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/379016">Memory sequencing reveals heritable single cell gene expression programs associated with distinct cellular behaviors</a></td><td>Shaffer, S. M., Emert, B. L., Reyes-Hueros, R., Cote, C., Harmange, G., Sizemore, A. E., Gupte, R., Torre, E., Singh, A., Bassett, D. S., Raj, A.</td><td>10.1101/379016</td><td>2019-12-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/2019.12.19.881979">Cell cycle dynamics of lamina associated DNA</a></td><td>van Schaik, T., Vos, M., Peric-Hupkes, D., van Steensel, B.</td><td>10.1101/2019.12.19.881979</td><td>2019-12-19</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/597518">Peax: Interactive Visual Pattern Search in Sequential Data Using Unsupervised Deep Representation Learning</a></td><td>Lekschas, F., Peterson, B., Haehn, D., Ma, E., Gehlenborg, N., Pfister, H.</td><td>10.1101/597518</td><td>2019-12-12</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/867341">Chromosomal-level genome assembly of the scimitar-horned oryx: insights into diversity and demography of a species extinct in the wild</a></td><td>Humble, E., Dobrynin, P., Senn, H., Chuven, J., Scott, A. F., Mohr, D. W., Dudchenko, O., Omer, A. D., Colaric, Z., Lieberman Aiden, E., Wildt, D., Oliagi, S., Tamazian, G., Pukazhenthi, B., Ogden, R., Koepfli, K.-P.</td><td>10.1101/867341</td><td>2019-12-08</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/864199">Efficient Homology-directed Repair with Circular ssDNA Donors</a></td><td>Iyer, S., Mir, A., Ibraheim, R., Lee, J., VegaBadillo, J., Roscoe, B., Zhu, L. J., Liu, P., Luk, K., Mintzer, E., de Brito, J. S., Zamore, P., Sontheimer, E. J., Wolfe, S.</td><td>10.1101/864199</td><td>2019-12-05</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/855254">High-Resolution 3D Fluorescent Imaging of Intact Tissues</a></td><td>El-Nachef, D., Martinson, A. M., Yang, X., Murry, C. E., MacLellan, W. R.</td><td>10.1101/855254</td><td>2019-11-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/534800">Capturing cell type-specific chromatin structural patterns by applying topic modeling to single-cell Hi-C data</a></td><td>Kim, H.-J., Yardimici, G. G., Bonora, G., Ramani, V., Liu, J., Qiu, R., Lee, C., Hesson, J., Ware, C. B., Shendure, J., Duan, Z., Noble, W. S.</td><td>10.1101/534800</td><td>2019-11-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/638809">Genetic screening for single-cell variability modulators driving therapy resistance</a></td><td>Torre, E. A., Arai, E., Bayatpour, S., Beck, L. E., Emert, B. L., Shaffer, S. M., Mellis, I. A., Fane, M., Alicea, G., Budinich, K. A., Weeraratna, A., Shi, J., Raj, A.</td><td>10.1101/638809</td><td>2019-11-10</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://arxiv.org/abs/1911.02613">Hyper-SAGNN: a self-attention based graph neural network for hypergraphs</a></td><td>Zhang, R., Zou, Y., Ma, J.</td><td>-</td><td>2019-11-06</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/829606">MS2-TRIBE evaluates protein-RNA interactions and nuclear organization of transcription by RNA editing</a></td><td>Biswas, J., Rahman, R., Gupta, V., Rosbash, M., Singer, R.</td><td>10.1101/829606</td><td>2019-11-04</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/824433">TSA-Seq 2.0 reveals both conserved and variable chromosomal distances to nuclear speckles</a></td><td>Zhang, L., Zhang, Y., Chen, Y., Gholamalamdari, O., Ma, J., Belmont, A. S.</td><td>10.1101/824433</td><td>2019-10-30</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/616102">Chromosome disentanglement driven via optimal compaction of loop-extruded brush structures</a></td><td>Brahmachari, S., Marko, J. F.</td><td>10.1101/616102</td><td>2019-10-25</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://arxiv.org/abs/1910.11370">Minimum Information guidelines for fluorescence microscopy: increasing the value, quality, and fidelity of image data</a></td><td>Huisman, M., Hammer, M., Rigano, A., Farzam, F., Gopinathan, R., Smith, C., Grunwald, D., Strambio-De-Castillia, C.</td><td>-</td><td>2019-10-24</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/816611">Cohesin depleted cells pass through mitosis and reconstitute a functional nuclear architecture</a></td><td>Cremer, M., Brandstetter, K., Maiser, A., Rao, S. S., Schmid, V., Mitra, N., Mamberti, S., Klein, K. N., Gilbert, D. M., Leonhardt, H., Cardoso, M. C., Lieberman Aiden, E., Harz, H., Cremer, T.</td><td>10.1101/816611</td><td>2019-10-24</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/815340">Chromosome organization by one-sided and two-sided loop extrusion</a></td><td>Banigan, E. J., van den Berg, A. A., Brandao, H. B., Marko, J. F., Mirny, L. A.</td><td>10.1101/815340</td><td>2019-10-22</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/809210">Composition dependent phase separation underlies directional flux through the nucleolus</a></td><td>Riback, J. A., Zhu, L., Ferrolino, M. C., Tolbert, M., Mitrea, D. M., Sanders, D. W., Wei, M.-T., Kriwacki, R. W., Brangwynne, C. P.</td><td>10.1101/809210</td><td>2019-10-22</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/796482">A cost-efficient open source laser engine for microscopy</a></td><td>Schroeder, D., Deschamps, J., Dasgupta, A., Matti, U., Ries, J.</td><td>10.1101/796482</td><td>2019-10-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/485342">Co-opted transposons help perpetuate conserved higher-order chromosomal structures</a></td><td>Choudhary, M. N., Friedman, R. Z., Wang, J. T., Jang, H. S., Zhuo, X., Wang, T.</td><td>10.1101/485342</td><td>2019-10-14</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/784835">Architectural RNA is required for heterochromatin organization</a></td><td>Thakur, J., Fang, H., Llagas, T., Disteche, C. M., Henikoff, S.</td><td>10.1101/784835</td><td>2019-09-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/779058">ESCO1 and CTCF enable formation of long chromatin loops by protecting cohesinSTAG1 from WAPL</a></td><td>Wutz, G., St. Hilaire, B. T. G., Ladurner, R., Stocsits, R., Nagasaka, K., Pignard, B., Sanborn, A., Tang, W., Varnai, C., Ivanov, M., Schoenfelder, S., van der Lelij, P., Huang, X., Duernberger, G., Roitinger, E., Mechtler, K., Davidson, I. F., Fraser, P., Aiden, E. L., Peters, J. M.</td><td>10.1101/779058</td><td>2019-09-23</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/772269">Direct, sensitive and specific detection of individual single- or double-strand DNA breaks by fluorescence microscopy</a></td><td>Kordon, M., Zarebski, M., Solarczyk, K., Ma, H., Pederson, T., Dobrucki, J. W.</td><td>10.1101/772269</td><td>2019-09-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/762872">Chromatin is frequently unknotted at the megabase scale</a></td><td>Goundaroulis, D., Aiden, E. L., Stasiak, A.</td><td>10.1101/762872</td><td>2019-09-09</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/755629">High Resolution Repli-Seq defines the temporal choreography of initiation, elongation and termination of replication in mammalian cells</a></td><td>Zhao, P.A., Sasaki, T., Gilbert, D. M.</td><td>10.1101/755629</td><td>2019-09-05</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/749754">NuSeT: A Deep Learning Tool for Reliably Separating and Analyzing Crowded Cells</a></td><td>Yang, L., Ghosh, R. P., Franklin, J. M., You, C., Liphardt, J.</td><td>10.1101/749754</td><td>2019-08-28</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/111674">PBAF regulates compartmentalization of actively transcribing chromatin hubs</a></td><td>Kenworthy, C. A., Liou, S.-H., Chandris, P., Wong, V., Dziuba, P., Lavis, L. D., Liu, W.-L., Singer, R. H., Coleman, R. A.</td><td>10.1101/111674</td><td>2019-08-22</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/737387">Nucleated transcriptional condensates amplify gene expression</a></td><td>Wei, M.-T., Chang, Y.-C., Shimobayashi, S. F., Shin, Y., Brangwynne, C. P.</td><td>10.1101/737387</td><td>2019-08-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/739698">A supervised learning framework for chromatin loop detection in genome-wide contact maps</a></td><td>Salameh, T. J., Wang, X., Song, F., Zhang, B., Wright, S. M., Khunsriraksakul, C., Yue, F.</td><td>10.1101/739698</td><td>2019-08-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/740480">Distinct features of nucleolus-associated domains in mouse embryonic stem cells</a></td><td>Bizhanova, A., Yan, A., Yu, J., Zhu, L. J., Kaufman, P. D.</td><td>10.1101/740480</td><td>2019-08-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/734483">Temporal-Spatial Visualization of Endogenous Chromosome Rearrangements in Living Cells</a></td><td>Wang, H., Nakamura, M., Zhao, D., Nguyen, C. M., Yu, C., Lo, A., Daley, T., La Russa, M., Liu, Y., Qi, L. S.</td><td>10.1101/734483</td><td>2019-08-14</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/727768">Measuring significant changes in chromatin conformation with ACCOST</a></td><td>Cook, K. B., Le Roch, K. G., Vert, J.-P., Noble, W.</td><td>10.1101/727768</td><td>2019-08-06</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/301036">Pattern-Driven Navigation in 2D Multiscale Visualizations with Scalable Insets</a></td><td>Lekschas, F., Behrisch, M., Bach, B., Kerpedjiev, P., Gehlenborg, N., Pfister, H.</td><td>10.1101/301036</td><td>2019-07-25</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/712950">Dynamic changes in RNA-chromatin interactome promote endothelial dysfunction</a></td><td>Calandrelli, R., Xu, L., Luo, Y., Wu, W., Fan, X., Nguyen, T., Chen, C., Sriram, K., Natarajan, R., Chen, Z., Zhong, S.</td><td>10.1101/712950</td><td>2019-07-24</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/708024">Versatile multi-transgene expression using improved BAC TG-EMBED toolkit, novel BAC episomes, and BAC-MAGIC</a></td><td>Zhao, B., Chaturvedi, P., Zimmerman, D. L., Belmont, A. S.</td><td>10.1101/708024</td><td>2019-07-23</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/709683">pyBedGraph: a Python package for fast operations on 1-dimensional genomic signal tracks</a></td><td>Zhang, H. B., Kim, M., Chuang, J. H., Ruan, Y.</td><td>10.1101/709683</td><td>2019-07-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/704247">Gene networks with transcriptional bursting recapitulate rare transient coordinated expression states in cancer</a></td><td>Schuh, L., Saint-Antoine, M., Sanford, E., Emert, B., Singh, A., Marr, C., Goyal, Y., Raj, A.</td><td>10.1101/704247</td><td>2019-07-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/704957">Compartment-dependent chromatin interaction dynamics revealed by liquid chromatin Hi-C</a></td><td>Belaghzal, H., Borrman, T., Stephens, A. D., Lafontaine, D. L., Venev, S. V., Marko, J. F., Weng, Z., Dekker, J.</td><td>10.1101/704957</td><td>2019-07-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/687236">Trans- and cis-acting effects of the lncRNA Firre on epigenetic and structural features of the inactive X chromosome</a></td><td>FANG, H., Bonora, G., Lewandowski, J., Thakur, J., Filippova, G. N., Henikoff, S., Shendure, J., Duan, Z., Rinn, J., Deng, X., Noble, W. S., Disteche, C. M.</td><td>10.1101/687236</td><td>2019-06-30</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/683557">Chemogenetic control of nanobodies</a></td><td>Farrants, H., Tarnawski, M., Müller, T. G., Otsuka, S., Hiblot, J., Koch, B., Kueblbeck, M., Kräusslich, H.-G., Ellenberg, J., Johnsson, K.</td><td>10.1101/683557</td><td>2019-06-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/685255">Local rewiring of genome - nuclear lamina interactions by transcription.</a></td><td>Brueckner, L., Zhao, P. A., van Schaik, T., Leemans, C., Sima, J., Peric-Hupkes, D., Gilbert, D. M., van Steensel, B.</td><td>10.1101/685255</td><td>2019-06-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/678474">A chromosome folding intermediate at the condensin-to-cohesin transition during telophase</a></td><td>Abramo, K., Valton, A.-L., Venev, S. V., Ozadam, H., Fox, A. N., Dekker, J.</td><td>10.1101/678474</td><td>2019-06-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/665232">Multiplex chromatin interaction analysis by signal processing and statistical algorithms</a></td><td>Kim, M., Zheng, M., Tian, S. Z., Capurso, D., Lee, B., Chuang, J. H., Ruan, Y.</td><td>10.1101/665232</td><td>2019-06-10</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/657080">HiNT: a computational method for detecting copy number variations and translocations from Hi-C data</a></td><td>Wang, S., Lee, S., Chu, C., Jain, D., Nelson, G., Walsh, J. M., Alver, B. H., Park, P. J.</td><td>10.1101/657080</td><td>2019-06-03</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/325373">DNA-segment-capture model for loop extrusion by structural maintenance of chromosome (SMC) protein complexes</a></td><td>Marko, J. F., De Los Rios, P., Barducci, A., Gruber, S.</td><td>10.1101/325373</td><td>2019-05-23</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/644294">Inferring diploid 3D chromatin structures from Hi-C data</a></td><td>Cauer, A., Yardimci, G. G., Vert, J.-P., Varoquaux, N., Noble, W.</td><td>10.1101/644294</td><td>2019-05-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/644310">Jointly embedding multiple single-cell omics measurements</a></td><td>Liu, J., Huang, Y., Singh, R., Vert, J.-P., Noble, W.</td><td>10.1101/644310</td><td>2019-05-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/642322">Spring Model - chromatin modeling tool based on OpenMM</a></td><td>Kadlof, M., Rozycka, J., Plewczynski, D.</td><td>10.1101/642322</td><td>2019-05-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/639922">Ultrastructural details of mammalian chromosome architecture</a></td><td>Krietenstein, N., Abraham, S., Venev, S., Abdennur, N., Gibcus, J., Hsieh, T.-H., Parsi, K. M., Yang, L., Maehr, R., Mirny, L., Dekker, J., Rando, O.</td><td>10.1101/639922</td><td>2019-05-17</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/638775">Resolving the 3D landscape of transcription-linked mammalian chromatin folding</a></td><td>Hsieh, T.-H. S., Slobodyanyuk, E., Hansen, A. S., Cattoglio, C., Rando, O. J., Tjian, R., Darzacq, X.</td><td>10.1101/638775</td><td>2019-05-17</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/569129">Improved CUT&RUN chromatin profiling and analysis tools</a></td><td>Meers, M. P., Bryson, T. D., Henikoff, S.</td><td>10.1101/569129</td><td>2019-05-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/539049">Concerted localization resets precede YAP-dependent transcription</a></td><td>Franklin, J. M., Ghosh, R. P., Shi, Q., Liphardt, J. T.</td><td>10.1101/539049</td><td>2019-05-06</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/628214">Spatial genome re-organization between fetal and adult hematopoietic stem cells</a></td><td>Chen, C., Yu, W., Tober, J., Gao, P., He, B., Lee, K., Trieu, T., Blobel, G., Speck, N., Tan, K.</td><td>10.1101/628214</td><td>2019-05-05</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/626853">Photoactivation of silicon rhodamines via a light-induced protonation</a></td><td>Frei, M. S., Hoess, P., Lampe, M., Nijmeijer, B., Kueblbeck, M., Ellenberg, J., Ries, J., Pitsch, S., Reymond, L., Johnsson, K.</td><td>10.1101/626853</td><td>2019-05-04</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/440974">Tibanna: software for scalable execution of portable pipelines on the cloud</a></td><td>Lee, S., Johnson, J., Vitzthum, C., Kırlı, K., Alver, B. H., Park, P. J.</td><td>10.1101/440974</td><td>2019-04-29</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/621920">Super resolution imaging of a distinct chromatin loop in human lymphoblastoid cells</a></td><td>Ruan, Y., Plewczynski, D., Zhu, J. J., Parteka, Z., Lee, B., Szalaj, P., Wang, P., Jodkowska, K., Arron, J., Chew, T. L.</td><td>10.1101/621920</td><td>2019-04-29</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/619288">FIREcaller: an R package for detecting frequently interacting regions from Hi-C data</a></td><td>Crowley, C., Yang, Y., Qiu, Y., Hu, B., Won, H., Ren, B., Hu, M., Li, Y.</td><td>10.1101/619288</td><td>2019-04-29</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://dx.doi.org/10.17504/protocols.io.2cxgaxn">Mapping RNA-chromatin interactions by sequencing with iMARGI V.2</a></td><td>Wu, W., Yan, Z., Nguyen, T. C., Chen, Z., Chien, S., Zhong, S.</td><td>-</td><td>2019-04-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/613034">ChIA-DropBox: a novel analysis and visualization pipeline for multiplex chromatin interactions</a></td><td>Tian, S. Z., Capurso, D., Kim, M., Lee, B., Zheng, M., Ruan, Y.</td><td>10.1101/613034</td><td>2019-04-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/364976">Multi-scale deep tensor factorization learns a latent representation of the human epigenome</a></td><td>Schreiber, J., Durham, T. J., Bilmes, J., Noble, W. S.</td><td>10.1101/364976</td><td>2019-04-11</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/604355">Re-configuration of Chromatin Structure During the Mitosis-G1 Phase Transition</a></td><td>Zhang, H., Emerson, D., Gilgenast, T., Titus, K., Lan, Y., Huang, P., Zhang, D., Wang, H., Keller, C., Giardine, B., Hardison, R., Phillips-Cremins, J., Blobel, G.</td><td>10.1101/604355</td><td>2019-04-11</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/604280">RNA polymerases as moving barriers to condensin loop extrusion</a></td><td>Brandao, H. B., Wang, X., Paul, P., van den Berg, A., Rudner, D. Z., Mirny, L. A.</td><td>10.1101/604280</td><td>2019-04-10</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/604298">Transcription amplification by nuclear speckle association</a></td><td>Kim, J., Khanna, N., Belmont, A. S.</td><td>10.1101/604298</td><td>2019-04-09</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/592741">Common DNA sequence variation influences 3-dimensional conformation of the human genome</a></td><td>Gorkin, D., Qiu, Y., Hu, M., Fletez-Brant, K., Liu, T., Schmitt, A., Noor, A., Chiou, J., Gaulton, K., Sebat, J., Li, Y., Hansen, K., Ren, B.</td><td>10.1101/592741</td><td>2019-03-30</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/579961">Direct visualization of single nuclear pore complex proteins using genetically-encoded probes for DNA-PAINT</a></td><td>Schlichthaerle, T., Strauss, M. T., Schueder, F., Auer, A., Nijmeijer, B., Kueblbeck, M., Jimenez Sabinina, V., Thevatasan, J. V., Ries, J., Ellenberg, J., Jungmann, R.</td><td>10.1101/579961</td><td>2019-03-23</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/582668">Nuclear pores as versatile reference standards for quantitative superresolution microscopy</a></td><td>Thevathasan, J. V., Kahnwald, M., Cieslinski, K., Hoess, P., Peneti, S. K., Reitberger, M., Heid, D., Kasuba, K. C., Hoerner, S. J., Li, Y., Wu, Y.-L., Mund, M., Matti, U., Pereira, P. M., Henriques, R., Nijmeijer-Winter, B., Kueblbeck, M., Jimenez Sabinina, V., Ellenberg, J., Ries, J.</td><td>10.1101/582668</td><td>2019-03-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/579573">Sci-Hi-C: a single-cell Hi-C method for mapping 3D genome organization in large number of single cells</a></td><td>Ramani, V., Deng, X., Qiu, R., Lee, C., Disteche, C. M., Noble, W. S., Duan, Z., Shendure, J.</td><td>10.1101/579573</td><td>2019-03-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/568915">CUT&Tag for efficient epigenomic profiling of small samples and single cells</a></td><td>Kaya-Okur, H. S., Wu, S. J., Codomo, C. A., Pledger, E. S., Bryson, T. D., Henikoff, J. G., Ahmad, K., Henikoff, S.</td><td>10.1101/568915</td><td>2019-03-06</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/560425">Absolute quantification of cohesin, CTCF and their regulators in human cells</a></td><td>Holzmann, J., Politi, A. Z., Nagasaka, K., Hantsche-Grininger, M., Walther, N., Koch, B., Fuchs, J., Dürnberger, G., Tang, W., Ladurner, R., Stocsits, R. R., Busslinger, G. A., Novak, B., Mechtler, K., Davidson, I. F., Ellenberg, J., Peters, J.-M.</td><td>10.1101/560425</td><td>2019-02-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/562199">EpiAlignment: alignment with both DNA sequence and epigenomic data</a></td><td>Lu, J., Cao, X., Zhong, S.</td><td>10.1101/562199</td><td>2019-02-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/538652">Supervised classification enables rapid annotation of cell atlases</a></td><td>Pliner, H. A., Shendure, J., Trapnell, C.</td><td>10.1101/538652</td><td>2019-02-25</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/370650">Determining cellular CTCF and cohesin abundances to constrain 3D genome models</a></td><td>Cattoglio, C., Pustova, I., Walther, N., Ho, J. J., Hantsche-Grininger, M., Inouye, C. J., Hossain, M. J., Dailey, G. M., Ellenberg, J., Darzacq, X., Tjian, R., Hansen, A. S.</td><td>10.1101/370650</td><td>2019-02-24</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/557660">Cooler: scalable storage for Hi-C data andother genomically-labeled arrays</a></td><td>Abdennur, N., Mirny, L.</td><td>10.1101/557660</td><td>2019-02-22</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/554337">Localization microscopy at doubled precision with patterned illumination</a></td><td>Cnossen, J., Hinsdale, T., Thorsen, R., Schueder, F., Jungmann, R., Smith, C. S., Rieger, B., Stallinga, S.</td><td>10.1101/554337</td><td>2019-02-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/552505">Comparing 3D genome organization in multiple species using Phylo-HMRF</a></td><td>Yang, Y., Zhang, Y., Ren, B., Dixon, J., Ma, J.</td><td>10.1101/552505</td><td>2019-02-19</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/555250">Chromatin compartment dynamics in a haploinsufficient model of cardiac laminopathy</a></td><td>Bertero, A., Fields, P. A., Smith, A. S., Leonard, A., Beussman, K., Sniadecki, N. J., Kim, D.-H., Tse, H.-F., Pabon, L., Shendure, J., Noble, W. S., Murry, C. E.</td><td>10.1101/555250</td><td>2019-02-19</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/448514">Dynamics of gene expression in single root cells of A. thaliana</a></td><td>Jean-Baptiste, K., McFaline-Figueroa, J. L., Alexandre, C. M., Dorrity, M. W., Saunders, L., Bubb, K. L., Trapnell, C., Fields, S., Queitsch, C., Cuperus, J. T.</td><td>10.1101/448514</td><td>2019-02-11</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/542092">MOCHI enables discovery of heterogeneous interactome modules in 3D nucleome</a></td><td>Tian, D., Zhang, R., Zhang, Y., Zhu, X., Ma, J.</td><td>10.1101/542092</td><td>2019-02-07</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/406330">Using three-dimensional regulatory chromatin interactions from adult and fetal cortex to interpret genetic results for psychiatric disorders and cognitive traits</a></td><td>Giusti-Rodriguez, P. M., Sullivan, P. F.</td><td>10.1101/406330</td><td>2019-01-30</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/533273">Completing the ENCODE3 compendium yields accurate imputations across a variety of assays and human biosamples</a></td><td>Schreiber, J., Bilmes, J., Noble, W.</td><td>10.1101/533273</td><td>2019-01-29</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/528877">Preformed Chromatin Topology Assists TranscriptionalRobustness of Shh during Limb Development</a></td><td>Paliou, C., Guckelberger, P., Schöpflin, R., Heinrich, V., Esposito, A., Chiariello, A. M. M., Bianco, S., Annunziatella, C., Helmuth, J., Haas, S., Jerkovic, I., Brieske, N., Wittler, L., Timmermann, B., Nicodemi, M., Vingron, M., Mundlos, S., Andrey, G.</td><td>10.1101/528877</td><td>2019-01-23</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/423541">Effects of altering histone post-translational modifications on mitotic chromosome structure and mechanics</a></td><td>Biggs, R., Stephens, A. D., Liu, P., Marko, J. F.</td><td>10.1101/423541</td><td>2019-01-04</td></tr>

        <!--2018--> 
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/506683">ChIA-PIPE: A fully automated pipeline for ChIA-PET data analysis and visualization</a></td><td>Capurso, D., Wang, J., Tian, S. Z., Cai, L., Namburi, S., Lee, B., Tjong, H., Tang, Z., Wang, P., Wei, C.-L., Ruan, Y., Li, S.</td><td>10.1101/506683</td><td>2018-12-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/505503">Revealing Hi-C subcompartments by imputing high-resolution inter-chromosomal chromatin interactions</a></td><td>Xiong, K., Ma, J.</td><td>10.1101/505503</td><td>2018-12-23</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/495432">An RNA-binding region regulates CTCF clustering and chromatin looping</a></td><td>Hansen, A. S., Hsieh, T.-H. S., Cattoglio, C., Pustova, I., Darzacq, X., Tjian, R.</td><td>10.1101/495432</td><td>2018-12-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/495457">Guided nuclear exploration increases CTCF target search efficiency</a></td><td>Hansen, A. S., Amitai, A., Cattoglio, C., Tjian, R., Darzacq, X.</td><td>10.1101/495457</td><td>2018-12-13</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/486415">Identification of new transmembrane proteins concentrated at the nuclear envelope using organellar proteomics of mesenchymal cells</a></td><td>Cheng, L.-C., Baboo, S., Lindsay, C., Brusman, L. E., Martinez-Bartolome, S., Tapia, O., Zhang, X., Yates, J. R., Gerace, L.</td><td>10.1101/486415</td><td>2018-12-03</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/484568">Two Contrasting Classes of Nucleolus-Associated Domains in Mouse Fibroblast Heterochromatin</a></td><td>Vertii, A., Ou, J., Yu, J., Yan, A., Pages, H., Liu, H., Zhu, L. J., Kaufman, P. D.</td><td>10.1101/484568</td><td>2018-12-03</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/476424">Limits of chromosome compaction by loop-extruding motors</a></td><td>Banigan, E. J., Mirny, L. A.</td><td>10.1101/476424</td><td>2018-11-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/472019">Genome-wide co-localization of RNA-DNA interactions and fusion RNA pairs</a></td><td>Yan, Z., Huang, N., Wu, W., Chen, W., Jiang, Y., Chen, J., Huang, X., Wen, X., Xu, J., Jin, Q., Zhang, K., Chen, Z., Chien, S., Zhong, S.</td><td>10.1101/472019</td><td>2018-11-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/473553">Hi-C yields chromosome-length scaffolds for a legume genome, Trifolium subterraneum</a></td><td>Dudchenko, O., Pham, M., Lui, C., Batra, S. S., Hoeger, M., Nyquist, S. K., Durand, N. C., Shamim, M. S., Machol, I., Erskine, W., Aiden, E. L., Kaur, P.</td><td>10.1101/473553</td><td>2018-11-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/470963">Simultaneous profiling of DNA methylation and chromatin architecture in mixed populations and in single cells</a></td><td>Li, G., Liu, Y., Zhang, Y., Fang, R., Kellis, M., Ren, B.</td><td>10.1101/470963</td><td>2018-11-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/464081">Promoter-intrinsic and local chromatin features determine gene repression in lamina-associated domains</a></td><td>Leemans, C., van der Zwalm, M., Brueckner, L., Comoglio, F., van Schaik, T., Pagie, L., van Arensbergen, J., van Steensel, B.</td><td>10.1101/464081</td><td>2018-11-06</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/450262">Satb1 integrates DNA sequence, shape, motif density and torsional stress to differentially bind targets in nucleosome-dense regions.</a></td><td>GHOSH, R. P., SHI, Q., YANG, L., REDDICK, M. P., NIKITINA, T., ZHURKIN, V. B., FORDYCE, P., STASEVICH, T. J., CHANG, H. Y., GREENLEAF, W. J., LIPHARDT, J. T.</td><td>10.1101/450262</td><td>2018-10-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/443887">Highly Structured Homolog Pairing Reflects Functional Organization of the Drosophila Genome</a></td><td>AlHaj Abed, J., Erceg, J., Goloborodko, A., Nguyen, S. C., McCole, R. B., Saylor, W., Fudenberg, G., Lajoie, B. R., Dekker, J., Mirny, L. A., Wu, T.</td><td>10.1101/443887</td><td>2018-10-17</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/443028">The genome-wide, multi-layered architecture of chromosome pairing in early Drosophila embryos</a></td><td>Erceg, J., AlHaj Abed, J., Goloborodko, A., Lajoie, B. R., Fudenberg, G., Abdennur, N., Imakaev, M., McCole, R. B., Nguyen, S. C., Saylor, W., Joyce, E. F., Senaratne, T. N., Hannan, M. A., Nir, G., Dekker, J., Mirny, L. A., Wu, C.-t.</td><td>10.1101/443028</td><td>2018-10-17</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/441444">Antigen receptor locus dynamics is orchestrated near the sol-gel phase transition to enforce stepwise VDJ gene rearrangement</a></td><td>Khanna, N., Zhang, J., Dudko, O., Murre, C.</td><td>10.1101/441444</td><td>2018-10-12</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/437459">Condensin II inactivation in interphase does not affect chromatin folding or gene expression</a></td><td>Abdennur, N., Schwarzer, W., Pekowska, A., Shaltiel, I. A., Huber, W., Haering, C. H., Mirny, L., Spitz, F.</td><td>10.1101/437459</td><td>2018-10-07</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/423442">Physicochemical mechanotransduction alters nuclear shape and mechanics via heterochromatin formation</a></td><td>Stephens, A. D., Liu, P. Z., Kandula, V., Chen, H., Almassalha, L. M., Backman, V., O'Halloran, T., Adam, S. A., Goldman, R. D., Banigan, E. J., Marko, J. F.</td><td>10.1101/423442</td><td>2018-09-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/422683">Twist-bend coupling and the statistical mechanics of DNA: perturbation theory and beyond</a></td><td>Nomidis, S. K., Skoruppa, E., Carlon, E., Marko, J. F.</td><td>10.1101/422683</td><td>2018-09-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/266981">Spatial Chromatin Architecture Alteration by Structural Variations in Human Genomes at Population Scale</a></td><td>Sadowski, M., Kraft, A., Szalaj, P., Wlasnowolski, M., Tang, Z., Ruan, Y., Plewczynski, D.</td><td>10.1101/266981</td><td>2018-09-14</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/411835">MAPS: model-based analysis of long-range chromatin interactions from PLAC-seq and HiChIP experiments</a></td><td>Juric, I., Yu, M., Abnousi, A., Raviram, R., Fang, R., Zhao, Y., Zhang, Y., Yang, Y., Li, Y., Ren, B., Hu, M.</td><td>10.1101/411835</td><td>2018-09-08</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/407668">The accessible chromatin landscape of the hippocampus at single-cell resolution</a></td><td>Sinnamon, J. R., Torkenczy, K. A., Linhoff, M. W., Vitak, S. A., Pliner, H. A., Trapnell, C., Steemers, F. J., Mandel, G., Adey, A. C.</td><td>10.1101/407668</td><td>2018-09-04</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/399139">EndoC-βH1 multi-genomic profiling defines gene regulatory programs governing human pancreatic β cell identity and function</a></td><td>Lawlor, N., Marquez, E. J., Orchard, P., Narisu, N., Shamim, M. S., Thibodeau, A., Varshney, A., Kursawe, R., Erdos, M. R., Kanke, M., Gu, H., Pak, E., Dutra, A., Russell, S., Li, X., Piecuch, E., Luo, O., Chines, P. S., Fuchbserger, C., Sethupathy, P., Aiden, A. P., Ruan, Y., Aiden, E. L., Collins, F. S., Ucar, D., Parker, S. C. J., Stitzel, M. L.</td><td>10.1101/399139</td><td>2018-08-23</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/396424">Photon count estimation in single-molecule localization microscopy</a></td><td>Rieger, B., Stallinga, S., Hulleman, C. N., Thorsen, R. O., Grunwald, D., Hammer, M.</td><td>10.1101/396424</td><td>2018-08-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/392662">BAMM-SC: A Bayesian mixture model for clustering droplet-based single cell transcriptomic data from population studies</a></td><td>Sun, Z., Chen, L., Xin, H., Huang, Q., Cillo, A., Tabib, T., Ding, Y., Kolls, J., Bruno, T., Lafyatis, R., Vignali, D., Chen, K., Hu, M., Chen, W.</td><td>10.1101/392662</td><td>2018-08-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/386359">Allele-specific RNA imaging shows that allelic imbalances can arise in tissues through transcriptional bursting</a></td><td>Symmons, O., Chang, M., Mellis, I., Kalish, J. M., Bartolomei, M. S., Raj, A.</td><td>10.1101/386359</td><td>2018-08-07</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/385047">A combination of transcription factors mediates inducible interchromosomal pairing</a></td><td>Kim, S., Dunham, M., Shendure, J.</td><td>10.1101/385047</td><td>2018-08-05</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/384982">Condensin controls mitotic chromosome stiffness and stability without forming a structurally contiguous scaffold</a></td><td>Marko, J., Biggs, R., Sun, M., Hornick, J.</td><td>10.1101/384982</td><td>2018-08-04</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/375071">Transient DNA Binding Induces RNA Polymerase II Compartmentalization During Herpesviral Infection Distinct From Phase Separation</a></td><td>McSwiggen, D. T., Hansen, A. S., Marie-Nelly, H., Teves, S., Heckert, A., Dugast-Darzacq, C., Hao, Y., Umemoto, K. K., Tjian, R., Darzacq, X.</td><td>10.1101/375071</td><td>2018-07-30</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/374058">Walking along chromosomes with super-resolution imaging, contact maps, and integrative modeling</a></td><td>Nir, G., Farabella, I., Perez Estrada, C., Ebeling, C. G., Beliveau, B. J., Sasaki, H. M., Lee, S. H., Nguyen, S. C., McCole, R. B., Chattoraj, S., Erceg, J., AlHaj Abed, J., Martins, N. M. C., Nguyen, H. Q., Hannan, M. A., Russell, S., Durand, N. C., Rao, S. S. P., Kishi, J. Y., Soler-Vila, P., Di Pierro, M., Onuchic, J. N., Callahan, S., Schreiner, J., Stuckey, J., Yin, P., Lieberman Aiden, E., Marti-Renom, M. A., Wu, C.- t.</td><td>10.1101/374058</td><td>2018-07-28</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/377812">Dynamic multifactor hubs interact transiently with sites of active transcription in Drosophila embryos</a></td><td>Mir, M., Stadler, M. R., Ortiz, S. A., Harrison, M. M., Darzacq, X., Eisen, M. B.</td><td>10.1101/377812</td><td>2018-07-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/370395">Receptor-ligand rebinding kinetics in confinement</a></td><td>Erbas, A., Olvera de la Cruz, M., Marko, J.</td><td>10.1101/370395</td><td>2018-07-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/103614">Nucleotide sequence and DNaseI sensitivity are predictive of 3D chromatin architecture</a></td><td>Schreiber, J., Libbrecht, M., Bilmes, J., Noble, W.</td><td>10.1101/103614</td><td>2018-07-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/365866">CTCF sites display cell cycle dependent dynamics in factor binding and nucleosome positioning</a></td><td>Oomen, M. E., Hansen, A., Liu, Y., Darzacq, X., Dekker, J.</td><td>10.1101/365866</td><td>2018-07-11</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/349340">LADL: Light-activated dynamic looping for endogenous gene expression control</a></td><td>Rege, M., Kim, J. H., Valeri, J., Dunagin, M., Metzger, A., Gong, W., Beagan, J., Raj, A., Phillips-Cremins, J. E.</td><td>10.1101/349340</td><td>2018-06-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/349811">Feasibility of constructing multi-dimensional genomic maps of juvenile idiopathic arthritis</a></td><td>Jarvis, J. N., Zhu, L., Jiang, K., Wong, L., Buck, M. J., Chen, Y., Moncrief, H., Macintosh, L. A., Liu, T., Sing, X., Li, D., Wang, T., O'Neil, K. M.</td><td>10.1101/349811</td><td>2018-06-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/347955">Nuclear speckle fusion via long-range directional motion regulates the number and size of speckles</a></td><td>Kim, J., Han, K. Y., Khanna, N., Ha, T., Belmont, A. S.</td><td>10.1101/347955</td><td>2018-06-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/316372">RNA polymerase II clustering through CTD phase separation</a></td><td>Boehning, M., Dugast-Darzacq, C., Rankovic, M., Hansen, A. S., Yu, T.-K., Marie-Nelly, H., McSwiggen, D. T., Kokic, G., Dailey, G. M., Cramer, P., Darzacq, X., Zweckstetter, M.</td><td>10.1101/316372</td><td>2018-06-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/338053">High-throughput mapping of meiotic crossover and chromosome mis-segregation events in interspecific hybrid mice</a></td><td>Yin, Y., Jiang, Y., Berletch, J. B., Disteche, C. M., Noble, W. S., Steemers, F. J., Adey, A. C., Shendure, J. A.</td><td>10.1101/338053</td><td>2018-06-12</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/121889">HiGlass: Web-based Visual Exploration and Analysis of Genome Interaction Maps</a></td><td>Kerpedjiev, P., Abdennur, N., Lekschas, F., McCallum, C., Dinkla, K., Strobelt, H., Luber, J. M., Ouellette, S. B., Azhir, A., Kumar, N., Hwang, J., Lee, S., Alver, B. H., Pfister, H., Mirny, L. A., Park, P. J., Gehlenborg, N.</td><td>10.1101/121889</td><td>2018-06-04</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/325209">Cohesin interacts with a panoply of splicing factors required for cell cycle progression and genomic organization</a></td><td>Kim, J.-S., He, X., Liu, J., Duan, Z., Kim, T., Gerard, J., Kim, B., Lane, W. S., Noble, W. S., Budnik, B., Waldman, T.</td><td>10.1101/325209</td><td>2018-05-17</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/208710">Dynamic and Selective Low-Complexity Domain Interactions Revealed by Live-Cell Single-Molecule Imaging</a></td><td>Chong, S., Dugast-Darzacq, C., Liu, Z., Dong, P., Dailey, G., Cattoglio, C., Banala, S., Lavis, L., Darzacq, X., Tjian, R.</td><td>10.1101/208710</td><td>2018-05-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/259515">GITAR: an Open Source Tool for Analysis and Visualization of Hi-C Data</a></td><td>Calandrelli, R., Wu, Q., Guan, J., Zhong, S.</td><td>10.1101/259515</td><td>2018-05-08</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/222794">Exponential fluorescent amplification of individual RNAs using clampFISH probes</a></td><td>Rouhanifard, S. H., Mellis, I. A., Dunagin, M., Bayatpour, S., Symmons, O., Cote, A., Raj, A.</td><td>10.1101/222794</td><td>2018-05-07</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/307892">TSA-Seq Mapping of Nuclear Genome Organization</a></td><td>Chen, Y., Zhang, Y., Wang, Y., Zhang, L., Brinkman, E. K., Adam, S. A., Goldman, R., van Steensel, B., Ma, J., Belmont, A. S.</td><td>10.1101/307892</td><td>2018-04-25</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/305359">Complementary chromosome folding by transcription factors and cohesin</a></td><td>Pereira, M. C. F., Brackley, C. A., Michieletto, D., Annunziatella, C., Bianco, S., Chiariello, A. M., Nicodemi, M., Marenduzzo, D.</td><td>10.1101/305359</td><td>2018-04-23</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/219253">Quantitative imaging of chromatin decompaction in living cells</a></td><td>Dultz, E., Mancini, R., Polles, G., Vallotton, P., Alber, F., Weis, K.</td><td>10.1101/219253</td><td>2018-04-22</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/174649">Mitotic chromosomes fold by condensin-dependent helical winding of chromatin loop arrays</a></td><td>Gibcus, J. H., Samejima, K., Goloborodko, A., Samejima, I., Naumova, N., Kanemaki, M., Xie, L., Paulson, J. R., Earnshaw, W. C., Mirny, L. A., Dekker, J.</td><td>10.1101/174649</td><td>2018-04-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/300483">RNAs as proximity labeling media for identifying nuclear speckle positions relative to the genome</a></td><td>Chen, W., Yan, Z., Li, S., Huang, N., Huang, X., Zhang, J., Zhong, S.</td><td>10.1101/300483</td><td>2018-04-12</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/295915">An inter-chromosomal transcription hub activates the unfolded protein response in plasma cells</a></td><td>Bortnick, A., He, Z., Aubrey, M., Chandra, V., Denholtz, M., Chen, K., Lin, Y. C., Murre, C.</td><td>10.1101/295915</td><td>2018-04-07</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/237834">A quantitative map of human Condensins provides new insights into mitotic chromosome architecture</a></td><td>Walther, N., Hossain, M. J., Politi, A. Z., Koch, B., Kueblbeck, M., Oedegaard-Fougner, O., Lampe, M., Ellenberg, J.</td><td>10.1101/237834</td><td>2018-03-22</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/285650">Identification of cis elements for spatio-temporal control of DNA replication</a></td><td>Sima, J., Chakraborty, A., Dileep, V., Michalski, M., Rivera-Mulia, J. C., Trevilla-Garcia, C., Klein, K. N., Bartlett, D., Washburn, B. K., Paulsen, M. T., Vera, D., Nora, E. P., Kraft, K., Mundlos, S., Bruneau, B. G., Ljungman, M., Fraser, P., Ay, F., Gilbert, D. M.</td><td>10.1101/285650</td><td>2018-03-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/283655">Mapping local and global liquid-liquid phase behavior in living cells using light-activated multivalent seeds</a></td><td>Bracha, D., Walls, M. T., Wei, M.-T., Zhu, L., Kurian, M., Toettcher, J. E., Brangwynne, C. P.</td><td>10.1101/283655</td><td>2018-03-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/283093">Continuous-trait probabilistic model for comparing multi-species functional genomic data</a></td><td>Yang, Y., Gu, Q., Zhang, Y., Sasaki, T., Crivello, J., O'Neill, R. J., Gilbert, D. M., Ma, J.</td><td>10.1101/283093</td><td>2018-03-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/177832">GIVE: toward portable genome browsers for personal websites</a></td><td>Cao, X., Yan, Z., Wu, Q., Zheng, A., Zhong, S.</td><td>10.1101/177832</td><td>2018-03-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/138677">SAVER: Gene expression recovery for UMI-based single cell RNA sequencing</a></td><td>Huang, M., Wang, J., Torre, E., Dueck, H., Shaffer, S., Bonasio, R., Murray, J., Raj, A., Li, M., Zhang, N. R.</td><td>10.1101/138677</td><td>2018-03-08</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/275354">Transcriptional burst initiation and polymerase pause release are key control points of transcriptional regulation</a></td><td>Bartman, C. R., Keller, C. A., Giardine, B., Hardison, R. C., Blobel, G. A., Raj, A.</td><td>10.1101/275354</td><td>2018-03-02</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/115436">ChromoTrace: Reconstruction of 3D Chromosome Configurations by Super-Resolution Microscopy</a></td><td>Barton, C., Morganella, S., Oedegaard, O., Alexander, S., Ries, J., Fitzgerald, T., Ellenberg, J., Birney, E.</td><td>10.1101/115436</td><td>2018-02-22</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/264648">Emerging Evidence of Chromosome Folding by Loop Extrusion</a></td><td>Fudenberg, G., Abdennur, N., Imakaev, M., Goloborodko, A., Mirny, L.</td><td>10.1101/264648</td><td>2018-02-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/085241">Predicting Enhancer-Promoter Interaction from Genomic Sequence with Deep Neural Networks</a></td><td>Singh, S., Yang, Y., Poczos, B., Ma, J.</td><td>10.1101/085241</td><td>2018-02-05</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/188755">Measuring the reproducibility and quality of Hi-C data</a></td><td>Yardımcı, G. G., Ozadam, H., Sauria, M. E. G., Ursu, O., Yan, K.-K., Yang, T., Chakraborty, A., Kaul, A., Lajoie, B. R., Song, F., Zhan, Y., Ay, F., Gerstein, M., Kundaje, A., Li, Q., Taylor, J., Yue, F., Dekker, J., Noble, W. S.</td><td>10.1101/188755</td><td>2018-02-05</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/259689">Defect-Facilitated Buckling in Supercoiled Double-Helix DNA</a></td><td>Brahmachari, S., Dittmore, A., Takagi, Y., Neuman, K. C., Marko, J. F.</td><td>10.1101/259689</td><td>2018-02-04</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/259416">Predicting CTCF-mediated chromatin loops using CTCF-MP</a></td><td>Zhang, R., Wang, Y., Yang, Y., Zhang, Y., Ma, J.</td><td>10.1101/259416</td><td>2018-02-02</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/171819">C-BERST: Defining subnuclear proteomic landscapes at genomic elements with dCas9-APEX2</a></td><td>Gao, X. D., Tu, L.-C., Mir, A., Rodriguez, T., Ding, Y., Leszyk, J., Dekker, J., Shaffer, S. A., Zhu, L. J., Wolfe, S. A., Sontheimer, E. J.</td><td>10.1101/171819</td><td>2018-01-31</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/256818">Bend-Induced Twist Waves and the Structure of Nucleosomal DNA</a></td><td>Skoruppa, E., Nomidis, S. K., Marko, J. F., Carlon, E.</td><td>10.1101/256818</td><td>2018-01-30</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/254797">The Juicebox Assembly Tools module facilitates de novo assembly of mammalian genomes with chromosome-length scaffolds for under $1000</a></td><td>Dudchenko, O., Shamim, M. S., Batra, S., Durand, N. C., Musial, N. T., Mostofa, R., Pham, M., Glenn St Hilaire, B., Yao, W., Stamenova, E., Hoeger, M., Nyquist, S. K., Korchina, V., Pletch, K., Flanagan, J. P., Tomaszewicz, A., McAloose, D., Perez Estrada, C., Novak, B. J., Omer, A. D., Aiden, E. L.</td><td>10.1101/254797</td><td>2018-01-28</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/252049">Multiplex Chromatin Interaction Analysis with Single-Molecule Precision</a></td><td>Zheng, M., Tian, S. Z., Maurya, R., Lee, B., Kim, M., Capurso, D., Piecuch, E., Gong, L., Zhu, J. J., Wong, C. H., Ngan, C. Y., Wang, P., Ruan, X., Wei, C.-L., Ruan, Y.</td><td>10.1101/252049</td><td>2018-01-25</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/189373">Correlative live and super-resolution imaging reveals the dynamic structure of replication domains</a></td><td>Xiang, W., Roberti, M. J., Heriche, J.-K., Huet, S., Alexander, S., Ellenberg, J.</td><td>10.1101/189373</td><td>2018-01-24</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/244038">Heterochromatin drives organization of conventional and inverted nuclei</a></td><td>Falk, M., Feodorova, Y., Naumova, N., Imakaev, M., Lajoie, B. R., Leonhardt, H., Joffe, B., Dekker, J., Fudenberg, G., Solovei, I., Mirny, L.</td><td>10.1101/244038</td><td>2018-01-09</td></tr>

        <!--2017--> 
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/240747">Improved Aedes aegypti mosquito reference genome assembly enables biological discovery and vector control</a></td><td>Matthews, B. J., Dudchenko, O., Kingan, S., Koren, S., Antoshechkin, I., Crawford, J. E., Glassford, W. J., Herre, M., Redmond, S. N., Rose, N. H., Weedall, G. D., Wu, Y., Batra, S. S., Brito-Sierra, C. A., Buckingham, S. D., Campbell, C. L., Chan, S., Cox, E., Evans, B. R., Fansiri, T., Filipovic, I., Fontaine, A., Gloria-Soria, A., Hall, R., Joardar, V. S., Jones, A. K., Kay, R. G. G., Kodali, V., Lee, J., Lycett, G. J., Mitchell, S. N., Muehling, J., Murphy, M. R., Omer, A., Partridge, F. A., Peluso, P., Aiden, A. P., Ramasamy, V., Rasic, G., Roy, S., Saavedra-Rodriguez, K., Sharan, S., Sha</td><td>10.1101/240747</td><td>2017-12-29</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/239368">Chromatin interaction data visualization in the WashU Epigenome Browser</a></td><td>Li, D., Hsu, S., Purushotham, D., Wang, T.</td><td>10.1101/239368</td><td>2017-12-24</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/165340">Orientation-dependent Dxz4 contacts shape the 3D structure of the inactive X chromosome</a></td><td>Bonora, G., Deng, X., Fang, H., Ramani, V., Qui, R., Berletch, J., Filippova, G. N., Duan, Z., Schendure, J., Noble, W. S., Disteche, C. M.</td><td>10.1101/165340</td><td>2017-12-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/221762">Allele-specific control of replication timing and genome organization during development</a></td><td>Rivera-Mulia, J. C., Dimond, A., Vera, D., Trevilla-Garcia, C., Sasaki, T., Zimmerman, J., Dupont, C., Gribnau, J., Fraser, P., Gilbert, D. M.</td><td>10.1101/221762</td><td>2017-11-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/222877">Dynamic reorganization of nuclear architecture during human cardiogenesis</a></td><td>Fields, P. A., Ramani, V., Bonora, G., Yardimci, G. G., Bertero, A., Reinecke, H., Pabon, L., Noble, W. S., Shendure, J., Murry, C.</td><td>10.1101/222877</td><td>2017-11-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/219683">Higher-order inter-chromosomal hubs shape 3-dimensional genome organization in the nucleus</a></td><td>Quinodoz, S. A., Ollikainen, N., Tabak, B., Palla, A., Schmidt, J. M., Detmar, E., Lai, M., Shishkin, A., Bhat, P., Trinh, V., Aznauryan, E., Russell, P., Cheng, C., Jovanovic, M., Chow, A., McDonel, P., Garber, M., Guttman, M.</td><td>10.1101/219683</td><td>2017-11-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/218768">CHRAC/ACF Contribute to the Repressive Ground State of Chromatin</a></td><td>Scacchetti, A., Brueckner, L., Jain, D., Schauer, T., Zhang, X., Schnorrer, F., van Steensel, B., Straub, T., Becker, P. B.</td><td>10.1101/218768</td><td>2017-11-13</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/212928">A Cell Type-Specific Class of Chromatin Loops Anchored at Large DNA Methylation Nadirs</a></td><td>Jeong, M., Huang, X., Zhang, X., Su, J., Shamim, M., Bochkov, I., Reyes, J., Jung, H., Heikamp, E., Presser Aiden, A., Li, W., Aiden, E., Goodell, M. A.</td><td>10.1101/212928</td><td>2017-11-09</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/214361">Removing unwanted variation between samples in Hi-C experiments</a></td><td>Fletez-Brant, K., Qiu, Y., Gorkin, D. U., Hu, M., Hansen, K. D.</td><td>10.1101/214361</td><td>2017-11-06</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/206367">Chromatin histone modifications and rigidity affect nuclear morphology independent of lamins</a></td><td>Stephens, A. D., Liu, P. Z., Banigan, E. J., Almassalha, L. M., Backman, V., Adam, S. A., Goldman, R., Marko, J.</td><td>10.1101/206367</td><td>2017-10-29</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/171983">Spot-On: robust model-based analysis of single-particle tracking experiments</a></td><td>Hansen, A. S., Woringer, M., Grimm, J. B., Lavis, L. D., Tjian, R., Darzacq, X.</td><td>10.1101/171983</td><td>2017-10-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/206094">Locus-Specific Enhancer Hubs And Architectural Loop Collisions Uncovered From Single Allele DNA Topologies</a></td><td>Allahyar, A., Vermeulen, C., Bouwman, B., Krijger, P., Verstegen, M., Geeven, G., van Kranenburg, M., Pieterse, M., Straver, R., Haarhuis, J., Teunissen, H., Renkens, I., Kloosterman, W., Rowland, B., de Wit, E., de Ridder, J., de Laat, W.</td><td>10.1101/206094</td><td>2017-10-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/205740">Juicebox.js provides a cloud-based visualization system for Hi-C data</a></td><td>Robinson, J., Turner, D., Durand, N. C., Thorvaldsdottir, H., Mesirov, J. P., Aiden, E. L.</td><td>10.1101/205740</td><td>2017-10-19</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/204776">Fluorescence polarization control for on-off switching of single molecules at cryogenic temperatures</a></td><td>Hulleman, C., Huisman, M., Moerland, R., Grunwald, D., Stallinga, S., Rieger, B.</td><td>10.1101/204776</td><td>2017-10-17</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/196261">Chromatin Organization by an Interplay of Loop Extrusion and Compartmental Segregation</a></td><td>Nuebler, J., Fudenberg, G., Imakaev, M., Abdennur, N., Mirny, L.</td><td>10.1101/196261</td><td>2017-10-03</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/197566">Mechanics and buckling of biopolymeric shells and cell nuclei</a></td><td>Banigan, E. J., Stephens, A. D., Marko, J. F.</td><td>10.1101/197566</td><td>2017-10-02</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/196345">Nucleation of multiple buckled structures in intertwined DNA double helices</a></td><td>Brahmachari, S., Gunn, K. H., Giuntoli, R. D., Mondragon, A., Marko, J. F.</td><td>10.1101/196345</td><td>2017-09-30</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/195966">CRISPR-Based DNA Imaging in Living Cells Reveals Cell Cycle-Dependent Chromosome Dynamics</a></td><td>Ma, H., Tu, L.-C., Naseri, A., Chung, Y.-C., Grunwald, D., Zhang, S., Pederson, T.</td><td>10.1101/195966</td><td>2017-09-29</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/191213">Genetic drivers of repeat expansion disorders localize to 3-D chromatin domain boundaries</a></td><td>Sun, J., Zhou, L., Emerson, D. J., Gilgenast, T. G., Titus, K., Beagan, J. A., Phillips-Cremins, J. E.</td><td>10.1101/191213</td><td>2017-09-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/189654">High-throughput identification of RNA nuclear enrichment sequences</a></td><td>Shukla, C. J., McCorkindale, A. L., Gerhardinger, C., Korthauer, K. D., Cabili, M. N., Shechner, D. M., Irizarry, R. A., Maass, P. G., Rinn, J. L.</td><td>10.1101/189654</td><td>2017-09-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/187203">SHAMAN: bin-free randomization, normalization and screening of Hi-C matrices</a></td><td>Cohen, N. M., Olivares-Chauvet, P., Lubling, Y., Baran, Y., Lifshitz, A., Hoichman, M., Tanay, A.</td><td>10.1101/187203</td><td>2017-09-12</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/184846">Using DNase Hi-C techniques to map global and local three-dimensional genome architecture at high resolution</a></td><td>Ma, W., Ay, F., Lee, C., Gulsoy, G., Deng, X., Cook, S., Hesson, J., Cavanaugh, C., Ware, C. B., Krumm, A., Shendure, J., Blau, C. A., Disteche, C. M., Noble, W. S., Duan, Z.</td><td>10.1101/184846</td><td>2017-09-05</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/177766">A Mechanism of Cohesin-Dependent Loop Extrusion Organizes Zygotic Genome Architecture</a></td><td>Gassler, J., Brandao, H. B., Imakaev, M., Flyamer, I. M., Ladstatter, S., Bickmore, W. A., Peters, J.-M., Mirny, L. A., Tachibana-Konwalski, K.</td><td>10.1101/177766</td><td>2017-08-17</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/135699">3D Single-Molecule Super-Resolution Microscopy With A Tilted Light Sheet</a></td><td>Gustavsson, A.-K., Petrov, P. N., Lee, M. Y., Shechtman, Y., Moerner, W. E.</td><td>10.1101/135699</td><td>2017-08-14</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/172643">Fast, robust and precise 3D localization for arbitrary point spread functions</a></td><td>Li, Y., Mund, M., Hoess, P., Matti, U., Nijmeijer, B., Jimenez Sabinina, V., Ellenberg, J., Schoen, I., Ries, J.</td><td>10.1101/172643</td><td>2017-08-10</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/101386">HiCRep: assessing the reproducibility of Hi-C data using a stratum-adjusted correlation coefficient</a></td><td>Yang, T., Zhang, F., Yardimci, G. G., Song, F., Hardison, R. C., Noble, W. S., Yue, F., Li, Q.</td><td>10.1101/101386</td><td>2017-08-04</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/171801">Heterogeneity and Intrinsic Variation in Spatial Genome Organization</a></td><td>Finn, E., Pegoraro, G., Brandao, H. B., Valton, A.-L., Oomen, M. E., Dekker, J., Mirny, L., Misteli, T.</td><td>10.1101/171801</td><td>2017-08-03</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/083212">Natural Selection has Shaped Coding and Non-coding Transcription in Primate CD4+ T-cells</a></td><td>Danko, C. G., Choate, L. A., Marks, B. A., Rice, E. J., Wang, Z., Chu, T., Martins, A. L., Dukler, N., Coonrod, S. A., Tait Wojno, E. D., Lis, J. T., Kraus, W. L., Siepel, A.</td><td>10.1101/083212</td><td>2017-07-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/158352">Single-cell replication profiling reveals stochastic regulation of the mammalian replication-timing program.</a></td><td>Dileep, V., Gilbert, D. M.</td><td>10.1101/158352</td><td>2017-07-14</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/162156">CRISPR/Cas9-mediated Knock-in of an Optimized TetO Repeat for Live Cell Imaging of Endogenous Loci</a></td><td>Tasan, I., Sustackova, G., Zhang, L., Kim, J., Sivaguru, M., HamediRad, M., Wang, Y., Genova, J., Ma, J., Belmont, A. S., Zhao, H.</td><td>10.1101/162156</td><td>2017-07-12</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/123588">HiPiler: Visual Exploration Of Large Genome Interaction Matrices With Interactive Small Multiples</a></td><td>Lekschas, F., Bach, B., Kerpedjiev, P., Gehlenborg, N., Pfister, H.</td><td>10.1101/123588</td><td>2017-07-09</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/159004">A Fluorogenic Array Tag for Temporally Unlimited Single Molecule Tracking</a></td><td>Ghosh, R. P., Franklin, M. J., Draper, W. E., Shi, Q., Liphardt, J. T.</td><td>10.1101/159004</td><td>2017-07-03</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/156091">Single-molecule FISH in Drosophila muscle reveals location dependent mRNA composition of megaRNPs</a></td><td>Noma, A., Smith, C. S., Huisman, M., Martin, R. M., Moore, M. J., Grunwald, D.</td><td>10.1101/156091</td><td>2017-06-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/155473">Chromatin accessibility dynamics of myogenesis at single cell resolution</a></td><td>Pliner, H., Packer, J., McFaline-Figueroa, J., Cusanovich, D., Daza, R., Srivatsan, S., Qiu, X., Jackson, D., Minkina, A., Adey, A., Steemers, F., Shendure, J., Trapnell, C.</td><td>10.1101/155473</td><td>2017-06-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/153098">Live-cell mapping of organelle-associated RNAs via proximity biotinylation combined with protein-RNA crosslinking</a></td><td>Kaewsapsak, P., Shechner, D. M., Mallard, W., Rinn, J. L., Ting, A. Y.</td><td>10.1101/153098</td><td>2017-06-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/133124">Dense Bicoid Hubs Accentuate Binding along the Morphogen Gradient</a></td><td>Mir, M., Reimer, A., Haines, J. E., Li, X.-Y., Stadler, M., Garcia, H., Eisen, M. B., Darzacq, X.</td><td>10.1101/133124</td><td>2017-06-08</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/146241">qSR: A software for quantitative analysis of single molecule and super-resolution data</a></td><td>Andrews, J. O., Narayanan, A., Spille, J.-H., Cho, W.-K., Thaler, J. D., Cisse, I. I.</td><td>10.1101/146241</td><td>2017-06-05</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/145110">Reconstructing Spatial Transport Distributions In The Nuclear Pore Complex From 2D Images -- How Reliable Is It?</a></td><td>Tu, L.-C., Huisman, M., Chung, Y.-C., Smith, C., Grunwald, D.</td><td>10.1101/145110</td><td>2017-06-02</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/142604">Specific Virus-Host Genome Interactions Revealed By Tethered Chromosome Conformation Capture</a></td><td>Li, H., Kalhor, R., Li, B., Su, T., Berk, A., Kurdistani, S., Alber, F., Chen, L.</td><td>10.1101/142604</td><td>2017-05-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/142026">Static And Dynamic DNA Loops Form AP-1 Bound Activation Hubs During Macrophage Development</a></td><td>Phanstiel, D. H., Van Bortle, K., Spacek, D. V., Hess, G. T., Saad Shamim, M., Machol, I., Love, M. I., Lieberman Aiden, E., Bassik, M. C., Snyder, M. P.</td><td>10.1101/142026</td><td>2017-05-25</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/138289">A Comparison Between Single Cell RNA Sequencing And Single Molecule RNA FISH For Rare Cell Analysis</a></td><td>Torre, E. A., Dueck, H., Shaffer, S., Gospocic, J., Gupte, R., Bonasio, R., Kim, J., Murray, J., Raj, A.</td><td>10.1101/138289</td><td>2017-05-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/139782">Cohesin Loss Eliminates All Loop Domains, Leading To Links Among Superenhancers And Downregulation Of Nearby Genes</a></td><td>Rao, S., Huang, S.-C., Glenn St. Hilaire, B., Engreitz, J. M., Perez, E. M., Kieffer-Kwon, K.-R., Sanborn, A. L., Johnstone, S. E., Bochkov, I. D., Huang, X., Shamim, M. S., Omer, A. D., Bernstein, B. E., Casellas, R., Lander, E. S., Lieberman Aiden, E.</td><td>10.1101/139782</td><td>2017-05-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/135905">Torque and buckling in stretched intertwined double-helix DNAs</a></td><td>Marko, J., Brahmachari, S.</td><td>10.1101/135905</td><td>2017-05-09</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/135947">Facilitated Dissociation Of Transcription Factors From Single DNA Binding Sites</a></td><td>Kamar, R. I., Banigan, E. J., Erbas, A., Giuntoli, R. D., Olvera de la Cruz, M., Johnson, R. C., Marko, J. F.</td><td>10.1101/135947</td><td>2017-05-09</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/134767">Ki-67 Contributes To Normal Cell Cycle Progression And Inactive X Heterochromatin In p21 Checkpoint-Proficient Human Cells</a></td><td>Sun, X., Bizhanova, A., Matheson, T. D., Yu, J., Zhu, L. J., Kaufman, P. D.</td><td>10.1101/134767</td><td>2017-05-08</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/130567">Supercoiling DNA locates mismatches</a></td><td>Dittmore, A., Brahmachari, S., Takagi, Y., Marko, J. F., Neuman, K. C.</td><td>10.1101/130567</td><td>2017-04-25</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/128280">Nuclear Microenvironments Modulate Transcription From Low-Affinity Enhancers</a></td><td>Crocker, J., Tsai, A., Muthusamy, A. K., Lavis, L. D., Singer, R. H., Stern, D. L.</td><td>10.1101/128280</td><td>2017-04-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/125815">A Versatile Compressed Sensing Scheme For Faster And Less Phototoxic 3D Fluorescence Microscopy</a></td><td>Woringer, M., Darzacq, X., Zimmer, C., Mir, M.</td><td>10.1101/125815</td><td>2017-04-10</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/119651">An Integrative Framework For Detecting Structural Variations In Cancer Genomes</a></td><td>Dixon, J., Xu, J., Dileep, V., Zhan, Y., Song, F., Le, V. T., Yardimci, G. G., Chakraborty, A., Bann, D. V., Wang, Y., Clark, R., Zhang, L., Yang, H., Liu, T., Iyyanki, S., An, L., Pool, C., Sasaki, T., Mulia, J. C. R., Ozadam, H., Lajoie, B. R., Kaul, R., Buckley, M., Lee, K., Diegel, M., Pezic, D., Ernst, C., Hadjur, S., Odom, D. T., Stamatoyannopoulos, J. A., Broach, J. R., Hardison, R., Ay, F., Noble, W. S., Dekker, J., Gilbert, D. M., Yue, F.</td><td>10.1101/119651</td><td>2017-03-28</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/118737">Preservation of Chromatin Organization after Acute Loss of CTCF in Mouse Embryonic Stem Cells</a></td><td>Kubo, N., Ishii, H., Gorkin, D., Meitinger, F., Xiong, X., Fang, R., Liu, T., Ye, Z., Li, B., Dixon, J., Desai, A., Zhao, H., Ren, B.</td><td>10.1101/118737</td><td>2017-03-20</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/117531">HUGIn: Hi-C Unifying Genomic Interrogator</a></td><td>Martin, J. S., Xu, Z., Reiner, A. P., Mohlke, K. L., Sullivan, P., Ren, B., Hu, M., Li, Y.</td><td>10.1101/117531</td><td>2017-03-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/105676">Free energy based high-resolution modeling of CTCF-mediated chromatin loops for human genome</a></td><td>Dawson, W., Plewczynski, D.</td><td>10.1101/105676</td><td>2017-03-08</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/104653">Repli-seq: genome-wide analysis of replication timing by next-generation sequencing</a></td><td>Marchal, C., Sasaki, T., Vera, D., Wilson, K., Sima, J., Rivera-Mulia, J.-C., Trevilla Garcia, C., Nogues, C., Nafie, E., Gilbert, D. M.</td><td>10.1101/104653</td><td>2017-03-01</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/112631">HiCPlus: Resolution Enhancement of Hi-C interaction heatmap</a></td><td>Zhang, Y., An, L., Hu, M., Tang, J., Yue, F.</td><td>10.1101/112631</td><td>2017-03-01</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/112268">The 3D Genome Browser: a web-based browser for visualizing 3D genome organization and long-range chromatin interactions</a></td><td>Wang, Y., Zhang, B., Zhang, L., An, L., Xu, J., Li, D., Choudhary, M. N., Li, Y., Hu, M., Hardison, R., Wang, T., Yue, F.</td><td>10.1101/112268</td><td>2017-02-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/111690">A fluorogenic nanobody array tag for prolonged single molecule imaging in live cells</a></td><td>Ghosh, R., Draper, W., Franklin, J. M., Shi, Q., Liphardt, J.</td><td>10.1101/111690</td><td>2017-02-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/083014">p53 dynamically directs TFIID assembly on target gene promoters</a></td><td>Coleman, R. A., Qiao, Z., Singh, S. K., Peng, C. S., Cianfrocco, M., Zhang, Z., Piasecka, A., Aldeborgh, H., Basishvili, G., Liu, W.-L.</td><td>10.1101/083014</td><td>2017-02-24</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/111179">Real-time chromatin dynamics at the single gene levelduring transcription activation</a></td><td>Germier, T., Kocanova, S., Walther, N., Bancaud, A., Shaban, H. A., Sellou, H., Politi, A., Ellenberg, J., Gallardo, F., Bystricky, K.</td><td>10.1101/111179</td><td>2017-02-23</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/110239">Histone H3 Lysine 4 methyltransferases MLL3 and MLL4 Modulate Long-range Chromatin Interactions at Enhancers</a></td><td>Yan, J., Chen, S.-A. A., Local, A., Liu, T., Qiu, Y., Lee, A.-Y., Jung, I., Preissl, S., Rivera, C. M., Wang, C., Ishii, H., Fang, R., Ye, Z., Ge, K., Hu, M., Ren, B.</td><td>10.1101/110239</td><td>2017-02-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/110668">Reversed graph embedding resolves complex single-cell developmental trajectories</a></td><td>Qiu, X., Mao, Q., Tang, Y., Wang, L., Chawla, R., Pliner, H., Trapnell, C.</td><td>10.1101/110668</td><td>2017-02-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/104844">Comprehensive single cell transcriptional profiling of a multicellular organism by combinatorial indexing</a></td><td>Cao, J., Packer, J. S., Ramani, V., Cusanovich, D. A., Huynh, C., Daza, R., Qiu, X., Lee, C., Furlan, S. N., Steemers, F. J., Adey, A., Waterston, R. H., Trapnell, C., Shendure, J.</td><td>10.1101/104844</td><td>2017-02-02</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/099523">Direct visualization of transcriptional activation by physical enhancer-promoter proximity</a></td><td>Chen, H., Fujioka, M., Jaynes, J. B., Gregor, T.</td><td>10.1101/099523</td><td>2017-01-31</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/103499">The 4D Nucleome Project</a></td><td>Dekker, J., Belmont, A. S., Guttman, M., Leshyk, V. O., Lis, J. T., Lomvardas, S., Mirny, L. A., O'Shea, C. C., Park, P. J., Ren, B., Ritland, J. C., Shendure, J., Zhong, S., The 4D Nucleome Network</td><td>10.1101/103499</td><td>2017-01-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/103358">PGS: a dynamic and automated population-based genome structure software</a></td><td>Hua, N., Tjong, H., Shin, H., Gong, K., Zhou, X. J., Alber, F.</td><td>10.1101/103358</td><td>2017-01-26</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/101477">Multiplexed dynamic imaging of genomic loci in single cells by combined CRISPR imaging and DNA sequential FISH</a></td><td>Takei, Y., Shah, S., Harvey, S., Qi, L. S., Cai, L.</td><td>10.1101/101477</td><td>2017-01-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/099911">The 3D genome organization of Drosophila melanogaster through data integration</a></td><td>Li, Q., Tjong, H., Li, X., Gong, K., Zhou, X. J., Chiolo, I., Alber, F.</td><td>10.1101/099911</td><td>2017-01-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/100198">Comprehensive characterization of neutrophil genome topology</a></td><td>Zhu, Y., Gong, K., Denholtz, M., Chandra, V., Kamps, M. P., Alber, F., Murre, C.</td><td>10.1101/100198</td><td>2017-01-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/099804">Polycomb-Mediated Chromatin Loops Revealed by a Sub-Kilobase Resolution Chromatin Interaction Map</a></td><td>Eagen, K. P., Lieberman Aiden, E., Kornberg, R. D.</td><td>10.1101/099804</td><td>2017-01-12</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/095802">Targeted degradation of CTCF decouples local insulation of chromosome domains from higher-order genomic compartmentalization</a></td><td>Nora, E. P., Goloborodko, A., Valton, A.-L., Gibcus, J. H., Uebersohn, A., Abdennur, N., Dekker, J., Mirny, L., Bruneau, B.</td><td>10.1101/095802</td><td>2017-01-09</td></tr>

        <!--2016--> 
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/094946">Cohesin dependent compaction of mitotic chromosomes</a></td><td>Schalbetter, S. A., Goloborodko, A., Fudenberg, G., Belton, J. M., Miles, C., Yu, M., Dekker, J., Mirny, L., Baxter, J.</td><td>10.1101/094946</td><td>2016-12-17</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/094185">Two independent modes of chromosome organization are revealed by cohesin removal</a></td><td>Schwarzer, W., Abdennur, N., Goloborodko, A., Pekowska, A., Fudenberg, G., Loe-Mie, Y., Fonseca, N. A., Huber, W., Haering, C., Mirny, L., Spitz, F.</td><td>10.1101/094185</td><td>2016-12-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/094466">Cell cycle dynamics of chromosomal organisation at single-cell resolution</a></td><td>Nagano, T., Lubling, Y., Varnai, C., Dudley, C., Leung, W., Baran, Y., Mandelson Cohen, N., Wingett, S., Fraser, P., Tanay, A.</td><td>10.1101/094466</td><td>2016-12-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/093476">CTCF and Cohesin Regulate Chromatin Loop Stability with Distinct Dynamics</a></td><td>Hansen, A. S., Pustova, I., Cattoglio, C., Tjian, R., Darzacq, X.</td><td>10.1101/093476</td><td>2016-12-13</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/091827">The dynamic three-dimensional organization of the diploid yeast genome</a></td><td>Kim, S., Liachko, I., Brickner, D. G., Cook, K., Noble, W. S., Brickner, J. H., Shendure, J., Dunham, M.</td><td>10.1101/091827</td><td>2016-12-05</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/090001">HI-C 2.0: AN OPTIMIZED HI-C PROCEDURE FOR HIGH-RESOLUTION GENOME-WIDE MAPPING OF CHROMOSOME CONFORMATION</a></td><td>Belaghzal, H., Dekker, J., Gibcus, J. H.</td><td>10.1101/090001</td><td>2016-11-27</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/089011">Detecting hierarchical 3-D genome domain reconfiguration with network modularity</a></td><td>Norton, H. K., Huang, H., Emerson, D. J., Kim, J., Gu, S., Bassett, D. S., Phillips-Cremins, J. E.</td><td>10.1101/089011</td><td>2016-11-22</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://arxiv.org/abs/1611.05654">Calibrating photon counts from a single image</a></td><td>Heintzmann, R., Relich, P. K., Nieuwenhuizen, R. P. J., Lidke, K. A., Rieger, B.</td><td>-</td><td>2016-11-17</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/088146">Visualizing adenosine to inosine RNA editing in single mammalian cells</a></td><td>Mellis, I. A., Gupte, R. K., Raj, A., Rouhanifard, S. H.</td><td>10.1101/088146</td><td>2016-11-16</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/086017">Software tools for visualizing Hi-C data</a></td><td>Yardimci, G. G., Noble, W. S.</td><td>10.1101/086017</td><td>2016-11-07</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/082339">The p150N domain of Chromatin Assembly Factor-1 regulates Ki-67 accumulation on the mitotic perichromosomal layer</a></td><td>Matheson, T., Kaufman, P.</td><td>10.1101/082339</td><td>2016-10-21</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/081448">FISH-ing for captured contacts: towards reconciling FISH and 3C</a></td><td>Fudenberg, G., Imakaev, M.</td><td>10.1101/081448</td><td>2016-10-17</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/076893">Comparative analysis of 2D and 3D distance measurements to study spatial genome organization</a></td><td>Elizabeth Finn, Gianluca Pegoraro, Sigal Shachar, Tom Misteli</td><td>10.1101/076893</td><td>2016-09-23</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/074294">MAPPING OF LONG-RANGE CHROMATIN INTERACTIONS BY PROXIMITY LIGATION ASSISTED CHIP-SEQ</a></td><td>Rongxin Fang, Miao Yu, Guoqiang Li, Sora Chee, Tristin Liu, Anthony Schmitt, Bing Ren</td><td>10.1101/074294</td><td>2016-09-09</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/071357">Micro-C XL: assaying chromosome conformation at length scales from the nucleosome to the entire genome</a></td><td>Tsung-Han Hsieh, Geoffrey Fudenberg, Anton Goloborodko, Oliver Rando</td><td>10.1101/071357</td><td>2016-08-24</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/066464">A Dynamic Mode of Mitotic Bookmarking by Transcription Factors</a></td><td>Sheila S Teves, Luye An, Anders S Hansen, Liangqi Xie, Xavier Darzacq, Robert Tjian</td><td>10.1101/066464</td><td>2016-08-01</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/065052">Massively multiplex single-cell Hi-C</a></td><td>Vijay Ramani, Xinxian Deng, Kevin L Gunderson, Frank J Steemers, Christine M Disteche, William S Noble, Zhijun Duan, Jay Shendure</td><td>10.1101/065052</td><td>2016-07-23</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/054031">Initiation of mtDNA transcription is followed by pausing, and diverge across human cell types and during evolution</a></td><td>Amit Blumberg, Edward J. Rice, Anshul Kundaje, Charles G. Danko, Dan Mishmar</td><td>10.1101/054031</td><td>2016-05-18</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/024620">Formation of Chromosomal Domains by Loop Extrusion</a></td><td>Geoffrey Fudenberg, Maxim Imakaev, Carolyn Lu, Anton Goloborodko, Nezar Abdennur, Leonid A Mirny</td><td>10.1101/024620</td><td>2016-03-15</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/038281">Compaction and segregation of sister chromatids via active loop extrusion</a></td><td>Anton Goloborodko, Maxim V. Imakaev, John F. Marko, Leonid A. Mirny</td><td>10.1101/038281</td><td>2016-01-30</td></tr>
        <tr><td><a class="ex1" target="_blank" href="https://doi.org/10.1101/036053">RTFBSDB: an integrated framework for transcription factor binding site analysis</a></td><td>Zhong Wang, Andre L Martins, Charles G Danko</td><td>10.1101/036053</td><td>2016-01-06</td></tr>
        
        
        
        
    </table>
</html>

			<!-- InstanceEndEditable -->
          <div id="latestDevelopments"></div>
        </section>
        <section id="sidebar">
          <nav>
            <div> <a href="https://data.4dnucleome.org/" target="_blank">
              <div id="dataPortalSideButton" class="sideButton">Launch Data Portal
                <div class="sideButtonArrow">➧</div>
              </div>
              </a>
              <div class="sideText">The 4D Nucleome Data Portal hosts data generated by the 4DN Network and other
                reference nucleomics data sets, and an expanding tool set for open data processing and visualization. </div>
              <a href="http://vis.nucleome.org/" target="_blank">
              <div id="nucleomeBrowserSideButton" class="sideButton">Nucleome Browser
                <div class="sideButtonArrow">➧</div>
              </div>
              </a>
              <div class="sideText">Nucleome Browser is a multimodal, interactive data visualization and exploration platform to integrate genomic, imaging, and 3D structure data.</div>
              <a href="https://wiki.4dnucleome.org/" target="_blank">
              <div id="wikiSideButton" class="sideButton">Internal Wiki
                <div class="sideButtonArrow">➧</div>
              </div>
              </a>
              <div class="sideText">The 4D Nucleome Network Wiki is an internal collaboration site, accessible to 4DN
                investigators and members only. </div>
              <a href="https://people.4dnucleome.org/" target="_blank">
              <div id="peopleSideButton" class="sideButton">people.4dnucleome
                <div class="sideButtonArrow">➧</div>
              </div>
              </a>
              <div class="sideText"><strong>people.4dnucleome</strong> provides a one-stop directory for all current 4DN members and allows logged
                in 4DN members to manage their and their team members' profiles, participate in the interval working groups, and more. </div>
            </div>
            <div class="sidePanel framedContainer">
              <div class="sidePanelHeader"> Multimedia Library </div>
              <div class="sidePanelBody">
                <div><a href="https://www.youtube.com/channel/UCe6r3u3rmaYAKSEuTLmfFxA" target="_blank"> <img
                        src="Assets/images/icons/youtube_social_icon_red.png" width="24" alt="Youtube icon" />Youtube </a></div>
              </div>
            </div>
            <div class="sidePanel framedContainer">
              <div class="sidePanelHeader"> Stay Connected </div>
              <div class="sidePanelBody">
                <div><a href="https://www.facebook.com/4DNucleome-1840252952876682/" target="_blank"> <img
                        src="Assets/images/icons/f-ogo_RGB_HEX-58.svg" alt="Facebook icon" />Facebook </a></div>
                <div><a href="https://twitter.com/4DNucleome" target="_blank"><img
                        src="Assets/images/icons/Twitter_Logo_Blue.svg" alt="Twitter icon" />Twitter </a></div>
                <div><a href="https://www.linkedin.com/in/4d-nucleome-857952126/" target="_blank"><img
                        src="Assets/images/icons/In-2C-28px-R.png" id="LinkedIn" alt="LinkedIn icon" />LinkedIn </a> </div>
                <div><a href="mailto:oh@4dnucleome.org"><img src="Assets/images/icons/Email-iron-icon.svg"
                        alt="Email icon" />Email </a></div>
              </div>
            </div>
          </nav>
        </section>
      </div>
      <footer>
        <div class="logoes"> <a href="https://www.facebook.com/4DNucleome-1840252952876682/" target="_blank"><img
                src="Assets/images/icons/flogo-HexRBG-Wht-58.svg" alt="" /></a> <a
              href="https://twitter.com/4DNucleome" target="_blank"><img
                src="Assets/images/icons/Twitter_Logo_White.svg" alt="" /></a> <a
              href="mailto:oh@4dnucleome.org"><img src="Assets/images/icons/Email-iron-icon-white.svg" alt="" /></a> <a href="https://www.youtube.com/channel/UCe6r3u3rmaYAKSEuTLmfFxA" target="_blank"> <img
                src="Assets/images/icons/youtube_dark.png" class="youtube" alt="" /></a> <a
              href="https://www.linkedin.com/in/4d-nucleome-857952126/" target="_blank"><img
                src="Assets/images/icons/In-White-28px-R.png" alt="" /></a> </div>
        <div>4D Nucleome Organizational Hub</div>
      </footer>
    </div>
  </div>
</div>
<div class="mainMargin"> 
  <!-- right margin --> 
</div>
</body>
<!-- InstanceEnd --></html>
