<!doctype html><html><!-- InstanceBegin template="/Templates/mainPage.dwt" codeOutsideHTMLIsLocked="false" -->

<head>
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-74740093-2"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag() { dataLayer.push(arguments); }
    gtag('js', new Date());

    gtag('config', 'UA-74740093-2');
  </script>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <!-- InstanceBeginEditable name="doctitle" -->
<title>4DN Preprints</title>
<!-- InstanceEndEditable -->
  <link href="Assets/styles/mainStyle.css" rel="stylesheet" type="text/css">
  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
  <link rel="stylesheet" type="text/css" href="Assets/js/slick/slick.css" />
  <link rel="stylesheet" type="text/css" href="Assets/js/slick/slick-theme.css" />
  <!-- InstanceBeginEditable name="head" -->
<!-- InstanceEndEditable -->
  <script defer type="text/javascript" src="Assets/js/jquery/jquery-3.3.1.min.js"></script>
  <script defer type="text/javascript" src="Assets/js/jquery/jquery-migrate-1.4.1.min.js"></script>
  <script defer type="text/javascript" src="Assets/js/jquery/jquery-migrate-3.0.0.min.js"></script>
  <script defer type="text/javascript" src="Assets/js/slick/slick.min.js"></script>
  <script defer type="text/javascript" src="Assets/js/main.js"></script>
</head>

<body>
  <div class="mainMargin">
    <!-- left margin -->
  </div>
  <div id="columnWrapper">
    <div id="mainwrapper">
      <header class="framedContainer">
        <div id="logo"><a href="index.html"><img src="Assets/images/4dn-logo_1.png" alt="" /></a>
          <div>4DN Web Portal</div>
        </div>
        <div id="headerRight">
          <nav> <span> <a href="https://data.4dnucleome.org/" target="_blank" title="Launch Data Portal">Launch Data
                Portal</a> </span> <span> <a href="mailto:oh@4dnucleome.org" title="Contact us">Contact Us</a> </span>
          </nav>
          <div id="searchInput">
             
		  <script>
			(function() {
			  var cx = '012010507559311424712:8m_5bphxwsi';
			  var gcse = document.createElement('script');
			  gcse.type = 'text/javascript';
			  gcse.async = true;
			  gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
			  var s = document.getElementsByTagName('script')[0];
			  s.parentNode.insertBefore(gcse, s);
			})();
		  </script>
          <gcse:searchbox-only gname="4dnsitewidesearch"></gcse:searchbox-only>
          
          </div>
        </div>
      </header>
      <div id="content" class="framedContainer">
        <nav id="topMenu" tabindex="0">
          <div class="mobileOnly">
            <i class="material-icons" id="navMenuIcon">
              menu
            </i>
            Navigation Menu</div>
          <div id="topMenuContent">
            <div class="topMenuItem"> <a href="index.html">Home</a> </div>
            <div class="topMenuItem"> <a href="https://commonfund.nih.gov/4dnucleome" target="_blank">Overview
				<i class="material-icons openInNewIcons">open_in_new</i></a>
            </div>
            <div class="topMenuItem"> <a href="https://data.4dnucleome.org/" target="_blank">Data
				<i class="material-icons openInNewIcons">open_in_new</i></a>
			</div>
            <div class="topMenuItem"> Resources
              <div class="subMenu">
                <div> <a href="cell-lines.html">Cell Lines</a> </div>
                <div> <a href="protocols.html">Protocols</a> </div>
                <div> <a href="software.html">Software</a> </div>
              </div>
            </div>
            <div class="topMenuItem"> Publications
              <div class="subMenu">
                <div> <a href="https://www.ncbi.nlm.nih.gov/sites/myncbi/1vMpf2j6i3bke/collections/50779401/public/"
                    target="_blank">4DN Publications <i class="material-icons openInNewIcons">open_in_new</i></a> </div>
                <div> <a href="pre-prints.html">4DN Preprints</a> </div>
                <div> Further Reading
                  <div class="subMenu">
                    <div> <a href="https://www.ncbi.nlm.nih.gov/sites/myncbi/1vMpf2j6i3bke/collections/50130949/public/"
                        target="_blank">Genome Interactions <i class="material-icons openInNewIcons">open_in_new</i></a> </div>
                    <div> <a href="https://www.ncbi.nlm.nih.gov/sites/myncbi/1vMpf2j6i3bke/collections/50131107/public/"
                        target="_blank">Data Analysis <i class="material-icons openInNewIcons">open_in_new</i></a> </div>
                  </div>
                </div>
              </div>
            </div>
            <div class="topMenuItem"> <a href="policies.html">Policies</a> </div>
            <div class="topMenuItem"> <a href="outreach.html">Outreach</a> </div>
			<div class="topMenuItem"> 2019 4DN-ASCB Meeting
			  <div class="subMenu">
				<div> <a href="4dn-ascb-meeting-2019.html">Meeting Home</a> </div>
				<div> <a href="4dn-ascb-meeting-2019-speakers.html">Keynote and Invited Speakers</a> </div>
                <div> <a href="4dn-ascb-meeting-2019-hotel-travel.html">Hotel/Travel</a> </div>
                <div> <a href="4dn-ascb-meeting-2019-abstract-guidelines.html">Abstract guidelines</a> </div>
				<div> <a href="https://washington.org/" target="_blank">Explore Washington, DC</a> </div>
                <div> <a href="4dn-ascb-meeting-2019-committee.html">Planning Committee Members</a> </div>
              </div>
				</div>
            <div class="topMenuItem"> Contacts
              <div class="subMenu">
                <div> <a href="centers-and-teams.html">4DN Centers and Teams</a> </div>
                <div> <a href="TCPA-members.html">TCPA Awardees</a> </div>
                <div> <a href="associate-members.html">Associate Members</a> </div>
                <div> <a href="external-program-consultants.html">External Program Consultants</a> </div>
              </div>
            </div>
          </div>
        </nav>
        <div id="middleContainer">
          <section id="mainContent">
            <!-- InstanceBeginEditable name="Main contents" -->
			<h2>4DN pre-prints</h2>
			<b>See the <a target="_blank" href="http://connect.biorxiv.org/relate/content/66">4DN bioRxiv channel.</a></b>
			
			<br/>
			<br/>
			<br/>
			<table id="GeneralTables" style="width:100%">
        
        <tr id="Header1">
            <th colspan="2">Pre-print publications funded by NIH 4DN program</th>
        </tr>
        
        <tr id="Header2">
            <th width="80%">Pre-print Publication</th>
            <th width="20%">Posted date</th>
        </tr>
        
		<tr>
			<td><b>A chromosome folding intermediate at the condensin-to-cohesin transition during telophase</b>, Kristin Abramo, Anne-Laure Valton, Sergey V Venev, Hakan Ozadam, Amy N Fox, Job Dekker, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/678474">https://doi.org/10.1101/678474</a></td>
			<td>Jun 21, 2019</td>
		</tr>
		
		<tr>
			<td><b>HiNT: a computational method for detecting copy number variations and translocations from Hi-C data</b>, Su Wang, Soohyun Lee, Chong Chu, Dhawal Jain, Geoff Nelson, Jennifer M. Walsh, Burak H. Alver, Peter J. Park, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/657080">https://doi.org/10.1101/657080</a></td>
			<td>Jun 3, 2019</td>
		</tr>
		
		<tr>
			<td><b>Jointly embedding multiple single-cell omics measurements</b>, Jie Liu, Yuanhao Huang, Ritambhara Singh, Jean-Philippe Vert, William Stafford Noble, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/644310">https://doi.org/10.1101/644310</a></td>
			<td>May 21, 2019</td>
		</tr>
		
		<tr>
			<td><b>Spring Model – chromatin modeling tool based on OpenMM</b>, Michal Kadlof, Julia Rozycka, Dariusz Plewczynski, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/642322">https://doi.org/10.1101/642322</a></td>
			<td>May 20, 2019</td>
		</tr>
		
		<tr>
			<td><b>Ultrastructural details of mammalian chromosome architecture</b>, Nils Krietenstein, Sameer Abraham, Sergey V. Venev, Nezar Abdennur, Johan Gibcus, Tsung-Han S. Hsieh, Krishna Mohan Parsi, Liyan Yang, René Maehr, Leonid A. Mirny, Job Dekker, Oliver J. Rando, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/639922">https://doi.org/10.1101/639922</a></td>
			<td>May 17, 2019</td>
		</tr>
		
		<tr>
		<td><b>Super resolution imaging of a distinct chromatin loop in human lymphoblastoid cells</b>, Jacqueline Jufen Zhu, Zofia Parteka, Byoungkoo Lee, Przemyslaw Szalaj, Ping Wang, Karolina Jodkowska, Jesse Aaron, Teng-Leong Chew, Dariusz Plewczynski, Yijun Ruan, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/621920">https://doi.org/10.1101/621920</a></td>
		<td>April 29, 2019</td>
		</tr>
		
		<tr>
			<td><b>iMARGI: Mapping RNA-chromatin interactome by sequencing</b>, Weixin Wu, Zhangming Yan, Zhen Chen, Shu Chien, Sheng Zhong, doi: <a class="ex1" target="_blank" href="https://dx.doi.org/10.17504/protocols.io.w75fhq6">https://dx.doi.org/10.17504/protocols.io.w75fhq6</a></td>
			<td>April 18, 2019</td>
		</tr>
		
		<tr>
			<td><b>ChIA-DropBox: a novel analysis and visualization pipeline for multiplex chromatin interactions</b>, Simon Zhongyuan Tian, Daniel Capurso, Minji Kim, Byoungkoo Lee, Meizhen Zheng, Yijun Ruan, doi: <a class="ex1" target="_blank" href=" https://doi.org/10.1101/613034"> https://doi.org/10.1101/613034</a></td>
			<td>April 18, 2019</td>
		</tr>
		
		<tr>
			<td><b>RNA polymerases as moving barriers to condensin loop extrusion</b>, Hugo B. Brandão, Xindan Wang, Payel Paul, Aafke A. van den Berg, David Z. Rudner,Leonid A. Mirny, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/604280">https://doi.org/10.1101/604280</a></td>
			<td>April 10, 2019</td>
		</tr>
		
		<tr>
			<td><b>Transcription amplification by nuclear speckle association</b>, Jiah Kim, Nimish Khanna, Andrew S. Belmont, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/604298">https://doi.org/10.1101/604298</a></td>
			<td>April 9, 2019</td>
		</tr>
		
		<tr>
			<td><b>Peax: Interactive Visual Pattern Search in Sequential Data Using Unsupervised Deep Representation Learning</b>, Fritz Lekschas, Brant Peterson, Daniel Haehn, Eric Ma, Nils Gehlenborg, Hanspeter Pfister, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/597518">https://doi.org/10.1101/597518</a></td>
			<td>April 4, 2019</td>
		</tr>
		
		<tr>
			<td><b>Common DNA sequence variation influences 3-dimensional conformation of the human genome</b>, David U. Gorkin, Yunjiang Qiu, Ming Hu, Kipper Fletez-Brant, Tristin Liu, Anthony D. Schmitt, Amina Noor, Joshua Chiou, Kyle J Gaulton, Jonathan Sebat, Yun Li, Kasper D. Hansen, Bing Ren, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/592741">https://doi.org/10.1101/592741</a></td>
			<td>March 30, 2019</td>
		</tr>
		
		<tr>
			<td><b>Direct visualization of single nuclear pore complex proteins using genetically-encoded probes for DNA-PAINT</b>, Thomas Schlichthaerle, Maximilian T. Strauss, Florian Schueder, Alexander Auer, Bianca Nijmeijer, Moritz Kueblbeck, Vilma Jimenez Sabinina, Jervis V. Thevathasan, Jonas Ries,  Jan Ellenberg, Ralf Jungmann, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/579961">https://doi.org/10.1101/579961</a></td>
			<td>March 23, 2019</td>
		</tr>
		
		<tr>
			<td><b>Nuclear pores as versatile reference standards for quantitative superresolution microscopy</b>, Jervis Vermal Thevathasan, Maurice Kahnwald, Konstanty Cieśliński, Philipp Hoess, Sudheer Kumar Peneti, Manuel Reitberger, Daniel Heid, Krishna Chaitanya Kasuba, Sarah Janice Hoerner, Yiming Li, Yu-Le Wu, Markus Mund, Ulf Matti, Pedro Matos Pereira, Ricardo Henriques, Bianca Nijmeijer, Moritz Kueblbeck, Vilma Jimenez Sabinina, Jan Ellenberg, Jonas Ries, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/582668">https://doi.org/10.1101/582668</a></td>
			<td>March 20, 2019</td>
		</tr>
		
        <tr>
            <td><b>Absolute quantification of cohesin, CTCF and their regulators in human cells</b>, Johann Holzmann, Antonio Z. Politi, Kota Nagasaka, Merle Hantsche-Grininger, Nike Walther, Birgit Koch, Johannes Fuchs, Gerhard Dürnberger, Wen Tang, Rene Ladurner, Roman R. Stocsits, Georg A. Busslinger, Béla Novák, Karl Mechtler, Iain F. Davidson, Jan Ellenberg, Jan-Michael Peters, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/560425">https://doi.org/10.1101/560425</a></td>
            <td>February 27, 2019</td>
        </tr>
        
        <tr>
            <td><b>EpiAlignment: alignment with both DNA sequence and epigenomic data</b>, Jia Lu, Xiaoyi Cao, Sheng Zhong, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/562199">https://doi.org/10.1101/562199</a></td>
            <td>February 27, 2019</td>
        </tr>
        
        <tr>
            <td><b>Determining cellular CTCF and cohesin abundances to constrain 3D genome models</b>, Claudia Cattoglio, Iryna Pustova, Nike Walther, Jaclyn J Ho, Merle Hantsche-Grininger, Carla J Inouye, M. Julius Hossain, Gina M Dailey, Jan Ellenberg, Xavier Darzacq, Robert Tjian, Anders S Hansen, doi: <a class="ex1" target="_blank"
                href=" https://doi.org/10.1101/370650">https://doi.org/10.1101/370650</a></td>
            <td>February 24, 2019</td>
        </tr>

        <tr>
            <td><b>Cooler: scalable storage for Hi-C data and other genomically-labeled arrays</b>, Nezar Abdennur, Leonid Mirny, doi: <a class="ex1" target="_blank"
                href="https://doi.org/10.1101/557660">https://doi.org/10.1101/557660</a></td>
            <td>February 22, 2019</td>
        </tr>
        
        <tr>
            <td><b>Localization microscopy at doubled precision with patterned illumination</b>, Jelmer Cnossen, Taylor Hinsdale, Ramsus Thorsen, Florian Schueder, Ralf Jungmann, Carlas S Smith, Bernd Rieger, Sjoerd Stallinga, doi: <a class="ex1" target="_blank"
                href="https://doi.org/10.1101/554337">https://doi.org/10.1101/554337</a></td>
            <td>February 20, 2019</td>
        </tr>
        
        <tr>
            <td><b>Chromatin compartment dynamics in a haploinsufficient model of cardiac laminopathy</b>, Alessandro Bertero, Paul A Fields, Alec ST Smith, Andrea Leonard, Kevin Beussman, Nathan J Sniadecki, Deok-Ho Kim, Hung-Fat Tse, Lil Pabon, Jay Shendure, William S Noble,  Charles E Murry, doi: <a class="ex1" target="_blank"
                href="https://doi.org/10.1101/555250">https://doi.org/10.1101/555250</a></td>
            <td>February 19, 2019</td>
        </tr>
        
        <tr>
            <td><b>Comparing 3D genome organization in multiple species using Phylo-HMRF</b>, Yang Yang, Yang Zhang, Bing Ren, Jesse Dixon, Jian Ma, doi: <a class="ex1" target="_blank"
                href="https://doi.org/10.1101/552505">https://doi.org/10.1101/552505</a></td>
            <td>February 19, 2019</td>
        </tr>
        
        <tr>
            <td><b>MOCHI enables discovery of heterogeneous interactome modules in 3D nucleome</b>, Dechao Tian, Ruochi Zhang, Yang Zhang,  Xiaopeng Zhu, Jian Ma, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/542092">https://doi.org/10.1101/542092</a></td>
            <td>February 7, 2018</td>
        </tr>
        
        <tr>
            <td><b>Revealing Hi-C subcompartments by imputing high-resolution inter-chromosomal chromatin interactions</b>, Kyle Xiong, Jian Ma, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/505503">https://doi.org/10.1101/505503</a></td>
            <td>December 23, 2018</td>
        </tr>
        
        <tr>
            <td><b>An RNA-binding region regulates CTCF clustering and chromatin looping</b>, Anders S Hansen, Tsung-Han S Hsieh, Claudia Cattoglio, Iryna Pustova, Xavier Darzacq, Robert Tjian, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/49543">https://doi.org/10.1101/49543</a></td>
            <td>December 16, 2018</td>
        </tr>
        
        <tr>
            <td><b>Guided nuclear exploration increases CTCF target search efficiency</b>, Anders S Hansen, Assaf Amitai, Claudia Cattoglio, Robert Tjian, Xavier Darzacq, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/495457">https://doi.org/10.1101/495457</a></td>
            <td>December 13, 2018</td>
        </tr>
				
                
		<tr>
            <td><b>Co-opted transposons help perpetuate conserved higher-order chromosomal structures</b>, Mayank NK Choudhary, Ryan Z Friedman, Julia T Wang, Hyo Sik Jang, Xiaoyu Zhuo, Ting Wang, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/485342">https://doi.org/10.1101/485342</a></td>
            <td>December 5, 2018</td>
        </tr>
			
		<tr>
			<td><b>Identification of new transmembrane proteins concentrated at the nuclear envelope using organellar proteomics of mesenchymal cells</b>, Li-Chun Cheng, Sabyasachi Baboo, Cory Lindsay, Liza Brusman, Salvador Martinez-Bartolomé, Olga Tapia, Xi Zhang, John R. Yates III, Larry Gerace, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/486415">https://doi.org/10.1101/486415</a></td>
			<td>December 3, 2018</td>
		</tr>
			
        <tr>
            <td><b>Two Contrasting Classes of Nucleolus-Associated Domains in Mouse Fibroblast Heterochromatin</b>, Anastassiia Vertii, Jianhong Ou, Jun Yu, Aimin Yan, Hervé Pagès, Haibo Liu, Lihua Julie Zhu, Paul D. Kaufman, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/484568">https://doi.org/10.1101/484568</a></td>
            <td>December 3, 2018</td>
        </tr>
        
        <tr>
            <td><b>Genome-wide co-localization of RNA-DNA interactions and fusion RNA pairs</b>, Yan, Z., Huang, N., Wu, W., Chen, W., Jiang, Y., Chen, J., Huang, X., Wen, X., Xu, J., Jin, Q., Zhang, K., Chen, Z., Chien, S., Zhong, S., doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/472019">https://doi.org/10.1101/472019</a></td>
            <td>November 20, 2018</td>
        </tr>
        
        <tr>
            <td><b>Spatial Chromatin Architecture Alteration by Structural Variations in Human Genomes at Population Scale</b>, Sadowski, M., Kraft, A., Szalaj, P., Wlasnowolski, M., Tang, Z., Ruan, Y., Plewczynski, D., doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/266981">https://doi.org/10.1101/266981</a></td>
            <td>September 14, 2018</td>
        </tr>
        
        <tr>
            <td><b>Photon count estimation in single-molecule localization microscopy</b>, Rieger, B., Stallinga, S., Hulleman, C. N., Thorsen, R. O., Grunwald, D., Hammer, M., doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/396424">https://doi.org/10.1101/396424</a></td>
            <td>August 20, 2018</td>
        </tr>
        
        <tr>
            <td><b>A combination of transcription factors mediates inducible interchromosomal pairing</b>, Kim, S., Dunham, M., Shendure, J., doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/385047">https://doi.org/10.1101/385047</a></td>
            <td>August 5, 2018</td>
        </tr>
        
        <tr>
            <td><b>Nuclear speckle fusion via long-range directional motion regulates the number and size of speckles</b>, Jiah Kim, Kyu Young Han, Nimish Khanna, Taekjip Ha, Andrew S. Belmont, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/347955">https://doi.org/10.1101/347955</a></td>
            <td>June 15, 2018</td>
        </tr>
        
        <tr>
            <td><b>RNAs as proximity labeling media for identifying nuclear speckle positions relative to the genome</b>, Weizhong Chen, Zhangming Yan, Simin Li, Norman Huang, Xuerui Huang, Jin Zhang, Sheng Zhong, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/300483">https://doi.org/10.1101/300483</a></td>
            <td>April 12, 2018</td>
        </tr>
        
        <tr>
            <td><b>Identification of cis elements for spatio-temporal control of DNA replication</b>, Jiao Sima, Abhijit Chakraborty, Vishnu Dileep, Marco Michalski, Juan Carlos Rivera-Mulia, Claudia Trevilla-Garcia, Kyle N Klein, Daniel Bartlett, Brian K Washburn, Michelle T Paulsen, Daniel Vera, Elphège P Nora, Katerina Kraft, Stefan Mundlos, Benoit G Bruneau, Mats Ljungman, Peter Fraser, Ferhat Ay, David M Gilbert, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/285650">https://doi.org/10.1101/285650</a></td>
            <td>March 21, 2018</td>
        </tr>
        
        <tr>
            <td><b>Predicting Enhancer-Promoter Interaction from Genomic Sequence with Deep Neural Networks</b>, Singh, S., Yang, Y., Poczos, B., Ma, J., doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/085241">https://doi.org/10.1101/085241</a></td>
            <td>February 5, 2018</td>
        </tr>
        
        <tr>
            <td><b>GITAR: An open source tool for analysis and visualization of Hi-C data</b>, Calandrelli, R., Wu, Q., Guan, J., Zhong, S., doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/259515">https://https://doi.org/10.1101/259515</a></td>
            <td>February 4, 2018</td>
        </tr>
        
        <tr>
            <td><b>C-BERST: Defining subnuclear proteomic landscapes at genomic elements with dCas9-APEX2</b>, Gao, X. D., Tu, L.-C., Mir, A., Rodriguez, T., Ding, Y., Leszyk, J., Dekker, J., Shaffer, S. A., Zhu, L. J., Wolfe, S. A., Sontheimer, E. J., doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/171819">https://doi.org/10.1101/171819</a></td>
            <td>January 31, 2018</td>
        </tr>
        
        <tr>
            <td><b>Correlative live and super-resolution imaging reveals the dynamic structure of replication domains</b>, Xiang, W., Roberti, M. J., Heriche, J.-K., Huet, S., Alexander, S., Ellenberg, J., doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/189373">https://doi.org/10.1101/189373</a></td>
            <td>January 24, 2018</td>
        </tr>
        
        <tr>
            <td><b>Building a genome browser with GIVE</b>, Cao, X., Yan, Z., Wu, Q., Zheng, A., Zhong, S., doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/177832">https://doi.org/10.1101/177832</a></td>
            <td>January 13, 2018</td>
        </tr>
        
        <tr>
            <td><b>Orientation-dependent Dxz4 contacts shape the 3D structure of the inactive X chromosome</b>, Bonora, G., Deng, X., Fang, H., Ramani, V., Qui, R., Berletch, J., Filippova, G. N., Duan, Z., Schendure, J., Noble, W. S., Disteche, C. M., doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/165340">https://doi.org/10.1101/165340</a></td>
            <td>December 15, 2017</td>
        </tr>
        
        <tr>
            <td><b>Allele-specific control of replication timing and genome organization during development</b>, Juan Carlos Rivera-Mulia, Andrew Dimond, Daniel Vera, Claudia Trevilla-Garcia, Takayo Sasaki, Jared Zimmerman, Catherine Dupont, Joost Gribnau, Peter Fraser, David M Gilbert, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/221762">https://doi.org/10.1101/221762</a></td>
            <td>November 21, 2017</td>
        </tr>
        
        <tr>
            <td><b>Single-molecule fluorescent amplification of RNA using clampFISH probes</b>, Sara H Rouhanifard, Margaret Dunagin, Ian A Mellis, Sareh Bayatpour, Orsolya Symmons, Allison Cote, Arjun Raj, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/222794">https://doi.org/10.1101/222794</a></td>
            <td>November 21, 2017</td>
        </tr>
        
        <tr>
            <td><b>Dynamic reorganization of nuclear architecture during human cardiogenesis</b>, Paul A Fields, Vijay Ramani, Giancarlo Bonora, Galip Gurkan Yardimci, Alessandro Bertero, Hans Reinecke, Lil Pabon, William S Noble, Jay Shendure, Charles Murry, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/222877">https://doi.org/10.1101/222877</a></td>
            <td>November 21, 2017</td>
        </tr>
        
        <tr>
            <td><b>Higher-order inter-chromosomal hubs shape 3-dimensional genome organization in the nucleus</b>, Sofia A Quinodoz, Noah Ollikainen, Barbara Tabak, Ali Palla, Jan Marten Schmidt, Elizabeth Detmar, Mason Lai, Alexander Shishkin, Prashant Bhat, Vickie Trinh, Erik Aznauryan, Pamela Russell, Christine Cheng, Marko Jovanovic, Amy Chow, Patrick McDonel, Manuel Garber, Mitchell Guttman, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/219683">https://doi.org/10.1101/219683</a></td>
            <td>November 18, 2017</td>
        </tr>
        
        <tr>
            <td><b>Quantitative imaging of chromatin decompaction in living cells</b>, Elisa Dultz, Roberta Mancini, Guido Polles, Pascal Vallotton, Frank Alber, Karsten Weis, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/219253">https://doi.org/10.1101/219253</a></td>
            <td>November 15, 2017</td>
        </tr>
        
        <tr>
            <td><b>CHRAC/ACF Contribute to the Repressive Ground State of Chromatin</b>, Alessandro Scacchetti, Laura Brueckner, Dhawal Jain, Tamas Schauer, Xu Zhang, Frank Schnorrer, Bas van Steensel, Tobias Straub, Peter B. Becker, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/218768">https://doi.org/10.1101/218768</a></td>
            <td>November 13, 2017</td>
        </tr>
        
        <tr>
            <td><b>A Cell Type-Specific Class of Chromatin Loops Anchored at Large DNA Methylation Nadirs</b>, Mira Jeong, Xingfan Huang, Xiaotian Zhang, Jianzhong Su, Muhammad Shamim, Ivan Bochkov, Jaime Reyes, Haiyoung Jung, Emily Heikamp, Aviva Presser Aiden, Wei Li, Erez Aiden, Margaret A. Goodell, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/212928">https://doi.org/10.1101/212928</a></td>
            <td>November 9, 2017</td>
        </tr>
        
        <tr>
            <td><b>Removing unwanted variation between samples in Hi-C experiments</b>, Kipper Fletez-Brant, Yunjiang Qiu, David U Gorkin, Ming Hu, Kasper D Hansen, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/214361">https://doi.org/10.1101/214361</a></td>
            <td>November 6, 2017</td>
        </tr>
        
        <tr>
            <td><b>HiGlass: Web-based Visual Comparison And Exploration Of Genome Interaction Maps</b>, Peter Kerpedjiev, Nezar Abdennur, Fritz Lekschas, Chuck McCallum, Kasper Dinkla, Hendrik Strobelt, Jacob M Luber, Scott B Ouellette, Alaleh Ahzir, Nikhil Kumar, Jeewon Hwang, Burak H Alver, Hanspeter Pfister, Leonid A Mirny, Peter J Park, Nils Gehlenborg, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/121889">https://doi.org/10.1101/121889</a></td>
            <td>October 30, 2017</td>
        </tr>
        
        <tr>
            <td><b>Locus-Specific Enhancer Hubs And Architectural Loop Collisions Uncovered From Single Allele DNA Topologies</b>, Amin Allahyar, Carlo Vermeulen, Britta Bouwman, Peter Krijger, Marjon Verstegen, Geert Geeven, Melissa van Kranenburg, Mark Pieterse, Roy Straver, Judith Haarhuis, Hans Teunissen, Ivo Renkens, Wigard Kloosterman, Benjamin Rowland, Elzo de Wit, Jeroen de Ridder, Wouter de Laat, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/206094">https://doi.org/10.1101/206094</a></td>
            <td>October 20, 2017</td>
        </tr>
        
        <tr>
            <td><b>Juicebox.js provides a cloud-based visualization system for Hi-C data</b>, James Robinson, Douglass Turner, Neva C. Durand, Helga Thorvaldsdottir, Jill P. Mesirov, Erez Lieberman Aiden, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/205740">https://doi.org/10.1101/205740</a></td>
            <td>October 19, 2017</td>
        </tr>
        
        <tr>
            <td><b>Fluorescence polarization control for on-off switching of single molecules at cryogenic temperatures</b>, Christiaan Hulleman, Maximiliaan Huisman, Robert Moerland, David Grunwald, Sjoerd Stallinga, Bernd Rieger, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/204776">https://doi.org/10.1101/204776</a></td>
            <td>October 17, 2017</td>
        </tr>
        
        <tr>
            <td><b>Genomic binding dynamics of PBAF are regulated via select chromatin states</b>, Charles A Kenworthy, Vincent Wong, Patrycja Dziuba, Luke D Lavis, Wei-Li Liu, Robert H Singer, Robert A Coleman, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/111674">https://doi.org/10.1101/111674</a></td>
            <td>October 4, 2017</td>
        </tr>
        
        <tr>
            <td><b>Chromatin Organization by an Interplay of Loop Extrusion and Compartmental Segregation</b>, Johannes Nuebler, Geoffrey Fudenberg, Maxim Imakaev, Nezar Abdennur, Leonid Mirny, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/196261">https://doi.org/10.1101/196261</a></td>
            <td>October 3, 2017</td>
        </tr>
        
        <tr>
            <td><b>ChromoTrace: Reconstruction of 3D Chromosome Configurations by Super-Resolution Microscopy</b>, Sandro Morganella, Oeyvind Oedegaard, Stephanie Alexander, Jonas Ries, Tomas Fitzgerald, Jan Ellenberg, Ewan Birney, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/115436">https://doi.org/10.1101/115436</a></td>
            <td>September 28, 2017</td>
        </tr>
        
        <tr>
            <td><b>CRISPR-Based DNA Imaging in Living Cells Reveals Cell Cycle-Dependent Chromosome Dynamics</b>, Hanhui Ma, Li-Chun Tu, Ardalan Naseri, Yu-Chieh Chung, David Grunwald, Shaojie Zhang, Thoru Pederson, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/195966">https://doi.org/10.1101/195966</a></td>
            <td>September 29, 2017</td>
        </tr>
        
        <tr>
            <td><b>Genetic drivers of repeat expansion disorders localize to 3-D chromatin domain boundaries</b>, James Sun, Linda Zhou, Daniel J Emerson, Thomas G. Gilgenast, Katelyn Titus, Jonathan A. Beagan, Jennifer E. Phillips-Cremins, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/191213">https://doi.org/10.1101/191213</a></td>
            <td>September 20, 2017</td>
        </tr>
        
        <tr>
            <td><b>Correlative live and super-resolution imaging reveals the dynamic structure of replication domains</b>, Wanqing Xiang, M. Julia Roberti, Jean-Karim Heriche, Sebastian Huet, Stephanie Alexander, Jan Ellenberg, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/189373">https://doi.org/10.1101/189373</a></td>
            <td>September 17, 2017</td>
        </tr>

        
        <tr>
            <td><b>Measuring the reproducibility and quality of Hi-C data</b>, Galip Yardimci, Hakan Ozadam, Michael E.G. Sauria, Oana Ursu, Koon-Kiu Yan, Tao Yang, Abhijit Chakraborty, Arya Kaul, Bryan R Lajoie, Fan Song, Ye Zhan, Ferhat Ay, Mark Gerstein, Anshul Kundaje, Qunhua Li, James Taylor, Feng Yue, Job Dekker, William S Noble, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/188755">https://doi.org/10.1101/188755</a></td>
            <td>September 14, 2017</td>
        </tr>
        
        <tr>
            <td><b>SHAMAN: bin-free randomization, normalization and screening of Hi-C matrices</b>, Netta Mendelson Cohen, Pedro Olivares-Chauvet, Yaniv Lubling, Yael Baran, Aviezer Lifshitz, Michael Hoichman, Amos Tanay, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/187203">https://doi.org/10.1101/187203</a></td>
            <td>September 12, 2017</td>
        </tr>
    
        
        <tr>
            <td><b>Fast, robust and precise 3D localization for arbitrary point spread functions</b>, Yiming Li, Markus Mund, Philipp Hoess, Ulf Matti, Bianca Nijmeijer, Vilma Jimenez Sabinina, Jan Ellenberg, Ingmar Schoen,  Jonas Ries, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/172643">https://doi.org/10.1101/172643</a></td>
            <td>August 10, 2017</td>
        </tr>
        
        <tr>
            <td><b>Mitotic chromosomes fold by condensin-dependent helical winding of chromatin loop arrays</b>, Johan H. Gibcus, Kumiko Samejima, Anton Goloborodko, Itaru Samejima, Natalia Naumova, Masato Kanemaki, Linfeng Xie, James R. Paulson, William C. Earnshaw, Leonid A. Mirny, Job Dekker, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/174649">https://doi.org/10.1101/174649</a></td>
            <td>August 10, 2017</td>
        </tr>
        
        <tr>
            <td><b>Heterogeneity and Intrinsic Variation in Spatial Genome Organization</b>, Elizabeth Finn, Gianluca Pegoraro, Hugo B. Brandao, Anne-Laure Valton, Marlies E. Oomen, Job Dekker, Leonid Mirny, Tom Misteli, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/171801">https://doi.org/10.1101/171801</a></td>
            <td>August 3, 2017</td>
        </tr>
        
        <tr>
            <td><b>C-BERST: Defining subnuclear proteomic landscapes at genomic elements with dCas9-APEX2</b>, Xin D. Gao, Li-Chun Tu, Aamir Mir, Job Dekker,  Scott A. Shaffer, Lihua Julie Zhu, Scot A. Wolfe, Erik J. Sontheimer, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/171819">https://doi.org/10.1101/171819</a></td>
            <td>August 2, 2017</td>
        </tr>
        
        <tr>
            <td><b>Orientation-dependent Dxz4 contacts shape the 3D structure of the inactive X chromosome</b>, Giancarlo Bonora, Xinxian Deng, He Fang, Vijay Ramani, Ruolan Qui, Joel Berletch, Gala N Filippova, Zhijun Duan, Jay Schendure, William S Noble, Christine M Disteche, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/165340">https://doi.org/10.1101/165340</a></td>
            <td>July 19, 2017</td>
        </tr>
        
        <tr>
            <td><b>Natural Selection has Shaped Coding and Non-coding Transcription in Primate CD4+ T-cells</b>, Charles G. Danko, Lauren A. Choate, Brooke A. Marks, Edward J. Rice, Zhong Wang, Tinyi Chu, Andre L. Martins, Noah Dukler, Scott A. Coonrod, Elia D. Tait Wojno, John T. Lis, W. Lee Kraus, Adam Siepel, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/083212">https://doi.org/10.1101/083212</a></td>
            <td>July 16, 2017</td>
        </tr>
        
        <tr>
            <td><b>Single-cell replication profiling reveals stochastic regulation of the mammalian replication-timing program</b>, Vishnu Dileep, David M. Gilbert, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/158352">https://doi.org/10.1101/158352</a></td>
            <td>July 14, 2017</td>
        </tr>
        
        <tr>
            <td><b>CRISPR/Cas9-mediated Knock-in of an Optimized TetO Repeat for Live Cell Imaging of Endogenous Loci</b>, Ipek Tasan, Gabriela Sustackova, Liguo Zhang, Jiah Kim, Mayandi Sivaguru, Mohammad HamediRad, Yuchuan Wang, Justin Genova, Jian Ma, Andrew S. Belmont, Huimin Zhao, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/162156">https://doi.org/10.1101/162156</a></td>
            <td>July 12, 2017</td>
        </tr>
        
        <tr>
            <td><b>HiPiler: Visual Exploration Of Large Genome Interaction Matrices With Interactive Small Multiples</b>, Fritz Lekschas, Benjamin Bach, Peter Kerpedjiev, Nils Gehlenborg, Hanspeter Pfister, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/123588">https://doi.org/10.1101/123588</a></td>
            <td>July 9, 2017</td>
        </tr>
        
        <tr>
            <td><b>A Fluorogenic Array Tag for Temporally Unlimited Single Molecule Tracking</b>, Rajarshi P Ghosh, Matthew J Franklin, Will E Draper, Quanming Shi, Jan T Liphardt, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/159004">https://doi.org/10.1101/159004</a></td>
            <td>July 3, 2017</td>
        </tr>
        
        <tr>
            <td><b>Chromatin accessibility dynamics of myogenesis at single cell resolution</b>, Hannah Pliner, Jonathan Packer, Jose McFaline-Figueroa, Darren Cusanovich, Riza Daza, Sanjay Srivatsan, Xiaojie Qiu, Dana Jackson, Anna Minkina, Andrew Adey, Frank Steemers, Jay Shendure, Cole Trapnell, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/155473">https://doi.org/10.1101/155473</a></td>
            <td>June 26, 2017</td>
        </tr>
        
        <tr>
            <td><b>Single-molecule FISH in Drosophila muscle reveals location dependent mRNA composition of megaRNPs</b>, Akiko Noma, Carlas S. Smith, Maximillian Huisman, Robert M. Martin, Melissa J. Moore, David Grunwald, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/156091">https://doi.org/10.1101/156091</a></td>
            <td>June 26, 2017</td>
        </tr>
        
        <tr>
            <td><b>qSR: A software for quantitative analysis of single molecule and super-resolution data</b>, J. Owen Andrews, Arjun Narayanan, Jan-Hendrik Spille, Won-Ki Cho, Jesse D. Thaler, Ibrahim I. Cisse, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/146241">https://doi.org/10.1101/146241</a></td>
            <td>June 5, 2017</td>
        </tr>
        
        <tr>
            <td><b>Reconstructing Spatial Transport Distributions In The Nuclear Pore Complex From 2D Images — How Reliable Is It?</b>, Li-Chun Tu, Maximiliaan Huisman, Yu-Chieh Chung, Carlas Smith, David Grunwald, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/145110">https://doi.org/10.1101/145110</a></td>
            <td>June 2, 2017</td>
        </tr>
        
        <tr>
            <td><b>Specific Virus-Host Genome Interactions Revealed By Tethered Chromosome Conformation Capture</b>, Haochen Li, Reza Kalhor, Bing Li, Trent Su, Arnold Berk, Siavash Kurdistani, Frank Alber, Lin Chen, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/142604">https://doi.org/10.1101/142604</a></td>
            <td>May 26, 2017</td>
        </tr>
        
        <tr>
            <td><b>A Comparison Between Single Cell RNA Sequencing And Single Molecule RNA FISH For Rare Cell Analysis</b>, Eduardo A. Torre, Hannah Dueck, Sydney Shaffer, Janko Gospocic, Rohit Gupte, Roberto Bonasio, Junhyong Kim, John Murray, Arjun Raj, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/138289">https://doi.org/10.1101/138289</a></td>
            <td>May 18, 2017</td>
        </tr>
        
        <tr>
            <td><b>Gene Expression Recovery For Single Cell RNA Sequencing</b>, o Huang, Jingshu Wang, Eduardo Torre, Hannah Dueck, Sydney Shaffer, Roberto Bonasio, John Murray, Arjun Raj, Mingyao Li, Nancy R. Zhang, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/138677">https://doi.org/10.1101/138677</a></td>
            <td>May 17, 2017</td>
        </tr>
        
        <tr>
            <td><b>An Integrative Framework For Detecting Structural Variations In Cancer Genomes</b>, Jesse Dixon, Jie Xu, Vishnu Dileep, Ye Zhan, Fan Song, Victoria T. Le, Galip Gurkan Yardimci, Abhijit Chakraborty, Darrin V. Bann, Yanli Wang, Royden Clark, Lijun Zhang, Hongbo Yang, Tingting Liu, Sriranga Iyyanki, Lin An, Christopher Pool, Takayo Sasaki, Juan Carlos Rivera Mulia, Hakan Ozadam, Bryan R. Lajoie, Rajinder Kaul, Michael Buckley, Kristen Lee, Morgan Diegel, Dubravka Pezic, Christina Ernst, Suzana Hadjur, Duncan T. Odom, John A. Stamatoyannopoulos, James R. Broach, Ross Hardison, Ferhat Ay, William Stafford Noble, Job Dekker, David M Gilbert, Feng Yue, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/119651">https://doi.org/10.1101/119651</a></td>
            <td>March 28, 2017</td>
        </tr>
        
        <tr>
            <td><b>Preservation of Chromatin Organization after Acute Loss of CTCF in Mouse Embryonic Stem Cells</b>, Naoki Kubo, Haruhiko Ishii, David Gorkin, Franz Meitinger, Xiong Xiong, Rongxin Fang, Tristin Liu, Zhen Ye, Bin Li, Jesse Dixon, Arshad Desai, Huimin Zhao, Bing Ren, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/118737">https://doi.org/10.1101/118737</a></td>
            <td>March 20, 2017</td>
        </tr>
        
        <tr>
            <td><b>Free energy based high-resolution modeling of CTCF-mediated chromatin loops for human genome</b>, Wayne Dawson, Dariusz Plewczynski, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/105676">https://doi.org/10.1101/105676</a></td>
            <td>March 8, 2017</td>
        </tr>
        
        <tr>
            <td><b>HiCPlus: Resolution Enhancement of Hi-C interaction heatmap</b>, Yan Zhang, Lin An, Ming Hu, Jijun Tang, Feng Yue, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/112631">https://doi.org/10.1101/112631</a></td>
            <td>March 1, 2017</td>
        </tr>
        
        <tr>
            <td><b>Repli-seq: genome-wide analysis of replication timing by next-generation sequencing</b>, Claire Marchal, Takayo Sasaki, Daniel Vera, Korey Wilson, Jiao Sima, Juan-Carlos Rivera-Mulia, Claudia Trevilla Garcia, Coralin Nogues, Ebtesam Nafie, David M Gilbert, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/104653">https://doi.org/10.1101/104653</a></td>
            <td>March 1, 2017</td>
        </tr>
        
        <tr>
            <td><b>The 3D Genome Browser: a web-based browser for visualizing 3D genome organization and long-range chromatin interactions</b>, Yanli Wang, Bo Zhang, Lijun Zhang, Lin An, Jie Xu, Daofeng Li, Mayank NK Choudhary, Yun Li, Ming Hu, Ross Hardison, Ting Wang, Feng Yue, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/112268">https://doi.org/10.1101/112268</a></td>
            <td>February 27, 2017</td>
        </tr>
        
        <tr>
            <td><b>A fluorogenic nanobody array tag for prolonged single molecule imaging in live cells</b>, Rajarshi Ghosh, Will Draper, J. Matthew Franklin, Quanming Shi, Jan Liphardt, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/111690">https://doi.org/10.1101/111690</a></td>
            <td>February 27, 2017</td>
        </tr>
        
        <tr>
            <td><b>Real-time chromatin dynamics at the single gene level during transcription activation</b>, Thomas Germier, Silvia Kocanova, Nike Walther, Aurelien Bancaud, Haitham Ahmed Shaban, Hafida Sellou, Antonio Politi, Jan Ellenberg, Franck Gallardo, Kerstin Bystricky, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/111179">https://doi.org/10.1101/111179</a></td>
            <td>February 23, 2017</td>
        </tr>
        
        <tr>
            <td><b>Histone H3 Lysine 4 methyltransferases MLL3 and MLL4 Modulate Long-range Chromatin Interactions at Enhancers</b>, Jian Yan, Shi-An A. Chen, Andrea Local, Tristin Liu, Yunjiang Qiu, Ah-Young Lee, Inkyung Jung, Sebastian Preissl, Chloe M Rivera, Chaocheng Wang, Haruhiko Ishii, Rongxin Fang, Zhen Ye, Kai Ge, Ming Hu, Bing Ren, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/110239">https://doi.org/10.1101/110239</a></td>
            <td>February 21, 2017</td>
        </tr>
        
        <tr>
            <td><b>Comprehensive single cell transcriptional profiling of a multicellular organism by combinatorial indexing</b>, Junyue Cao, Jonathan S. Packer, Vijay Ramani, Darren A. Cusanovich, Chau Huynh, Riza Daza, Xiaojie Qiu, Choli Lee, Scott N. Furlan, Frank J. Steemers, Andrew Adey, Robert H. Waterston, Cole Trapnell, Jay Shendure, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/104844">https://doi.org/10.1101/104844</a></td>
            <td>February 2, 2017</td>
        </tr>
        
        <tr>
            <td><b>Direct visualization of transcriptional activation by physical enhancer-promoter proximity</b>, Hongtao Chen and Thomas Gregor, bioRxiv, doi: <a class="ex1" target="_blank" href="http://biorxiv.org/content/early/2017/01/11/099523">https://doi.org/10.1101/099523</a></td>
            <td>January 31, 2017</td>
        </tr>
        
        <tr>
            <td><b>Nucleotide sequence and DNaseI sensitivity are predictive of 3D chromatin architecture</b>, Jacob Schreiber, Maxwell Libbrecht, Jeffrey Bilmes, William Noble, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/103614">https://doi.org/10.1101/103614</a></td>
            <td>January 30, 2017</td>
        </tr>
        
        <tr>
            <td><b>PGS: a dynamic and automated population-based genome structure software</b>, Nan Hua, ..., Frank Alber, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/103358">https://doi.org/10.1101/103358</a></td>
            <td>January 26, 2017</td>
        </tr>
        
        <tr>
            <td><b>Multiplexed dynamic imaging of genomic loci in single cells by combined CRISPR imaging and DNA sequential FISH</b>, Yodai Takei, Sheel Shah, Sho Harvey, Lei S Qi, Long Cai, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/101477">https://doi.org/10.1101/101477</a></td>
            <td>January 18, 2017</td>
        </tr>
        
        <tr>
            <td><b>The 3D genome organization of Drosophila melanogaster through data integration</b>, Qingjiao Li, Harianto Tjong, Xiao Li, Ke Gong, Xianghong Jasmine Zhou, Irene Chiolo, Frank Alber, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/099911">https://doi.org/10.1101/099911</a></td>
            <td>January 15, 2017</td>
        </tr>
        
        <tr>
            <td><b>Polycomb-Mediated Chromatin Loops Revealed by a Sub-Kilobase Resolution Chromatin Interaction Map</b>, Kyle P. Eagen, Erez Lieberman Aiden, Roger D. Kornberg, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/099804">https://doi.org/10.1101/099804</a></td>
            <td>January 12, 2017</td>
        </tr>
        
        <tr>
            <td><b>Targeted degradation of CTCF decouples local insulation of chromosome domains from higher-order genomic compartmentalization</b>, Elphege P Nora, Anton Goloborodko, Anne-Laure Valton, Johan Harmen Gibcus, Alec Uebersohn, Nezar Abdennur, Job Dekker, Leonid Mirny, Benoit Bruneau, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/095802">https://doi.org/10.1101/095802</a></td>
            <td>January 9, 2017</td>
        </tr>
        
        <tr>
            <td><b>Cohesin dependent compaction of mitotic chromosomes</b>, Stephanie A Schalbetter, ..., Jon Baxter, bioRxiv, doi: <a class="ex1" target="_blank" href="http://biorxiv.org/content/early/2016/12/17/094946">https://doi.org/10.1101/094946</a></td>
            <td>December 17, 2016</td>
        </tr>
        
        <tr>
            <td><b>Cell cycle dynamics of chromosomal organisation at single-cell resolution</b>, Peter Fraser and Amos Tanay, bioRxiv, doi: <a class="ex1" target="_blank" href="http://biorxiv.org/content/early/2016/12/15/094466">https://doi.org/10.1101/094466</a></td>
            <td>December 15, 2016</td>
        </tr>
        
        <tr>
            <td><b>Detecting hierarchical 3-D genome domain reconfiguration with network modularity</b>, Heidi K Norton, Harvey Huang, Daniel J Emerson, Jesi Kim, Shi Gu, Danielle S Bassett, Jennifer E Phillips-Cremins, bioRxiv, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/089011">https://doi.org/10.1101/089011</a></td>
            <td>November 22, 2016</td>
        </tr>
        
        <tr>
            <td><b>Calibrating photon counts from a single image</b>, Bernd Rieger, arXiv: <a class="ex1" target="_blank" href="https://arxiv.org/abs/1611.05654">1611.05654</a></td>
            <td>November 17, 2016</td>
        </tr>
        
        <tr>
            <td><b>Predicting Enhancer-Promoter Interaction from Genomic Sequence with Deep Neural Networks</b>, Shashank Singh, Yang Yang, Barnabas Poczos, Jian Ma, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/085241">https://doi.org/10.1101/085241</a></td>
            <td>November 2, 2016</td>
        </tr>
        
        <tr>
            <td><b>Micro-C XL: assaying chromosome conformation at length scales from the nucleosome to the entire genome</b>, Tsung-Han Hsieh, Geoffrey Fudenberg, Anton Goloborodko, Oliver Rando, doi: <a class="ex1" target="_blank" href="https://doi.org/10.1101/071357">https://doi.org/10.1101/071357</a></td>
            <td>August 24, 2016</td>
        </tr>
        
    </table>
</html>

			<!-- InstanceEndEditable -->
            <div id="latestDevelopments"></div>
          </section>
          <section id="sidebar">
            <nav>
              <div><a href="https://data.4dnucleome.org/" target="_blank">
                  <div id="dataPortalSideButton" class="sideButton">Launch Data Portal
                    <div class="sideButtonArrow">➧</div>
                  </div>
                </a>
                <div class="sideText">The 4D Nucleome Data Portal hosts data generated by the 4DN Network and other
                  reference nucleomics data sets, and an expanding tool set for open data processing and visualization.
                </div>
                <a href="http://wiki.4dnucleome.org/" target="_blank">
                  <div id="wikiSideButton" class="sideButton">Internal Wiki
                    <div class="sideButtonArrow">➧</div>
                  </div>
                </a>
                <div class="sideText">The 4D Nucleome Network Wiki is an internal collaboration site, accessible to 4DN
					investigators and members only.
                </div>
              </div>
              <div class="sidePanel framedContainer">
                <div class="sidePanelHeader"> Multimedia Library </div>
                <div class="sidePanelBody">
                  <div><a href="https://www.youtube.com/channel/UCe6r3u3rmaYAKSEuTLmfFxA" target="_blank"> <img
                        src="Assets/images/icons/youtube_social_icon_red.png" width="24" alt="Youtube icon" />Youtube
                    </a></div>
                </div>
              </div>
              <div class="sidePanel framedContainer">
                <div class="sidePanelHeader"> Stay Connected </div>
                <div class="sidePanelBody">
                  <div><a href="https://www.facebook.com/4DNucleome-1840252952876682/" target="_blank"> <img
                        src="Assets/images/icons/f-ogo_RGB_HEX-58.svg" alt="Facebook icon" />Facebook </a></div>
                  <div><a href="https://twitter.com/4DNucleome" target="_blank"><img
                        src="Assets/images/icons/Twitter_Logo_Blue.svg" alt="Twitter icon" />Twitter </a></div>
                  <div><a href="https://www.linkedin.com/in/4d-nucleome-857952126/" target="_blank"><img
                        src="Assets/images/icons/In-2C-28px-R.png" id="LinkedIn" alt="LinkedIn icon" />LinkedIn </a>
                  </div>
                  <div><a href="mailto:oh@4dnucleome.org"><img src="Assets/images/icons/Email-iron-icon.svg"
                        alt="Email icon" />Email </a></div>
                </div>
              </div>
            </nav>
          </section>
        </div>
        <footer>
          <div class="logoes"> <a href="https://www.facebook.com/4DNucleome-1840252952876682/" target="_blank"><img
                src="Assets/images/icons/flogo-HexRBG-Wht-58.svg" alt="" /></a> <a
              href="https://twitter.com/4DNucleome" target="_blank"><img
                src="Assets/images/icons/Twitter_Logo_White.svg" alt="" /></a> <a
              href="mailto:oh@4dnucleome.org"><img src="Assets/images/icons/Email-iron-icon-white.svg" alt="" /></a>
            <a href="https://www.youtube.com/channel/UCe6r3u3rmaYAKSEuTLmfFxA" target="_blank"> <img
                src="Assets/images/icons/youtube_dark.png" class="youtube" alt="" /></a> <a
              href="https://www.linkedin.com/in/4d-nucleome-857952126/" target="_blank"><img
                src="Assets/images/icons/In-White-28px-R.png" alt="" /></a> </div>
          <div>4D Nucleome Organizational Hub</div>
        </footer>
      </div>
    </div>
  </div>
  <div class="mainMargin">
    <!-- right margin -->
  </div>
</body>

<!-- InstanceEnd --></html>
