<!doctype html>
<html><!-- InstanceBegin template="/Templates/mainPage.dwt" codeOutsideHTMLIsLocked="false" -->
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<!-- InstanceBeginEditable name="doctitle" -->
<title>4D Network Web Portal</title>
<!-- InstanceEndEditable -->
<link href="Assets/styles/mainStyle.css" rel="stylesheet" type="text/css">
<!-- InstanceBeginEditable name="head" -->
<!-- InstanceEndEditable -->
</head>

<body>
<div id="mainwrapper">
  <header> 
    <div id="logo"><img src="Assets/images/4dn-logo_1.png" alt=""/><div>4DN Web Portal</div></div>
	<div id="headerRight">
	  <nav> <a href="#" title="Link">Launch Data Portal</a><a href="#" title="Link">Contact Us</a></nav>
	  <input type="text" placeholder="Search">
	</div>
  </header>
  <div id="content">
	<nav id="topMenu">
	  <div>
		  Home
	  </div>
	  <div>
		  Cell Lines and Protocols
		  <div>
			  <div>
				  Cell Lines
			  </div>
			  <div>
				  Protocols
			  </div>
		  </div>
	  </div>
	  <div>
		  Publications
		  <div>
			  <div>
				  4DN Publications
			  </div>
			  <div>
				  4DN Preprints
			  </div>
			  <div>
				  Further Reading
			  </div>
		  </div>
	  </div>
	  <div>
		  About 4D Nucleome
		  <div>
			  <div>
				  Program Overview
			  </div>
			  <div>
				  People and Contacts
			  </div>
			  <div>
				  Policies
			  </div>
			  <div>
				  Funding Oppotunities
			  </div>
		  </div>
	  </div>
	  <div>
		  More...
		  <div>
		  <div>
			  Outreach
	      </div>
		  <div>
			  Policies
		  </div>
		  <div>
			  Funding Oppotunities
		  </div>
		  </div>
	  </div>
	</nav>
	<div id="middleContainer">
		<section id="mainContent">
			<div id="breadcrumb" class="active">Home > Templates</div>
			<!-- InstanceBeginEditable name="Main contents" -->
		
	<h2>4DN Protocols</h2>
	<p>Collection of genomic technologies currently in use or being developed in the 4DN network.</p>
	<br/>
			
		<table id="GeneralTables" style="width:100%">

        <tr id="Header1">
            <th colspan="3">4DN Steering Committee approved protocols</th>
        </tr>
        
		<tr id="Header2">
			<th width="25%">Protocol</th>
			<th width="25%">Date of approval</th>
			<th width="50%">Description</th>
		</tr>

<tr>
            <td><a class="ex1" target="_blank" href="https://drive.google.com/open?id=1ZBMtjLrEgpJdUCdmu4utrFChN0XjwYvJO9DOXNY_qsw">Single-cell Hi-C Protocol</a></td>
            <td>April 17, 2018</td>
            <td>Single-cell Hi-C Protocol and Quality Control Standards</td>
</tr>

<tr>
            <td><a class="ex1" target="_blank" href="https://www.nature.com/articles/nprot.2016.126">DNase Hi-C protocol</a></td>
            <td>January 16, 2018</td>
            <td>Mapping 3D genome architecture through in situ DNase Hi-C</td>
    </tr>


<tr>
            <td><a class="ex1" target="_blank" href="https://drive.google.com/open?id=1E9zPnHvcjsPvt2kiZhEH2nBx7AOU21Eu">ChIA-PET data analysis</a></td>
            <td>October 17, 2017</td>
            <td>ChIA-PET data processing pipeline and standards, 4D Nucleome Consortium</td>
    </tr>
	
<tr>
            <td><a class="ex1" target="_blank" href="https://drive.google.com/open?id=1rUAIJAFk5-kHzf7mUUME5mDwV892w7Ft">ChIA-PET experimental protocol</a></td>
            <td>October 17, 2017</td>
            <td>ChIA-PET Protocol and Standards, 4D Nucleome Consortium</td>
    </tr>
	
    <tr>
            <td><a class="ex1" target="_blank" href="https://drive.google.com/open?id=1xq4w8mMS76aHEL42GPOcsma0RUxoCJFnJL0UxLlknCo">E/L Repli-seq</a></td>
            <td>June 20, 2017</td>
            <td>E/L Repli-seq: Protocol and Quality Control Standards, 4D Nucleome Consortium</td>
    </tr>
	
    <tr>
        <td><a class="ex1" target="_blank" href="https://drive.google.com/open?id=1fqvDMx0MZVsURnqTyIoO7k8BLJcLkUCoy1kJHlhaMaA">E/L Repli-seq</a></td>
        <td>June 20, 2017</td>
        <td>4D Nucleome Consortium, Overall Standards and Guidelines for E/L Repli-seq Experiments</td>
    </tr>
    
    
	<tr>
            <td><a class="ex1" target="_blank" href="https://drive.google.com/open?id=1-NEldtpuDuYXcbWngETltNPCRv0RZlIK">Hi-C</a></td>
            <td>February 21, 2017</td>
	    <td>Standards and Guidelines for Hi-C experiments</td>
	</tr>
	
	<tr>
            <td><a class="ex1" target="_blank" href="https://drive.google.com/open?id=1wEAR_yGKMFBZ6BQ6CRRtBa8USYdRZahM">In situ Hi-C</a></td>
            <td>February 21, 2017</td>
	    <td>Protocol and Quality Control Standards for in situ Hi-C experiments</td>
	</tr>
	
    </table>
</html>

		<br/>
		<br/>
    <table id="GeneralTables" style="width:100%">
        
		<tr id="Header1">
			<th colspan="3">Other relevant protocols</th>
		</tr>
		
        <tr id="Header2">
            <th width="15%">Technology (Protocol)</th>
            <th width="35%">Reference paper</th>
            <th width="50%">Description</th>
        </tr>
        
        
        <tr>
            <td><a class="ex1" target="_blank" href="http://link.springer.com/protocol/10.1007/978-1-60327-461-6_7">3C-seq</a></td>
            <td><a class="ex1" target="_blank" href="http://science.sciencemag.org/content/sci/295/5558/1306.full.pdf">Capturing Chromosome Conformation
                (Dekker et al, Science, 2002)</a></td>
            <td><b>Chromosome conformation capture techniques</b> are used to analyze the organization of chromatin in a cell by quantifying the interactions between genomic loci that are nearby in 3-D space.
                3C quantifies interactions between a single pair of genomic loci (one-vs-one).</td>
        </tr>
        
        <tr>
            <td><a class="ex1" target="_blank" href="https://drive.google.com/file/d/0B4D9q1N_g3nCcXRPUlNGb0VWb3M/view">4C-seq</a></td>
            <td><a class="ex1" target="_blank" href="https://www.ncbi.nlm.nih.gov/pubmed/22929766">4C technology: protocols and data analysis
                (Van de Werken et al, Methods Enzymol, 2012)</a></td>
            <td><b>Chromosome conformation capture-on-chip (4C)</b> captures interactions between one locus and all other genomic loci (one-vs-all).</td>
        </tr>
        
        <tr>
            <td><a class="ex1" target="_blank" href="http://www.nature.com/nprot/journal/v2/n4/pdf/nprot.2007.116.pdf">5C-seq</a></td>
            <td><a class="ex1" target="_blank" href="http://www.nature.com/nprot/journal/v2/n4/abs/nprot.2007.116.html">Mapping networks of physical interactions between genomic elements using 5C technology
                (Jos√©e Dostie and Job Dekker, Nature Protocols, 2007)</a></td>
                <td><b>Chromosome conformation capture carbon copy (5C)</b> detects interactions between all restriction fragments within a given region, with this region's size typically no greater than a megabase (many-vs-many).</td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="http://science.sciencemag.org/content/sci/suppl/2009/10/08/326.5950.289.DC1/Lieberman-Aiden.SOM.pdf">Hi-C</a></td>
                <td><a class="ex1" target="_blank" href="http://science.sciencemag.org/content/sci/326/5950/289.full.pdf">Comprehensive Mapping of Long-Range Interactions Reveals Folding Principles of the Human Genome (Lieberman-Aiden et al. Science, 2009)</a></td>
                <td><b>Hi-C</b> uses high-throughput sequencing to find the nucleotide sequence of fragments ( all-vs-all).</td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="https://docs.google.com/a/eng.ucsd.edu/viewer?a=v&pid=sites&srcid=NGRudWNsZW9tZS5vcmd8NGQtbnVjbGVvbWUtd2lraXxneDoyZWY2OWFkY2Y4ZWZkMQ">ChIA-PET</a></td>
                <td><a class="ex1" target="_blank" href="http://onlinelibrary.wiley.com/doi/10.1002/jcb.22116/epdf">ChIP-based methods for the identification of long-range chromatin interactions
                (Fullwood and Ruan, Journal of Cellular Biochemistry, 2009)</a></td>
                <td><b>Chromatin Interaction Analysis by Paired-End Tag Sequencing (ChIA-PET)</b> is a technique that incorporates chromatin immunoprecipitation (ChIP)-based enrichment, chromatin proximity ligation, Paired-End Tags, and High-throughput sequencing to determine de novo long-range chromatin interactions genome-wide.</td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="http://www.nature.com/articles/ncomms7178#methods">Capture Hi-C</a></td>
                <td><a class="ex1" target="_blank" href="http://www.nature.com/ng/journal/v47/n6/pdf/ng.3286.pdf">Mapping long-range promoter contacts in human cells with high-resolution capture Hi-C
                (Mifsud et al, Nature Genetics, 2015)</a></td>
                <td><b>Capture Hi-C (CHi-C)</b> is is an adapted technology that selects and enriches few hundred promoters for genome-wide, long-range contacts of both active and inactive promoters.</td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="http://www.cell.com/cms/attachment/2033493464/2049506921/mmc1.pdf">In situ Hi-C</a></td>
                <td><a class="ex1" target="_blank" href="http://www.sciencedirect.com/science/article/pii/S0092867414014974">A 3D Map of the Human Genome at Kilobase Resolution Reveals Principles of Chromatin Looping
                (Rao et al., Cell, 2014)</a></td>
                <td><b>In situ Hi-C</b> method is used to evaluate all DNA-DNA proximity ligation in intact nuclei.</td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="http://www.nature.com/nprot/journal/v11/n11/pdf/nprot.2016.126.pdf">Dnase Hi-C</a></td>
                <td><a class="ex1" target="_blank" href="http://www.nature.com/nprot/journal/v11/n11/pdf/nprot.2016.126.pdf">Mapping 3D genome architecture through in situ DNase Hi-C (Ramani et al., Nature Protocols, 2016)</a></td>
                <td><b>DNAse Hi-C</b> complements high resolution Hi-C approach with restriction enzymes.</td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="http://www.nature.com/protocolexchange/protocols/5105#/procedure">Micro-C</a></td>
                <td><a class="ex1" target="_blank" href="http://www.nature.com/nmeth/journal/v13/n12/pdf/nmeth.4025.pdf">Micro-C XL: assaying chromosome conformation from the nucleosome to the entire genome (Hsieh et al., Nature Methods, 2016)</a></td>
                <td><b>Micro-C</b> enables mono nucleosome-resolution analysis of chromosome folding fragmentation using micrococcal nuclease (MNase). <b>Micro-C XL</b> is implemented by adding long x-linkers to the fragments.</td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="https://www.ganino.com/games/Science/science%20magazine%201993-1994/root/data/Science%201993-1994/pdf/1993_v261_n5118/p5118_0203.pdf">NLA</a></td>
                <td><a class="ex1" target="_blank" href="http://search.proquest.com/docview/213563655?pq-origsite=gscholar">Interaction between transcription regulatory regions of prolactin chromatin (Cullen et al., Science, 1993)</a></td>
                <td><b>Nuclear Ligation Assay (NLA)</b> is an historical method developped in 1993 to determine circularization frequencies of DNA in solution. It inspired 3C method.</td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="http://www.pnas.org/content/suppl/2016/07/13/1609643113.DCSupplemental/pnas.1609643113.sapp.pdf">COLA</a></td>
                <td><a class="ex1" target="_blank" href="http://www.pnas.org/content/113/31/E4504.full.pdf">    
                Deletion of DXZ4 on the human inactive X chromosome alters higher-order genome architecture 
                (Darrow et al., PNAS, 2016)</a></td>
                <td><b>COLA (Concatamer Ligation Assay)</b> is a modified in situ Hi-C protocol that uses CviJI restriction enzyme to digests chromatin into much finer fragments  the original Hi-C method, in order to increase the proportion of reads containing three or more nearby fragments. </td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="http://www.nature.com/nprot/journal/v10/n12/pdf/nprot.2015.127.pdf">Single-cell Hi-C</a></td>
                <td><a class="ex1" target="_blank" href="http://www.nature.com/nprot/journal/v10/n12/abs/nprot.2015.127.html">    
                Single-cell Hi-C for genome-wide detection of chromatin interactions that occur simultaneously in a single cell
                (Nagano et al., Nature Protocols, 2015)</a></td>
                <td><b>Single Cell Hi-C</b> is an adaptation of Hi-C to single-cell analysis, by including in-nucleus ligation.</td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="http://science.sciencemag.org/content/sci/suppl/2015/05/06/science.aab1601.DC1/Cusanovich-SM.pdf">Combinatorial single-cell Hi-C</a></td>
                <td><a class="ex1" target="_blank" href="http://science.sciencemag.org/content/348/6237/910">    
                Multiplex single-cell profiling of chromatin accessibility by combinatorial cellular indexing
                (Cusanovich et al., Science, 2015)</a></td>
                <td><b>Combinatorial single-cell Hi-C</b> complement single-cell Hi-C by adding unique cellular indexing to measure chromatin accessibility in thousands of single cells per assay, circumventing the need for compartmentalization of individual cells.</td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="http://www.sciencedirect.com/science/article/pii/S0092867415005498">Split-pool Barcoding</a></td>
                <td><a class="ex1" target="_blank" href="http://www.cell.com/cms/attachment/2032378908/2048933656/mmc7.pdf">    
                Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets
                (Macosko et al., Cell, 2015)</a></td>
                <td><b>Split-pool barcoding or Drop-seq</b> is a strategy for profiling thousands of individual cells by separating them into nanoliter-sized aqueous droplets and associating a different barcode with each cell's RNAs and sequencing them all together.</td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="http://www.pnas.org/content/suppl/2015/10/23/1518552112.DCSupplemental/pnas.1518552112.sapp.pdf">Hi-C^2</a></td>
                <td><a class="ex1" target="_blank" href="http://www.pnas.org/content/112/47/E6456.full.pdf">    
                Chromatin extrusion explains key features of loop and domain formation in wild-type and engineered genomes (Sanborn et al., PNAS, 2015)</a></td>
                <td><b>Hybrid Capture Hi-C (Hi-C^2)</b> is a technology that combines targeted genomic capture and existing situ Hi-C libraries to observe conformation changes in selected genomic regions.</td>
                </tr>
                
                <tr>
                <td>Repli-Seq</td>
                <td><a class="ex1" target="_blank" href="http://biorxiv.org/content/early/2017/03/01/104653">    
                Repli-seq: genome-wide analysis of replication timing by next-generation sequencing (Marchal et al., bioRxiv, 2017)</a></td>
                <td><b>Repli Seq</b> is a genome-scale approach to map temporally ordered replicating DNA using massively parallel sequencing.</td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="http://www.nature.com/nprot/journal/v9/n6/pdf/nprot.2014.072.pdf">TRIP</a></td>
                <td><a class="ex1" target="_blank" href="http://www.nature.com/nprot/journal/v9/n6/pdf/nprot.2014.072.pdf">    
                Using TRIP for genome-wide position effect analysis in cultured cells
                (Akhtar et al., Nature Protocols, 2014)</a></td>
                <td>This is protocol for analyzing <b>thousands of reporters integrated in parallel (TRIP)</b> at a genome-wide level. TRIP is based on tagging each reporter with a unique barcode, which is used for independent reporter expression analysis and integration site mapping? </td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="http://www.nature.com/nprot/journal/v2/n6/pdf/nprot.2007.148.pdf">DamID</a></td>
                <td><a class="ex1" target="_blank" href="http://www.nature.com/nbt/journal/v18/n4/pdf/nbt0400_424.pdf">    
                Identification of in vivo DNA targets of chromatin proteins using tethered Dam methyltransferase
                (Bas van Steensel and Steven Henikoff, Nature, 2000)</a></td>
                <td><b>DamID</b> enables mapping genome-wide occupancy of interaction sites in vivo, based on the expression of a fusion protein consisting of a protein of interest and DNA adenine methyltransferase (Dam). This leads to methylation of adenines near sites where the protein of interest interacts with the DNA. These methylated sequences are subsequently amplified by a methylation-specific PCR protocol and identified by hybridization to microarrays.</td>
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="http://www.sciencedirect.com/science/article/pii/S0092867413002171">Single-Cell DamID</a></td>
                <td><a class="ex1" target="_blank" href="http://www.sciencedirect.com/science/article/pii/S0092867413002171">    
                Single-Cell Dynamics of Genome-Nuclear Lamina Interactions
                (Kind et al., Cell, 2013)</a></td>
                <td><b>Single Cell DamID</b> enables visualization of in vivo with adenine-6-methylation  of intact cells by couopling DamID technique and engineered DpnI digestion enzyme.
                </tr>
                
                <tr>
                <td><a class="ex1" target="_blank" href="https://elifesciences.org/articles/21856">CUT&amp;RUN</a></td>
                <td><a class="ex1" target="_blank" href="https://elifesciences.org/articles/21856">    
                An efficient targeted nuclease strategy for high-resolution mapping of DNA binding sites
                (Skene and Henikoff, eLIFE, 2017)</a></td>
                <td><b>CUT&amp;RUN</b> is an efficient targeted nuclease strategy for high-resolution mapping of DNA binding sites. Cleavage Under Targets and Release Using Nuclease (CUT&amp;RUN) is an antibody-targeted chromatin profiling method in which micrococcal nuclease tethered to protein A binds to an antibody of choice and cuts immediately adjacent DNA, releasing DNA bound to the antibody target. The procedure is carried out in situ and produces precise transcription factor or histone modification profiles while avoiding crosslinking and solubilization issues. Extremely low backgrounds make profiling possible with typically one tenth of the sequencing depth required for ChIP, and permit profiling using low cell numbers without loss of quality. CUT&amp;RUN can also be used to map long-range genomic contacts.
                </tr>
                
                </table>
			<!-- InstanceEndEditable -->
			<div id="latestDevelopments"></div>
		</section>
		<section id="sidebar"> 
		  <nav>
			  <div id="dataPortalSideButton">Launch Data Portal<div>‚ûß</div></div>
			  <div>Data Portal description.</div>
			  <div id="twitterSidePanel">Twitter</div>
			  <div class="sidePanel">
				  <div class="sidePanelHeader">
					  Multimedia Library
				  </div>
				  <div class="sidePanelBody">
					  <div><img src="Assets/images/icons/youtube_social_icon_red.png" width="24" alt="Youtube icon"/>Youtube</div>
				  </div>
			  </div>
			  <div class="sidePanel">
				  <div class="sidePanelHeader">
					  Stay Connected
				  </div>
				  <div class="sidePanelBody">
					  <div><img src="Assets/images/icons/f-ogo_RGB_HEX-58.svg" alt="Facebook icon"/>Facebook</div>
					  <div><img src="Assets/images/icons/Twitter_Logo_Blue.svg" alt="Twitter icon"/>Twitter</div>
					  <div><img src="Assets/images/icons/In-2C-28px-R.png" id="LinkedIn" alt="LinkedIn icon"/>LinkedIn</div>
					  <div><img src="Assets/images/icons/Email-iron-icon.svg" alt="Email icon"/>Email</div>
				  </div>
			  </div>
		  </nav>
		</section>
	</div>
    <footer>
		<div class="logoes">
			<img src="Assets/images/icons/flogo-HexRBG-Wht-58.svg" alt=""/>
			<img src="Assets/images/icons/Twitter_Logo_White.svg" alt=""/>
			<img src="Assets/images/icons/Email-iron-icon-white.svg" alt=""/>
			<img src="Assets/images/icons/youtube_dark.png" class="youtube" alt=""/>
			<img src="Assets/images/icons/In-White-28px-R.png" alt=""/>
		</div>
		<div>¬© Copyright 2015-2018. All rights reserved.</div>
    </footer>
  </div>
</div>
</body>
<!-- InstanceEnd --></html>
