<!doctype html><html><!-- InstanceBegin template="/Templates/mainPage.dwt" codeOutsideHTMLIsLocked="false" -->

<head>
  <!-- Global site tag (gtag.js) - Google Analytics -->
  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-74740093-2"></script>
  <script>
    window.dataLayer = window.dataLayer || [];
    function gtag() { dataLayer.push(arguments); }
    gtag('js', new Date());

    gtag('config', 'UA-74740093-2');
  </script>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <!-- InstanceBeginEditable name="doctitle" -->
  <title>4DN Software</title>
  <!-- InstanceEndEditable -->
  <link href="Assets/styles/mainStyle.css" rel="stylesheet" type="text/css">
  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
  <link rel="stylesheet" type="text/css" href="Assets/js/slick/slick.css" />
  <link rel="stylesheet" type="text/css" href="Assets/js/slick/slick-theme.css" />
  <!-- InstanceBeginEditable name="head" -->
  <!-- InstanceEndEditable -->
  <script defer type="text/javascript" src="Assets/js/jquery/jquery-3.3.1.min.js"></script>
  <script defer type="text/javascript" src="Assets/js/jquery/jquery-migrate-1.4.1.min.js"></script>
  <script defer type="text/javascript" src="Assets/js/jquery/jquery-migrate-3.0.0.min.js"></script>
  <script defer type="text/javascript" src="Assets/js/slick/slick.min.js"></script>
  <script defer type="text/javascript" src="Assets/js/main.js"></script>
</head>

<body>
  <div class="mainMargin">
    <!-- left margin -->
  </div>
  <div id="columnWrapper">
    <div id="mainwrapper">
      <header class="framedContainer">
        <div id="logo"><a href="index.html"><img src="Assets/images/4dn-logo_1.png" alt="" /></a>
          <div>4DN Web Portal</div>
        </div>
        <div id="headerRight">
          <nav> <span> <a href="https://data.4dnucleome.org/" target="_blank" title="Launch Data Portal">Launch Data
                Portal</a> </span> <span> <a href="mailto:oh@4dnucleome.org" title="Contact us">Contact Us</a> </span>
          </nav>
          <div id="searchInput">
             
		  <script>
			(function() {
			  var cx = '012010507559311424712:8m_5bphxwsi';
			  var gcse = document.createElement('script');
			  gcse.type = 'text/javascript';
			  gcse.async = true;
			  gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
			  var s = document.getElementsByTagName('script')[0];
			  s.parentNode.insertBefore(gcse, s);
			})();
		  </script>
          <gcse:searchbox-only gname="4dnsitewidesearch"></gcse:searchbox-only>
          
          </div>
        </div>
      </header>
      <div id="content" class="framedContainer">
        <nav id="topMenu" tabindex="0">
          <div class="mobileOnly">
            <i class="material-icons" id="navMenuIcon">
              menu
            </i>
            Navigation Menu</div>
          <div id="topMenuContent">
            <div class="topMenuItem"> <a href="index.html">Home</a> </div>
            <div class="topMenuItem"> <a href="https://commonfund.nih.gov/4dnucleome" target="_blank">Overview
				<i class="material-icons openInNewIcons">open_in_new</i></a>
            </div>
            <div class="topMenuItem"> <a href="https://data.4dnucleome.org/" target="_blank">Data
				<i class="material-icons openInNewIcons">open_in_new</i></a>
			</div>
            <div class="topMenuItem"> Resources
              <div class="subMenu">
                <div> <a href="cell-lines.html">Cell Lines</a> </div>
                <div> <a href="protocols.html">Protocols</a> </div>
                <div> <a href="software.html">Software</a> </div>
              </div>
            </div>
            <div class="topMenuItem"> Publications
              <div class="subMenu">
                <div> <a href="https://www.ncbi.nlm.nih.gov/sites/myncbi/1vMpf2j6i3bke/collections/50779401/public/"
                    target="_blank">4DN Publications <i class="material-icons openInNewIcons">open_in_new</i></a> </div>
                <div> <a href="pre-prints.html">4DN Preprints</a> </div>
                <div> Further Reading
                  <div class="subMenu">
                    <div> <a href="https://www.ncbi.nlm.nih.gov/sites/myncbi/1vMpf2j6i3bke/collections/50130949/public/"
                        target="_blank">Genome Interactions <i class="material-icons openInNewIcons">open_in_new</i></a> </div>
                    <div> <a href="https://www.ncbi.nlm.nih.gov/sites/myncbi/1vMpf2j6i3bke/collections/50131107/public/"
                        target="_blank">Data Analysis <i class="material-icons openInNewIcons">open_in_new</i></a> </div>
                  </div>
                </div>
              </div>
            </div>
            <div class="topMenuItem"> <a href="policies.html">Policies</a> </div>
            <div class="topMenuItem"> <a href="outreach.html">Outreach</a> </div>
            <div class="topMenuItem"> Contacts
              <div class="subMenu">
                <div> <a href="centers-and-teams.html">4DN Centers and Teams</a> </div>
                <div> <a href="TCPA-members.html">TCPA Awardees</a> </div>
                <div> <a href="associate-members.html">Associate Members</a> </div>
                <div> <a href="external-program-consultants.html">External Program Consultants</a> </div>
              </div>
            </div>
          </div>
        </nav>
        <div id="middleContainer">
          <section id="mainContent">
            <!-- InstanceBeginEditable name="Main contents" -->
            <h2>4DN Software</h2>
            <div style="display: flex; flex-direction: row; flex-wrap: wrap;">
              <div class="sliderWrapper framedContainer">
                <div class="prevArrow">◀</div>
                <div id="mainSliderHolder" class="sliderHolder">
                  <div class="slickSlider unresolved">
                    <div>
                      <div class="flexColumnContainer"> <a href="http://www.aidenlab.org/juicebox/" class="flexColumnContainer flexAllCenterContainer">
                          <div class="sliderImg"><img src="Assets/images/schermata-2017-08-01-alle-19-26-07_orig.png"
                              style="margin-bottom: 1em;"></div>
                        </a>
                        <div>Juicebox is a cloud-based software for visualizing data from Hi-C and other proximity
                          mapping experiments. Juicebox allows users to zoom in and out of Hi-C maps interactively,
                          just as a user of Google Earth might zoom in and out of a geographic map. Maps can be
                          compared to one another, or to 1D tracks or 2D feature sets.</div>
                      </div>
                    </div>
                    <div>
                      <div class="flexColumnContainer"> <a href="http://epigenomegateway.wustl.edu/browser/" class="flexColumnContainer flexAllCenterContainer">
                          <div class="sliderImg"><img src="Assets/images/4dn-washu-highlight.jpg" style="margin-bottom: 1em;"></div>
                        </a>
                        <div>WashU Epigenome Browser is a web-based genomics browser which provides interactive
                          visualization, integration, comparison and analysis of large genomics data. It allows
                          investigators to explore epigenomic data in the context of higher-order chromatin
                          interactions. It hosts a large amount of datasets from international consortium including
                          Roadmap, ENCODE, and 4DN projects, and supports user's own data. The Browser can be easily
                          installed locally or on Cloud services for investigating private data.</div>
                      </div>
                    </div>
                    <div>
                      <div class="flexColumnContainer"> <a href="http://promoter.bx.psu.edu/hi-c/index.html" class="flexColumnContainer flexAllCenterContainer">
                          <div class="sliderImg"><img src="Assets/images/3dg_screenshot.png" style="margin-bottom: 1em;"></div>
                        </a>
                        <div>With 3D Genome Browser, users can join 30,000 other users from over 100 countries to
                          visualize and explore chromatin interaction data, such as Hi-C, ChIA-PET, Capture Hi-C,
                          PLAC-Seq, and more.</div>
                      </div>
                    </div>
                    <div>
                      <div class="flexColumnContainer"> <a href="https://www.givengine.org" class="flexColumnContainer flexAllCenterContainer">
                          <div class="sliderImg"><img src="Assets/images/give-hic_orig.png" style="margin-bottom: 1em;"></div>
                        </a>
                        <div>GIVE is an open source programming library that allows anyone with HTML programming
                          experience to build custom genome browser websites or apps. With a few lines of codes, one
                          can add to a personal webpage an interactive genome browser that host custom data. It
                          typically takes less than half a day to build a genome browser website with GIVE. </div>
                      </div>
                    </div>
                    <div>
                      <div class="flexColumnContainer"> <a href="http://higlass.io" class="flexColumnContainer flexAllCenterContainer">
                          <div class="sliderImg"><img src="Assets/images/higlass-screenshot_orig.png" style="margin-bottom: 1em;"></div>
                        </a>
                        <div>HiGlass is a web-based viewer for genome interaction maps featuring synchronized
                          navigation of multiple views as well as continuous zooming and panning for navigation across
                          genomic loci and resolutions. It provides a highly configurable user interface which
                          facilitates the visual comparison of Hi-C and other genomic data from different experimental
                          conditions. Examples and documentation can be found at http://higlass.io.</div>
                      </div>
                    </div>
                  </div>
                </div>
                <div class="nextArrow">▶</div>
              </div>
              <div class="widgetHolder">
                <p>Here we compiled a list of tools available for data analysis and/or visualization of 4DN-related
                  datasets.</p>
                <p>If you have any suggestions to add/update this list, please send them to us via email:
                  <script>document.write('<' + 'a' + ' ' + 'h' + 'r' + 'e' + 'f' + '=' + "'" + 'm' + 'a' + 'i' + 'l' + 't' + 'o' + '&' + '#' + '5' + '8' + ';' + 'r' + '&' + '#' + '9' + '9' +
                      ';' + '%' + '6' + '1' + 'l' + 'a' + 'n' + '&' + '#' + '3' + '7' + ';' + '6' + '4' + 'r' + 'e' + '%' + '6' + 'C' + 'l' + 'i' + '&' + '#' + '6' + '4' +
                      ';' + '&' + '#' + '1' + '0' + '1' + ';' + '%' + '&' + '#' + '5' + '4' + ';' + '&' + '#' + '6' + '9' + ';' + '&' + '#' + '1' + '0' + '3' + ';' + '&' +
                      '#' + '3' + '7' + ';' + '2' + 'E' + 'u' + '%' + '6' + '3' + 's' + '&' + '#' + '1' + '0' + '0' + ';' + '&' + '#' + '4' + '6' + ';' + 'e' + 'd' + '&' +
                      '#' + '1' + '1' + '7' + ';' + "'" + '>' + 'r' + 'c' + '&' + '#' + '9' + '7' + ';' + '&' + '#' + '1' + '0' + '8' + ';' + 'a' + 'n' + 'd' + 'r' + 'e' +
                      'l' + 'l' + 'i' + '&' + '#' + '6' + '4' + ';' + 'e' + '&' + '#' + '1' + '1' + '0' + ';' + 'g' + '&' + '#' + '4' + '6' + ';' + 'u' + 'c' + 's' + 'd' +
                      '&' + '#' + '4' + '6' + ';' + 'e' + 'd' + '&' + '#' + '1' + '1' + '7' + ';' + '<' + '/' + 'a' + '>');</script><noscript>[Turn
                    on JavaScript to see the
                    email address]</noscript>.</p>
              </div>
            </div>
            <table id="GeneralTables" style="width:100%">
              <tr id="Header2">
                <th width="20%">Software</th>
                <th width="10%">Data Type</th>
                <th width="70%">Description</th>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://promoter.bx.psu.edu/hi-c/">3D Genome
                    Browser</a></td>
                <td label="dataType">Hi-C,
                  Virtual 4C</td>
                <td>The 3D Genome Browser provides heatmap virtualizations of the Hi-C intra-chromosomal contract
                  matrices.
                  The browser allows the exploration of potential cis-regulatory elements and/or topologically
                  associating domains (TADs) for
                  the gene or genomic feature of interest.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://www.kobic.kr/3div/">3DIV</a></td>
                <td label="dataType">Hi-C</td>
                <td>3DIV is a database which processed 315 Hi-C experiments from 80 human samples. 3DIV is an
                  easy-to-use database providing epigenetic annotations and one-to-all chromatin interaction
                  visualization with a variety of options.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://github.com/rr1859/R.4Cker">4C-ker</a></td>
                <td label="dataType">4C</td>
                <td>4C-ker is a method developed to analyze 4C-Seq (circularized chromosome conformation capture) data.
                  4C-ker is a Hidden-Markov Model based pipeline that identifies regions throughout the genome that
                  interact with the 4C bait locus.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://ibi.hzau.edu.cn/3dmodel/index.php">AutoChrom3D</a></td>
                <td label="dataType">3C, 4C, 5C,
                  Hi-C</td>
                <td>AutoChrom3D is an automatic pipeline to model chromatin regions with genomic size ranging from
                  hundreds of kilobases to megabases at 8 kb resolution.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://projects.cbio.mines-paristech.fr/centurion/">Centurion</a></td>
                <td label="dataType">Hi-C</td>
                <td>A method that jointly calls all centromeres in a genome-wide Hi-C contact map.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://chiapet.gis.a-star.edu.sg/">ChIA-PET
                    Browser</a></td>
                <td label="dataType">ChIA-PET</td>
                <td>A software package for automatic processing of ChIA-PET sequence data, including linker filtering,
                  mapping tags to reference genomes,
                  identifying protein binding sites and chromatin interactions, and displaying the results on a
                  graphical genome browser.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://folk.uio.no/jonaspau/chiasig/">ChiaSig</a></td>
                <td label="dataType">ChIA-PET</td>
                <td>ChiaSig finds significant interactions based on the NCHG model.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://regulatorygenomicsgroup.org/chicago">CHiCAGO</a></td>
                <td label="dataType">Hi-C</td>
                <td>A set of tools for calling significant interactions in Capture Hi-C data, such as Promoter Capture
                  Hi-C.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://bioinfo.sls.kyushu-u.ac.jp/chromcontact/">ChromContact</a></td>
                <td label="dataType">Hi-C</td>
                <td>ChromContact is a web server for analyzing spatial contact of chromosomes from the publicly
                  available Hi-C data.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://cran.r-project.org/web/packages/chromoR/index.html">chromoR</a></td>
                <td label="dataType">Hi-C</td>
                <td>chromoR provides users with a statistical pipeline for analysing chromosomal interactions data
                  (Hi-C data).</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://biogpu.ddns.comp.nus.edu.sg/~chipseq/ChromSDE/">ChromSDE</a></td>
                <td label="dataType">Hi-C</td>
                <td>A deterministic method which applies semi-definite programming techniques to find the best
                  structure fitting the observed data and uses
                  golden section search to find the correct parameter for converting the contact frequency to spatial
                  distance.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://apps.cytoscape.org/apps/cytohicplugin">CytoHiC</a></td>
                <td label="dataType">Hi-C</td>
                <td>A Cytoscape plugin, which allow users to view and compare spatial maps of genomic landmarks, based
                  on normalized Hi-C datasets.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://www.bioconductor.org/packages/release/bioc/html/diffHic.html">diffHiC</a></td>
                <td label="dataType">Hi-C</td>
                <td>A software package for the detection of differential interactions from Hi-C data. diffHic provides
                  methods for read pair alignment and processing,
                  counting into bin pairs, filtering out low-abundance events and normalization of trended or
                  CNV-driven biases.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://epialign.ucsd.edu/">EpiAlignment</a></td>
                <td label="dataType">Bigwig, Genome annotations</td>
                <td>EpiAlignment is a dynamic programming algorithm for chromosomal similarity search, which is able to
                  align two regions with both sequences and epigenomic information, such as histone modifications.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://cran.r-project.org/web/packages/FisHiCal/index.html">FisHiCal</a></td>
                <td label="dataType">Hi-C</td>
                <td>FisHiCal integrates Hi-C and FISH data, offering a modular and easy-to-use tool for chromosomal
                  spatial analysis.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://noble.gs.washington.edu/proj/fit-hi-c/">Fit-Hi-C</a></td>
                <td label="dataType">Hi-C</td>
                <td>Fit-Hi-C is a tool for assigning statistical confidence estimates to intra-chromosomal contact maps
                  produced by genome-wide genome architecture
                  assays such as Hi-C.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://hicexplorer.usegalaxy.eu/">Galaxy
                    HiCExplorer</a></td>
                <td label="dataType">Hi-C</td>
                <td>A Galaxy based web server to give biomedical researchers an easy to use access to the HiCExplorer
                  (for more information see the related item below in the table) and additional high-throughput
                  analysis software like deeptools or mappers like BWA-MEM or Bowtie are provided.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://dna.engr.latech.edu/~gdash/GDash-landing-page/">G-Dash</a></td>
                <td label="dataType">Bigwig, 2Bit, 3D models in xyz/pdb</td>
                <td>G-Dash is a genome dashboard that integrates the informatics
                  capabilities of a genome browser with controllers for generating and
                  navigating 3D models of DNA, nucleosomes and chromatin from base pairs
                  to mega base pairs and beyond.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://bioconductor.org/packages/release/bioc/html/GenomicInteractions.html">GenomicInteractions</a></td>
                <td label="dataType">Hi-C,
                  ChIA-PET</td>
                <td>R package for handling Genomic interaction data, such as ChIA-PET/Hi-C, annotating genomic features
                  with interaction information
                  and producing various plots/statistics.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://www.genomegitar.org/">GITAR</a></td>
                <td label="dataType">Hi-C</td>
                <td>GITAR (Genome Interaction Tools and Resources) is a standardized way to analyze and visualize
                  genomic interaction data. The tool is composed
                  of two modules: HiCtool (a standardized Python library for processing and visualizing Hi-C data) and
                  processed data (a collection of datasets processed
                  using HiCtool). The pipeline performs also TADs analysis.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://www.givengine.org">GIVE</a></td>
                <td label="dataType">Hi-C,
                  ChIA-PET</td>
                <td>GIVE is an open source programming library that allows anyone with HTML programming experience to
                  build custom genome browser websites or apps. With a few lines of codes, one can add to a personal
                  webpage an interactive genome browser that host custom data. It typically takes less than half a day
                  to build a genome browser website with GIVE.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://www.bioconductor.org/packages/release/bioc/html/GOTHiC.html">GOTHiC</a></td>
                <td label="dataType">Hi-C</td>
                <td>A Hi-C analysis package using a cumulative binomial test to detect interactions between distal
                  genomic loci that have significantly
                  more reads than expected by chance in Hi-C experiments.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://hyperbrowser.uio.no/3d/">HiBrowse</a></td>
                <td label="dataType">Hi-C</td>
                <td>A statistical web toolkit for analyzing, interpreting and visualizing genome-wide chromosome
                  conformation capture data, such as Hi-C, TCC, GCC and similar.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://bitbucket.org/mthjwu/hicapp">HiCapp</a></td>
                <td label="dataType">Hi-C</td>
                <td>HiCapp is a Hi-C analysis pipeline which can correct for the copy number bias in tumor Hi-C data
                  using caICB correction algorithm.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0678-x">HiCdat</a></td>
                <td label="dataType">Hi-C</td>
                <td>HiCdat provides a simple graphical user interface for data pre-processing and a collection of
                  higher-level data analysis tools implemented in R.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://hic.umassmed.edu/welcome/welcome.php">Hi-C
                    Data Browser</a></td>
                <td label="dataType">Hi-C</td>
                <td>The tool enables the generation of genome-wide 3D proximity maps. You will also be able to visually
                  navigate the dataset and explore Hi-C proximity
                  maps of entire human genomes.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://github.com/HiC-inspector/HiC-inspector">HiC-inspector</a></td>
                <td label="dataType">Hi-C</td>
                <td>A bioinformatics pipeline for the automated analysis of data generated by high-throughput chromatin
                  conformation capture (HiC).</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://github.com/deeptools/HiCExplorer">HiCExplorer</a></td>
                <td label="dataType">Hi-C</td>
                <td>Hi-C analysis software which covers the whole pipeline of Hi-C data analysis. Starting with the
                  creation of the contact matrix, the correction, computation of TADs and A / B compartments to a
                  visualization of these. Additional data tracks from other high-throughput analysis like ChIP-Seq or
                  gene tracks can be added to the visualization.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://www.people.fas.harvard.edu/~junliu/HiCNorm/">HiCNorm</a></td>
                <td label="dataType">Hi-C</td>
                <td>A parametric model to remove systematic biases in the raw Hi-C contact maps, resulting in a simple,
                  fast, yet accurate normalization procedure.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://zhoulab.usc.edu/Hi-Corrector/">Hi-Corrector</a></td>
                <td label="dataType">Hi-C</td>
                <td>A fast, scalable and memory-efficient package for normalizing large-scale Hi-C data.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://compgenomics.weizmann.ac.il/tanay/?page_id=283">Hi-C
                    Pipeline</a></td>
                <td label="dataType">Hi-C</td>
                <td>Hicpipe is a set of scripts and programs that correct Hi-C contact maps, given a list restriction
                  enzyme sites and mapped paired reads.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://github.com/kcakdemir/HiCPlotter">HiCPlotter</a></td>
                <td label="dataType">Hi-C</td>
                <td>An easy-to-use open-source visualization tool to facilitate juxtaposition of Hi-C matrices with
                  diverse genomic assay outputs, as well as to
                  compare interaction matrices between various conditions.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-016-3387-6">HiC-bench</a></td>
                <td label="dataType">Hi-C</td>
                <td>HiC-bench is a tool for a comprehensive and reproducible Hi-C data analysis designed for parameter
                  exploration and benchmarking.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0831-x">HiC-Pro</a></td>
                <td label="dataType">Hi-C</td>
                <td>HiC-Pro is an optimized and flexible pipeline for processing Hi-C data from raw reads to normalized
                  contact maps.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://cran.r-project.org/web/packages/HiCseg/index.html">HiCseg</a></td>
                <td label="dataType">Hi-C</td>
                <td>This package allows you to detect domains in HiC data by rephrasing this problem as a
                  two-dimensional segmentation issue.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://sourceforge.net/projects/hictools/">hiCtools</a></td>
                <td label="dataType">Hi-C</td>
                <td>HiC is high-throughput sequencing technology applied to chromatin conformation techniques. This
                  collection of tools stream-lines the processing
                  of HiC data from raw sequence to contact matrices and beyond.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://www.bioinformatics.babraham.ac.uk/projects/hicup/">HiCUP</a></td>
                <td label="dataType">Hi-C</td>
                <td>A tool for mapping and performing quality control on Hi-C data.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://genomebiology.biomedcentral.com/articles/10.1186/s13059-015-0806-y">HiFive</a></td>
                <td label="dataType">5C,
                  Hi-C</td>
                <td>The HiFive tool suite provides efficient data handling and a variety of normalization approaches
                  for easy, fast analysis and method comparison.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://higlass.gehlenborglab.org/">HiGlass</a></td>
                <td label="dataType">Interaction matrices</td>
                <td>HiGlass is a tool for displaying and comparing large matrices within a web page. Matrices are
                  typically visualized as heatmaps.
                  In the case of large matrices, the heatmaps are too large to render all at once. Instead, we
                  aggregate their values and display summaries at
                  lower resolution while allowing one to zoom in and explore them in greater detail.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://hipiler.higlass.io">HiPiler</a></td>
                <td label="dataType">Hi-C</td>
                <td>HiPiler is an interactive web application for feature-centric exploration of Hi-C matrices. HiPiler
                  represents 2D features or annotations (like loops, TADs, CTCF sites, etc.) as individual
                  thumbnail-like snippets. Snippets can be laid out automatically based on their data and meta
                  attributes. They can be interactively aggregated into piles for stratification and are visually
                  linked back to the matrix to provide context.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://wanglab.pcbi.upenn.edu/hippie/">HIPPIE</a></td>
                <td label="dataType">Hi-C</td>
                <td>HIPPIE is the workflow for analyzing batches of Hi-C paired-end reads in compressed FASTQ format
                  (.fastq.gz) and predict enhancer-target gene interactions.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://www.bioconductor.org/packages//2.10/bioc/html/HiTC.html">HiTC</a></td>
                <td>5C,
                  Hi-C</td>
                <td>The package explores high-throughput 'C' data such as 5C or Hi-C.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://www.unc.edu/~yunmli/HMRFBayesHiC/">HMRFBayesHiC</a></td>
                <td label="dataType">Hi-C</td>
                <td>A hidden Markov random field based Bayesian peak caller to identify long range chromatin
                  interactions from Hi-C data.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://homer.ucsd.edu/homer/interactions/">HOMER</a></td>
                <td label="dataType">Hi-C</td>
                <td>HOMER contains several programs and analysis routines to facilitate the analysis of Hi-C data.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://github.com/huangjialiangcn/HubPredictor">HubPredictor</a></td>
                <td label="dataType">Hi-C</td>
                <td>A computational model integrating Hi-C and histone mark ChIP-seq data to predict two important
                  features of chromatin organization:
                  chromatin interaction hubs and topologically associated domain (TAD) boundaries.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://mirnylab.bitbucket.io/hiclib/index.html?">ICE</a></td>
                <td label="dataType">Hi-C</td>
                <td>A computational pipeline that integrates a strategy to map sequencing reads with a data-driven
                  method for iterative correction of biases,
                  yielding genome-wide maps of relative contact probabilities.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://www.aidenlab.org/juicebox/">Juicebox</a></td>
                <td label="dataType">Hi-C</td>
                <td>Software for visualizing data from Hi-C and other proximity mapping experiments.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://aidenlab.org/juicer/">Juicer</a></td>
                <td label="dataType">Hi-C</td>
                <td>Juicer is a one-click system for analyzing loop-resolution Hi-C experiments. Specifically, the
                  software is an end-to-end automated pipeline
                  for converting raw reads into Hi-C maps and loop calls using only a single command.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://shendurelab.github.io/LACHESIS/">LACHESIS</a></td>
                <td label="dataType">Hi-C</td>
                <td>A software tool to measure the thread of life. LACHESIS exploits contact probability map data (e.g.
                  from Hi-C) for chromosome-scale de novo genome assembly.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://github.com/dphansti/mango">Mango</a></td>
                <td label="dataType">ChIA-PET</td>
                <td>A complete ChIA-PET data analysis pipeline that provides statistical confidence estimates for
                  interactions and corrects for major sources of bias
                  including differential peak enrichment and genomic proximity.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://www.stat.osu.edu/~statgen/SOFTWARE/MDM/">MDM</a></td>
                <td label="dataType">ChIA-PET</td>
                <td>MDM provides functions for the analysis of chromatin interactions using MC_DIST model, Two-Step
                  model and One-Step model, based on ChIA-PET data.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://bioinfo.au.tsinghua.edu.cn/member/xwwang/MICCusage/">MICC</a></td>
                <td label="dataType">ChIA-PET</td>
                <td>An easy-to-use R package to detect chromatin interactions from ChIA-PET sequencing data. By
                  applying a Bayesian mixture model to systematically
                  remove random ligation and random collision noise, MICC could identify chromatin interactions with a
                  significantly higher sensitivity than existing
                  methods at the same false discovery rate.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://calla.rnet.missouri.edu/mogen/">MOGEN</a></td>
                <td label="dataType">Hi-C</td>
                <td>A 3D chromosome reconstruction method to make it capable of reconstructing 3D models of genomes
                  from both intra- and inter-chromosomal Hi-C contact
                  data.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0075146">NuChart</a></td>
                <td label="dataType">Hi-C</td>
                <td>An R Package to Study Gene Spatial Neighbourhoods with Multi-Omics Annotations.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://cbio.ensmp.fr/~nvaroquaux/pastis/">PASTIS</a></td>
                <td label="dataType">Hi-C</td>
                <td>Novel approach to infer a consensus three-dimensional structure of a genome from Hi-C data. The
                  method incorporates a statistical model
                  of the contact counts, assuming that the counts between two loci follow a Poisson distribution whose
                  intensity decreases with the physical distances
                  between the loci.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://bioconductor.org/packages/release/bioc/html/R3CPET.html">R3CPET</a></td>
                <td label="dataType">ChIA-PET</td>
                <td>The package provides a method to infer the set of proteins that are more probably to work together
                  to maintain chormatin interaction given a
                  ChIA-PET experiment results.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://github.com/3DGenomes/TADbit">TADbit</a></td>
                <td label="dataType">Hi-C</td>
                <td>An algorithm for the identification of hierarchical topological domains in Hi-C data.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://compbio.cs.brown.edu/projects/tadtree/">TADtree</a></td>
                <td label="dataType">Hi-C</td>
                <td>TADbit is a complete Python library to deal with all steps to analyze, model and explore 3C-based
                  data.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="http://epigenomegateway.wustl.edu/">WashU
                    Epigenome Browser</a></td>
                <td label="dataType">5C,
                  Hi-C,
                  ChIA-PET</td>
                <td>The browser supports multiple types of long-range genome interaction data. This enables
                  investigators to explore epigenomic data in the context
                  of higher-order chromosomal domains and to generate multiple types of intuitive, publication-quality
                  figures of interactions.</td>
              </tr>
              <tr>
                <td label="name"><a class="ex1" target="_blank" href="https://www.nature.com/articles/s41467-017-00478-8.epdf">GMAP</a></td>
                <td label="dataType"><a class="ex1" target="_blank" href="https://elifesciences.org/articles/21856">
                    Hi-C </a></td>
                <td>GMAP (Gaussian Mixture model And Proportion test), is an algorithm to identify topologically
                  associating domains and subdomains (Yu et al., Nature Communications, 2017). It provides a new and
                  efficient tool to analyze simulated and experimental Hi-C data with increased statistical robustness
                  and better consistency of the predictions compared to three state-of-the-art methods
              </tr>
            </table>

</html>
<!-- InstanceEndEditable -->
            <div id="latestDevelopments"></div>
          </section>
          <section id="sidebar">
            <nav>
              <div><a href="https://data.4dnucleome.org/" target="_blank">
                  <div id="dataPortalSideButton" class="sideButton">Launch Data Portal
                    <div class="sideButtonArrow">➧</div>
                  </div>
                </a>
                <div class="sideText">The 4D Nucleome Data Portal hosts data generated by the 4DN Network and other
                  reference nucleomics data sets, and an expanding tool set for open data processing and visualization.
                </div>
                <a href="http://wiki.4dnucleome.org/" target="_blank">
                  <div id="wikiSideButton" class="sideButton">Internal Wiki
                    <div class="sideButtonArrow">➧</div>
                  </div>
                </a>
                <div class="sideText">The 4D Nucleome Network Wiki is an internal collaboration site, accessible to 4DN
					investigators and members only.
                </div>
              </div>
              <div class="sidePanel framedContainer">
                <div class="sidePanelHeader"> Multimedia Library </div>
                <div class="sidePanelBody">
                  <div><a href="https://www.youtube.com/channel/UCe6r3u3rmaYAKSEuTLmfFxA" target="_blank"> <img
                        src="Assets/images/icons/youtube_social_icon_red.png" width="24" alt="Youtube icon" />Youtube
                    </a></div>
                </div>
              </div>
              <div class="sidePanel framedContainer">
                <div class="sidePanelHeader"> Stay Connected </div>
                <div class="sidePanelBody">
                  <div><a href="https://www.facebook.com/4DNucleome-1840252952876682/" target="_blank"> <img
                        src="Assets/images/icons/f-ogo_RGB_HEX-58.svg" alt="Facebook icon" />Facebook </a></div>
                  <div><a href="https://twitter.com/4DNucleome" target="_blank"><img
                        src="Assets/images/icons/Twitter_Logo_Blue.svg" alt="Twitter icon" />Twitter </a></div>
                  <div><a href="https://www.linkedin.com/in/4d-nucleome-857952126/" target="_blank"><img
                        src="Assets/images/icons/In-2C-28px-R.png" id="LinkedIn" alt="LinkedIn icon" />LinkedIn </a>
                  </div>
                  <div><a href="mailto:oh@4dnucleome.org"><img src="Assets/images/icons/Email-iron-icon.svg"
                        alt="Email icon" />Email </a></div>
                </div>
              </div>
            </nav>
          </section>
        </div>
        <footer>
          <div class="logoes"> <a href="https://www.facebook.com/4DNucleome-1840252952876682/" target="_blank"><img
                src="Assets/images/icons/flogo-HexRBG-Wht-58.svg" alt="" /></a> <a
              href="https://twitter.com/4DNucleome" target="_blank"><img
                src="Assets/images/icons/Twitter_Logo_White.svg" alt="" /></a> <a
              href="mailto:oh@4dnucleome.org"><img src="Assets/images/icons/Email-iron-icon-white.svg" alt="" /></a>
            <a href="https://www.youtube.com/channel/UCe6r3u3rmaYAKSEuTLmfFxA" target="_blank"> <img
                src="Assets/images/icons/youtube_dark.png" class="youtube" alt="" /></a> <a
              href="https://www.linkedin.com/in/4d-nucleome-857952126/" target="_blank"><img
                src="Assets/images/icons/In-White-28px-R.png" alt="" /></a> </div>
          <div>4D Nucleome Organizational Hub</div>
        </footer>
      </div>
    </div>
  </div>
  <div class="mainMargin">
    <!-- right margin -->
  </div>
</body>

<!-- InstanceEnd --></html>